---

title: Genetic markers associated with drought tolerance in maize
abstract: The presently disclosed subject matter relates to methods and compositions for identifying, selecting, and/or producing drought tolerant maize plants or germplasm. Maize plants or germplasm that have been identified, selected, and/or produced by any of the methods of the presently disclosed subject matter are also provided.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09526218&OS=09526218&RS=09526218
owner: Syngenta Participations AG
number: 09526218
owner_city: Basel
owner_country: CH
publication_date: 20141001
---
The presently disclosed subject matter is a divisional of U.S. patent application Ser. No. 12 977 966 filed Dec. 23 2010 and claims the benefit of U.S. Provisional Patent Application Ser. No. 61 289 718 filed Dec. 23 2009 and U.S. Provisional Patent Application Ser. No. 61 369 999 filed Aug. 2 2010 the disclosure of each of which is incorporated herein by reference in its entirety.

A Sequence Listing in ASCII text format submitted under 37 C.F.R. 1.821 entitled 72670 US PX D NAT 2 Sequence Listing ST25 195 kilobytes and was filed with U.S. patent application Ser. No. 12 977 966 filed Dec. 23 2009 which claims the benefit of U.S. Provisional Patent Application Ser. No. 61 289 718 filed Dec. 23 2009 and U.S. Provisional Patent Application Ser. No. 61 369 999 filed Aug. 2 2010 for which this application claims the benefit thereof. The Sequence Listing is attached and filed herewith and is incorporated herein by reference.

The presently disclosed subject matter relates to maize such as maize of the species and methods of breeding the same. More particularly the presently disclosed subject matter relates to maize lines such as lines with one or more improved water optimization genotypes and methods for breeding the same which methods involve in some embodiments genetic marker analysis and or nucleic acid sequence analysis.

Drought is one of the major limitations to maize production worldwide 15 of the world s maize crop is lost every year to drought. Periods of drought stress can occur at any time during the growing season but maize is particularly sensitive to drought stress in the period just before and during flowering. When drought stress occurs during this critical period a significant decrease in grain yield can result.

Identifying genes that enhance the drought tolerance of maize could lead to more efficient crop production by allowing for the identification selection and production of maize plants with enhanced drought tolerance.

As such a goal of plant breeding is to combine in a single plant various desirable traits. For field crops such as corn these traits can include greater yield and better agronomic quality. However genetic loci that influence yield and agronomic quality are not always known and even if known their contributions to such traits are frequently unclear. Thus new loci that can positively influence such desirable traits need to be identified and or the abilities of known loci to do so need to be discovered.

Once discovered these desirable loci can be selected for as part of a breeding program in order to generate plants that carry desirable traits. An exemplary embodiment of a method for generating such plants includes the transfer by introgression of nucleic acid sequences from plants that have desirable genetic information into plants that do not by crossing the plants using traditional breeding techniques.

Desirable loci can be introgressed into commercially available plant varieties using marker assisted selection MAS or marker assisted breeding MAB . MAS and MAB involves the use of one or more of the molecular markers for the identification and selection of those progeny plants that contain one or more loci that encode the desired traits. Such identification and selection can be based on selection of informative markers that are associated with desired traits. MAB can also be used to develop near isogenic lines NIL harboring loci of interest allowing a more detailed study of the effect each locus can have on a desired trait and is also an effective method for development of backcross inbred line BIL populations.

Maize drought is one of the major limitations to maize production worldwide. When drought stress occurs just before or during the flowering period an increase in the length of the anthesis silking interval and a decrease in grain yield can result. 15 of the world s maize crop or in excess of 19 millions tons is lost every year to drought. Identifying candidate genes that can enhance drought stress tolerance in maize could lead to more efficient crop production in affected areas.

What are needed then are new methods and compositions for genetically analyzing varieties with respect to drought tolerance and for employing the information obtained for producing new plants that have improved water optimization traits.

This summary lists several embodiments of the presently disclosed subject matter and in many cases lists variations and permutations of these embodiments. This summary is merely exemplary of the numerous and varied embodiments. Mention of one or more representative features of a given embodiment is likewise exemplary. Such an embodiment can typically exist with or without the feature s mentioned likewise those features can be applied to other embodiments of the presently disclosed subject matter whether listed in this summary or not. To avoid excessive repetition this Summary does not list or suggest all possible combinations of such features.

Compositions and methods for identifying selecting and producing maize plants with enhanced drought tolerance are provided. A drought tolerant maize plant or germplasm is also provided.

In some embodiments methods of identifying a drought tolerant maize plant or germplasm are provided. Such methods can comprise detecting in the maize plant or germplasm a marker associated with enhanced drought tolerance.

In some embodiments methods of producing a drought tolerant maize plant are provided. Such methods can comprise detecting in a maize germplasm the presence of a marker associated with enhanced drought tolerance and producing a progeny plant from said maize germplasm.

In some embodiments the presence of a marker associated with enhanced drought tolerance is detected using a marker probe. In some such embodiments the presence of a marker associated with enhanced drought tolerance is detected in an amplification product from a nucleic acid sample isolated from a maize plant or germplasm. In some embodiments the marker comprises a haplotype and a plurality of probes is used to detect the alleles that make up the haplotype. In some such embodiments the alleles that make up the haplotype are detected in a plurality of amplification products from a nucleic acid sample isolated from a maize plant or germplasm.

In some embodiments methods of selecting a drought tolerant maize plant or germplasm are provided. Such methods can comprise crossing a first maize plant or germplasm with a second maize plant or germplasm wherein the first maize plant or germplasm comprises a marker associated with enhanced drought tolerance and selecting a progeny plant or germplasm that possesses the marker.

In some embodiments methods of introgressing an allele associated with enhanced drought tolerance into a maize plant or germplasm are provided. Such methods can comprise crossing a first maize plant or germplasm comprising an allele associated with enhanced drought tolerance with a second maize plant or germplasm that lacks said allele and repeatedly backcrossing progeny plants comprising said allele with the second maize plant or germplasm to produce a drought tolerant maize plant or germplasm comprising the allele associated with enhanced drought tolerance. Progeny comprising the allele associated with enhanced drought tolerance can be identified by detecting in their genomes the presence of a marker associated with said allele.

Maize plants and or germplasms identified produced or selected by any of the methods of the invention are also provided as are any progeny or seeds derived from a maize plant or germplasm identified produced or selected by these methods.

Non naturally occurring maize plants and or germplasms comprising one or more markers associated with enhanced drought tolerance are also provided.

Isolated and or purified markers associated with enhanced drought tolerance are also provided. Such markers can comprise a nucleotide sequence at least 85 90 95 or 99 identical to any of SEQ ID NOs 1 117 400 and 401 the reverse complement thereof or an informative or functional fragment thereof.

Compositions comprising a primer pair capable of amplifying a nucleic acid sample isolated from a maize plant or germplasm to generate a marker associated with enhanced drought tolerance are also provided. Such compositions can comprise consist essentially of or consist of one of the amplification primer pairs identified in Table 1.

A marker associated with enhanced drought tolerance can comprise consist essentially of and or consist of a single allele or a combination of alleles at one or more genetic loci.

Thus in some embodiments the presently disclosed subject matter provides methods for producing a hybrid plant with enhanced water optimization. In some embodiments the methods comprise a providing a first plant comprising a first genotype comprising any one of haplotypes A M b providing a second plant comprising a second genotype comprising any one of haplotypes A M wherein the second plant comprises at least one of haplotypes A M that is not present in the first plant c crossing the first plant and the second maize plant to produce an F1 generation identifying one or more members of the F1 generation that comprises a desired genotype comprising any combination of haplotypes A M wherein the desired genotype differs from both the first genotype of a and the second genotype of b whereby a hybrid plant with enhanced water optimization is produced. In some embodiments haplotypes A M are defined as follows 

i. Haplotype A comprises a G nucleotide at the position that corresponds to position 115 of SEQ ID NO 1 an A nucleotide at the position that corresponds to position 270 of SEQ ID NO 1 a T nucleotide at the position that corresponds to position 301 of SEQ ID NO 1 and an A nucleotide at the position that corresponds to position 483 of SEQ ID NO 1 on chromosome 8 in the first plant s genome 

ii. Haplotype B comprises a deletion at positions 4497 4498 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 4505 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4609 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4792 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4836 of SEQ ID NO 7 a C nucleotide at the position that corresponds to position 4844 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 4969 of SEQ ID NO 7 and a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 on chromosome 8 in the first plant s genome 

iii. Haplotype C comprises an A nucleotide at the position that corresponds to position 217 of SEQ ID NO 8 a G nucleotide at the position that corresponds to position 390 of SEQ ID NO 8 and an A nucleotide at the position that corresponds to position 477 of SEQ ID NO 8 on chromosome 2 in the first plant s genome 

iv. Haplotype D comprises a G nucleotide at the position that corresponds to position 182 of SEQ ID NO 19 an A nucleotide at the position that corresponds to position 309 of SEQ ID NO 19 a G nucleotide at the position that corresponds to position 330 of SEQ ID NO 19 and a G nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 on chromosome 8 in the first plant s genome 

v. Haplotype E comprises a C nucleotide at the position that corresponds to position 61 of SEQ ID NO 21 a C nucleotide at the position that corresponds to position 200 of SEQ ID NO 21 and a deletion of nine nucleotides at the positions that corresponds to positions 316 324 of SEQ ID NO 21 on chromosome 5 in the first plant s genome 

vi. Haplotype F comprises a G nucleotide at the position that corresponds to position 64 of SEQ ID NO 27 and a T nucleotide at the position that corresponds to position 254 of SEQ ID NO 27 on chromosome 8 in the first plant s genome 

vii. Haplotype G comprises an C nucleotide at the position that corresponds to position 98 of SEQ ID NO 28 a T nucleotide at the position that corresponds to position 147 of SEQ ID NO 28 a C nucleotide at the position that corresponds to position 224 of SEQ ID NO 28 and a T nucleotide at the position that corresponds to position 496 of SEQ ID NO 28 on chromosome 9 in the first plant s genome 

viii. Haplotype H comprises a T nucleotide at the position that corresponds to position 259 of SEQ ID NO 30 a T nucleotide at the position that corresponds to position 306 of SEQ ID NO 30 an A nucleotide at the position that corresponds to position 398 of SEQ ID NO 30 and a C nucleotide at the position that corresponds to position 1057 of SEQ ID NO 30 on chromosome 4 in the first plant s genome 

ix. Haplotype I comprises a C nucleotide at the position that corresponds to position 500 of SEQ ID NO 36 a G nucleotide at the position that corresponds to position 568 of SEQ ID NO 36 and a T nucleotide at the position that corresponds to position 698 of SEQ ID NO 36 on chromosome 6 in the first plant s genome 

x. Haplotype J comprises an A nucleotide at the position that corresponds to position 238 of SEQ ID NO 42 a deletion of the nucleotides that correspond to positions 266 268 of SEQ ID NO 42 and a C nucleotide at the position that corresponds to position 808 of SEQ ID NO 42 in the first plant s genome 

xi. Haplotype K comprises a C nucleotide at the position that corresponds to position 166 of SEQ ID NO 49 and A nucleotide at the position that corresponds to position 224 of SEQ ID NO 49 a G nucleotide at the position that corresponds to position 650 of SEQ ID NO 49 and a G nucleotide at the position that corresponds to position 892 of SEQ ID NO 49 on chromosome 8 in the first plant s genome 

xii. Haplotype L comprises a C nucleotide at the positions that correspond to positions 83 428 491 and 548 of SEQ ID NO 53 on chromosome 9 in the first plant s genome and

xiii. Haplotype M comprises a C nucleotide at the position that corresponds to position 83 in SEQ ID NO 400 an A nucleotide at the position that corresponds to position 119 of SEQ ID NO 400 and a T nucleotide at the position that corresponds to position 601 of SEQ ID NO 400.

In some embodiments the hybrid plant with enhanced water optimization comprises each of haplotypes A M that are present in the first plant as well as at least one additional haplotype selected from haplotypes A M that is present in the second plant. In some embodiments the first plant is a recurrent parent comprising at least one of haplotypes A M and the second plant is a donor that comprises at least one of haplotypes A M that is not present in the first plant. In some embodiments the first plant is homozygous for at least two three four or five of haplotypes A M. In some embodiments the hybrid plant comprises at least three four five six seven eight or nine of haplotypes A M.

In some embodiments the identifying comprises genotyping one or more members of an F1 generation produced by crossing the first plant and the second plant with respect to each of the haplotypes A M present in either the first plant or the second plant.

In some embodiments enhanced water optimization confers increased or stabilized yield in a water stressed environment as compared to a control plant. In some embodiments the hybrid with enhanced water optimization can be planted at a higher crop density. In some embodiments the hybrid with enhanced water optimization confers no yield drag when under favorable moisture levels.

The presently disclosed subject matter also provides in some embodiments hybrid plants produced by the presently disclosed methods or a cell tissue culture seed or part thereof.

The presently disclosed subject matter also provides in some embodiments inbred plants produced by backcrossing and or selfing and or producing double haploids from the hybrid plants disclosed herein or a cell tissue culture seed or part thereof.

The presently disclosed subject matter also provides in some embodiments inbred or hybrid plants the genome of which comprises at least three four five six seven eight or nine of haplotypes A M wherein haplotypes A M are associated with water optimization and are defined herein. In some embodiments the inbred or hybrid plant comprises a genome comprising Haplotypes C D and G Haplotypes C D and L Haplotypes C G and H Haplotypes C G and I Haplotypes C I and L Haplotypes E G and I Haplotypes F G and H Haplotypes A C F and G Haplotypes C E H and I Haplotypes C G H and I Haplotypes C H I and K Haplotypes C H I and L Haplotypes E F G and H Haplotypes A C G H and I Haplotypes B C D G and L Haplotypes C E G H and I Haplotypes C G H I and L Haplotypes A C G H I and K Haplotypes C E F G H I J K and L Haplotypes C D G and M Haplotypes C D L and M Haplotypes C G H and M Haplotypes C G I and M Haplotypes C I L and M Haplotypes E G I and M Haplotypes F G H and M Haplotypes A C F G and M Haplotypes C E H I and M Haplotypes C G H I and M Haplotypes C H I K and M Haplotypes C H I L and M Haplotypes E F G H and M Haplotypes A C G H I and M Haplotypes B C D G L and M Haplotypes C E G H I and M Haplotypes C G H I L and M Haplotypes A C G H I K and M and Haplotypes C E F G H I J K L and M. In some embodiments the inbred or hybrid plant is a hybrid plant that is homozygous for at least one of Haplotypes A M.

In some embodiments the inbred or hybrid plant comprises a genome comprising Haplotypes A C E G H and I optionally further comprising Haplotype M Haplotypes B C D E F G H I and L optionally further comprising Haplotype M Haplotypes C D E F G H and L optionally further comprising Haplotype M Haplotypes B C D G I and L optionally further comprising Haplotype M Haplotypes B C D E G H I and L optionally further comprising Haplotype M Haplotypes C D E F G H I J K and L optionally further comprising Haplotype M Haplotypes A C G H and I optionally further comprising Haplotype M Haplotypes C E F G H and I optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M Haplotypes C D E F G and H optionally further comprising Haplotype M Haplotypes D E F G and H optionally further comprising Haplotype M Haplotypes A C G H and I optionally further comprising Haplotype M Haplotypes A C E G H I and K optionally further comprising Haplotype M Haplotype C E G H I and L optionally further comprising Haplotype M Haplotypes C D E G H I and L optionally further comprising Haplotype M Haplotypes B C D E G H I and L optionally further comprising Haplotype M Haplotypes A C G H and I optionally further comprising Haplotype M Haplotypes A C G H I and K optionally further comprising Haplotype M Haplotypes C G H I and L optionally further comprising Haplotype M Haplotypes C D G H I and L optionally further comprising Haplotype M Haplotypes B C D G H I and L optionally further comprising Haplotype M Haplotypes A C E F G H and I optionally further comprising Haplotype M Haplotypes A C E F G H I and K optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M Haplotypes C D E F G H I and L optionally further comprising Haplotype M Haplotypes A C E F G H I J K and L optionally further comprising Haplotype M Haplotypes A C E F G H I J K and L optionally further comprising Haplotype M Haplotypes C E F G H I J K and L optionally further comprising Haplotype M Haplotypes C D E F G H I J K and L optionally further comprising Haplotype M Haplotypes B C D E F G H I J K and L optionally further comprising Haplotype M Haplotypes A C E F G H and I optionally further comprising Haplotype M Haplotypes A C E F G H I and K optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M Haplotypes B C D E F G H and L optionally further comprising Haplotype M Haplotypes C E F G H I J K and L optionally further comprising Haplotype M Haplotypes C D G H and L optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M and or Haplotypes B C D E G I and L optionally further comprising Haplotype M.

The presently disclosed subject matter also provides in some embodiments hybrid or inbred plants that have been modified to include a transgene. In some embodiments the transgene encodes a gene product that provides resistance to a herbicide selected from among glyphosate Sulfonylurea imidazolinione dicamba glufisinate phenoxy proprionic acid cycloshexome traizine benzonitrile and broxynil.

The presently disclosed subject matter also provides in some embodiments methods for identifying plants comprising at least one allele associated with water optimization as disclosed herein. In some embodiments the methods comprise a genotyping at least one plant with at least one nucleic acid marker selected from among SEQ ID NOs 1 60 and 400 and b selecting at least one plant comprising an allele of at least one of the at least one nucleic acid markers that is associated with water optimization.

The presently disclosed subject matter also provides in some embodiments plants produced by introgressing an allele of interest of a locus associated with a water optimization trait into germplasm. In some embodiments the introgressing comprises a selecting a plant that comprises an allele of interest of a locus associated with a water optimization trait wherein the locus associated with a water optimization trait comprises a nucleotide sequence that is at least 90 identical to any of SEQ ID NOs 1 117 400 and 401 and b introgressing the allele of interest into germplasm that lacks the allele.

The presently disclosed subject matter also provides in some embodiments methods for identifying and or selecting drought tolerant maize plants or germplasm. In some embodiments the presently disclosed methods comprise detecting in a maize plant or germplasm the presence of a marker associated with enhanced drought tolerance wherein the marker is selected from the group consisting of 

a G nucleotide at the position that corresponds to position 100 of SEQ ID NO 2 a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 116 of SEQ ID NO 23 an A nucleotide at the position that corresponds to position 391 of SEQ ID NO 33 an A nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 an A nucleotide at the position that corresponds to position 237 of SEQ ID NO 56 a T nucleotide at the position that corresponds to position 173 of SEQ ID NO 57 and a G nucleotide at the position that corresponds to position 267 of SEQ ID NO 60 

a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 309 of SEQ ID NO 19 a G nucleotide at the position that corresponds to position 562 of SEQ ID NO 25 a C nucleotide at the position that corresponds to position 1271 of SEQ ID NO 26 an A nucleotide at the position that corresponds to position 266 of SEQ ID NO 44 a C nucleotide at the position that corresponds to position 386 of SEQ ID NO 46 an A nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 and a G nucleotide at the position that corresponds to position 111 of SEQ ID NO 51 

a G nucleotide at the position that corresponds to position 100 an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 217 of SEQ ID NO 23 a C nucleotide at the position that corresponds to position 746 of SEQ ID NO 24 a C nucleotide at the position that corresponds to position 258 of SEQ ID NO 29 an A nucleotide at the position that corresponds to position 266 of SEQ ID NO 44 a G nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 a G nucleotide at the position that corresponds to position 193 of SEQ ID NO 55 and a C nucleotide at the position that corresponds to position 486 of SEQ ID NO 58 

a deletion at nucleotide at the position that corresponds to positions 264 271 of SEQ ID NO 2 an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 309 of SEQ ID NO 19 an A nucleotide at the position that corresponds to position 391 of SEQ ID NO 33 a G nucleotide at the position that corresponds to position 237 of SEQ ID NO 56 and a C nucleotide at the position that corresponds to position 486 of SEQ ID NO 58 

a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 a C nucleotide at the position that corresponds to position 254 of SEQ ID NO 27 an A nucleotide at the position that corresponds to position 391 of SEQ ID NO 33 a T nucleotide at the position that corresponds to position 475 of SEQ ID NO 45 a G nucleotide at the position that corresponds to position 193 of SEQ ID NO 55 a C nucleotide at the position that corresponds to position 516 of SEQ ID NO 56 a G nucleotide at the position that corresponds to position 729 of SEQ ID NO 59 and a G nucleotide at the position that corresponds to position 267 of SEQ ID NO 60 

an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 a C nucleotide at the position that corresponds to position 258 of SEQ ID NO 29 a G nucleotide at the position that corresponds to position 193 of SEQ ID NO 55 and a G nucleotide at the position that corresponds to position 237 of SEQ ID NO 56 

a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 an A nucleotide at the position that corresponds to position 237 of SEQ ID NO 56 and a T nucleotide at the position that corresponds to position 173 of SEQ ID NO 57 

a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 a C nucleotide at the position that corresponds to position 1271 of SEQ ID NO 26 an A nucleotide at the position that corresponds to position 266 of SEQ ID NO 44 a C nucleotide at the position that corresponds to position 386 of SEQ ID NO 46 an A nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 and a G nucleotide at the position that corresponds to position 111 of SEQ ID NO 51 

an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 a C nucleotide at the position that corresponds to position 258 of SEQ ID NO 29 a G nucleotide at the position that corresponds to position 87 of SEQ ID NO 47 a G nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 and a G nucleotide at the position that corresponds to position 193 of SEQ ID NO 55 

an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 309 of SEQ ID NO 19 and a G nucleotide at the position that corresponds to position 237 of SEQ ID NO 56 

a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 a T nucleotide at the position that corresponds to position 475 of SEQ ID NO 45 and a G nucleotide at the position that corresponds to position 193 of SEQ ID NO 55 

a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 and a C nucleotide at the position that corresponds to position 386 of SEQ ID NO 46 

an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 and a G nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 

and combinations thereof thereby identifying and or selecting a drought tolerant maize plant or germplasm.

Thus it is an object of the presently disclosed subject matter to provide methods for conveying one or more water optimization traits into maize germplasm.

An object of the presently disclosed subject matter having been stated hereinabove and which is achieved in whole or in part by the presently disclosed subject matter other objects will become evident as the description proceeds as best described herein below.

The instant disclosure includes a plurality of nucleotide and or amino acid sequences. Throughout the disclosure and the accompanying sequence listing the WIPO Standard ST.25 1998 hereinafter the ST.25 Standard is employed to identify nucleotides. This nucleotide identification standard is summarized below 

In certain instances the accompanying Sequence Listing includes one or more specifically identified definitions for certain nucleotide positions as set forth in lines through of the corresponding Sequence Listing entries. For example whereas under the ST.25 Standard the nucleotide n generally substitutes for any of a c g or t in SEQ ID NO 2 it is noted that the sequence nnnnnnnn at nucleotide positions 264 271 is defined to represent either the presence or the absence of the nucleotide sequence CACCAAGG . Similarly in SEQ ID NO 5 it is noted that the sequence nnnn at nucleotide positions 818 821 is defined to represent either the presence or the absence of the nucleotide sequence CGCG . As such whereas the ST.25 Standard is to be followed throughout the instant specification Statement s and Sequence Listing certain sequences disclosed herein represent specific departures from the ST.25 Standard and are noted accordingly.

Additionally whether specifically noted or not for each recitation of n in the Sequence Listing it is understood that any individual n including some or all n s in a sequence of consecutive n s can represent a c g t u unknown or other or can be absent. Thus unless specifically defined to the contrary in the Sequence Listing an n can in some embodiments represent no nucleotide. For example SEQ ID NO 7 includes a string of 52 n s between nucleotides 4549 and 4600 inclusive. It is understood that one or more of these n s can be absent including but not limited to all 52 or any subset thereof.

SEQ ID NO 1 is a nucleotide sequence that is associated with the water optimization locus ZmIga4 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 2 is a nucleotide sequence that is associated with the a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 3 is a nucleotide sequence that is associated with the water optimization locus ZmDr1 subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 4 is a nucleotide sequence that is associated with the water optimization locus ZmDrA encoding a voltage dependent anion channel subsequences of which can be amplified from chromosome 7 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 5 is a nucleotide sequence that is associated with the water optimization locus ZmDr2 subsequences of which can be amplified from chromosome 2 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 6 is a nucleotide sequence that is associated with the water optimization locus ZmDr3 subsequences of which can be amplified from chromosome 2 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 7 is a nucleotide sequence that is associated with the water optimization locus ZmDr4 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 8 is a nucleotide sequence that is associated with a water optimization locus ZmMa3 subsequences of which can be amplified from chromosome 2 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 9 is a nucleotide sequence that is associated with the water optimization locus ZmDr6 subsequences of which can be amplified from chromosome 4 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 10 is a nucleotide sequence that is associated with the water optimization locus ZmBglcn subsequences of which can be amplified from chromosome 3 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 11 is a nucleotide sequence that is associated with the water optimization locus ZmLOC100276591 subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NOs 12 and 13 are nucleotide sequences that are associated with the water optimization locus ZmDr7 subsequences of which can be amplified from chromosome 1 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 14 is a nucleotide sequence that is associated with the water optimization locus ZmDr8 subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 15 is a nucleotide sequence that is associated with the water optimization locus ZmHsp70 subsequences of which can be amplified from chromosome 1 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 16 is a nucleotide sequence that is associated with the water optimization locus ZmDr9 subsequences of which can be amplified from chromosome 4 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 17 is a nucleotide sequence that is associated with the water optimization locus ZmDrB subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 18 is a nucleotide sequence that is associated with the water optimization locus ZmAdh1 1s subsequences of which can be amplified from chromosome 1 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 19 is a nucleotide sequence that is associated with the water optimization locus ZmDr10 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 20 is a nucleotide sequence that is associated with the water optimization locus ZmDrC subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 21 is a nucleotide sequence that is associated with the water optimization locus ZmDr5 subsequences of which can be amplified from chromosome 5 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 22 is a nucleotide sequence that is associated with the water optimization locus ZmDrD encoding a subtilisin chymotrypsin inhibitor 2 subsequences of which can be amplified from chromosome 5 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 23 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 24 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 25 is a nucleotide sequence that is associated with the water optimization locus ZmDr12 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 26 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 27 is a nucleotide sequence that is associated with the water optimization locus ZmDrE encoding a legumin like protein cl2 1 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 28 is a nucleotide sequence that is associated with the water optimization locus ZmDrF encoding a putative cellulose synthase subsequences of which can be amplified from chromosome 9 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 29 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 30 is a nucleotide sequence that is associated with the water optimization locus ZmDhn2 subsequences of which can be amplified from chromosome 4 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 31 is a nucleotide sequence that is associated with the water optimization locus ZmDr16 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 32 is a nucleotide sequence that is associated with the water optimization locus ZmDr17 subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 33 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 34 is a nucleotide sequences that is associated with the water optimization locus ZmZCN6 subsequences of which can be amplified from chromosome 4 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 35 is a nucleotide sequence that is associated with the water optimization locus ZmDrG subsequences of which can be amplified from chromosome 5 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 36 is a nucleotide sequence that is associated with the water optimization locus ZmDhn1 subsequences of which can be amplified from chromosome 6 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 37 is a nucleotide sequence that is associated with the water optimization locus ZmDrH subsequences of which can be amplified from chromosome 5 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 38 is a nucleotide sequence that is associated with the water optimization locus ZmDrI subsequences of which can be amplified from chromosome 3 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 39 is a nucleotide sequence that is associated with the water optimization locus ZmDrJ encoding a mcm5 DNA replication factor subsequences of which can be amplified from chromosome 5 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 40 is a nucleotide sequence that is associated with the water optimization locus ZmH2B1 subsequences of which can be amplified from chromosome 4 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 41 is a nucleotide sequence that is associated with the water optimization locus ZmDr3 subsequences of which can be amplified from chromosome 2 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 42 is a nucleotide sequence that is associated with a the water optimization locus ZmDrK encoding an inorganic phosphatase subsequences of which can be amplified from the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 43 is a nucleotide sequence that is associated with water optimization locus ZmCat1 subsequences of which can be amplified from chromosome 5 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NOs 44 and 45 are nucleotide sequences that are associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 46 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 47 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 48 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 49 is a nucleotide sequence that is associated with the water optimization locus ZmRIC1 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NOs 50 and 51 are nucleotide sequences that are associated with the water optimization locus ZmPK4 subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 52 is a nucleotide sequence that is associated with the water optimization locus Zpu1 subsequences of which can be amplified from chromosome 2 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 53 is a nucleotide sequence that is associated with the water optimization locus ZmDrL subsequences of which can be amplified from chromosome 9 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 54 is a nucleotide sequence that is associated with the water optimization locus ZmDrM encoding a hexose transporter subsequences of which can be amplified from chromosome 7 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 55 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NOs 56 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 57 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 58 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 59 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 60 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 8 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NO 400 is a nucleotide sequence that is associated with a water optimization locus subsequences of which can be amplified from chromosome 4 of the genome using the polymerase chain reaction with amplification primers as set forth in Table 4 below.

SEQ ID NOs 61 117 and 401 are nucleotide sequences present in the GENBANK database available through the World Wide Web at the website for the National Center for Biotechnology Information NCBI of the United States National Institutes of Health that correspond to i.e. come from the same chromosomal loci in as SEQ ID NOs 1 60 and 400. The relationships among SEQ ID NOs 1 60 and 400 and 61 117 and 401 are set forth in Table 3.

SEQ ID NOs 61 117 and 401 have been added to the GENBANK database by the Genome Sequencing Center Washington University School of Medicine St. Louis Mo. United States of America. As set forth in the annotations to these database entries the sequences were part of an effort by The Maize Sequencing Consortium to sequence the genome of . Currently the sequencing effort has not been completed and various portions of the genome remain unsequenced and or the sequences have not been ordered or potentially have been misordered in the GENBANK database.

Table 4 lists SEQ ID NOs. for oligonucleotides that can be employed to amplify nucleic acids derived from the loci that correspond to SEQ ID NOs 1 117 400 and 401 and exemplary amplicons produced thereby. Table 4 also lists the nucleotide position in each locus sequence of SEQ ID NOs 1 60 of a polymorphism in some embodiments an SNP that is associated with a water optimization trait as well as the corresponding nucleotide position for the polymorphism in each amplicon.

As can be seen in Tables 3 and 4 certain of the sequences of SEQ ID NOs 1 399 are related to each other. By way of example SEQ ID NO 1 is a nucleotide sequence from that has been mapped to the ZmIga4 locus on chromosome 8. A subsequence of SEQ ID NO 1 can be amplified in an amplification reaction e.g. a PCR reaction using oligonucleotides having the sequences set forth in SEQ ID NOs 118 and 119 to yield an amplicon. At position 270 of SEQ ID NO 1 there is an SNP and the specific nucleotide that is present in any nucleic acid sample at this position can be determined using oligonucleotides that have the sequences set forth in SEQ ID NOs 232 and 233.

Additionally GENBANK Accession No. AC214546.3 includes a subsequence i.e. nucleotides 79 631 80 177 SEQ ID NO 61 that itself is highly similar to SEQ ID NO 1 i.e. 538 552 nucleotides identical 98 and thus is present at the same locus from which SEQ ID NO 1 is derived. The differences between the two sequences which can be identified using a BLAST algorithm a ClustalX algorithm or any other appropriate method of analysis can be attributable to normal variation within populations. A subsequence of SEQ ID NO 61 can also be amplified in an amplification reaction e.g. a PCR reaction using oligonucleotides having the sequences set forth in SEQ ID NOs 118 and 119 to yield an amplicon. Oligonucleotides with the sequences set forth in SEQ ID NOs 232 and 233 can also be used to assay the base that is present at the position that corresponds to position 270 of SEQ ID NO 1.

For SEQ ID NOs 2 399 similar interrelationships exist with SEQ ID NOs as are described hereinabove and would be identifiable by one of ordinary skill in the art using routine sequence analysis techniques. It is noted that with respect to certain of SEQ ID NOs 1 60 and 400 the complete nucleotide sequence of a genomic clone that includes the full length sequence that corresponds to these sequences might not been yet been added to the GENBANK database by The Maize Sequencing Consortium. Nonetheless with the sequence information disclosed herein one of ordinary skill in the art can unambigously identify the loci that correspond to SEQ ID NOs 1 117.

The presently disclosed subject matter provides compositions and methods for identifying selecting and or producing maize plants with enhanced drought tolerance also referred to herein as water optimization as well as maize plants identified selected and or produced by a method of this invention. In addition the presently disclosed subject matter provides maize plants and or germplasms having within their genomes one or more markers associated with enhanced drought tolerance.

To assess the value of alleles and or haplotypes under drought stress diverse germplasm was screened in controlled field experiments comprising a full irrigation control treatment and a limited irrigation treatment. A goal of the full irrigation treatment was to ensure that water did not limit the productivity of the crop. In contrast a goal of the limited irrigation treatment was to ensure that water became the major limiting constraint to grain yield. Main effects e.g. treatment and genotype and interactions e.g. genotype treatment could be determined when the two treatments were applied adjacent to one another in the field. Moreover drought related phenotypes could be quantified for each genotype in the panel thereby allowing for marker trait associations to be conducted.

In practice the method for the limited irrigation treatment can vary widely depending upon the germplasm being screened the soil type climatic conditions at the site pre season water supply and in season water supply to name just a few. Initially a site is identified where in season precipitation is low to minimize the chance of unintended water application and is suitable for cropping. In addition determining the timing of the stress can be important such that a target is defined to ensure that year to year or location to location screening consistency is in place. An understanding of the treatment intensity or in some cases the yield loss desired from the limited irrigation treatment can also be considered. Selection of a treatment intensity that is too light can fail to reveal genotypic variation. Selection of a treatment intensity that is too heavy can create large experimental error. Once the timing of stress is identified and treatment intensity is described irrigation can be managed in a manner that is consistent with these targets.

While the following terms are believed to be well understood by one of ordinary skill in the art the following definitions are set forth to facilitate explanation of the presently disclosed subject matter.

All technical and scientific terms used herein unless otherwise defined below are intended to have the same meaning as commonly understood by one of ordinary skill in the art. References to techniques employed herein are intended to refer to the techniques as commonly understood in the art including variations on those techniques or substitutions of equivalent techniques that would be apparent to one of skill in the art. While the following terms are believed to be well understood by one of ordinary skill in the art the following definitions are set forth to facilitate explanation of the presently disclosed subject matter.

Following long standing patent law convention the terms a an and the refer to one or more when used in this application including the claims. For example the phrase a marker refers to one or more markers. Similarly the phrase at least one when employed herein to refer to an entity refers to for example 1 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 75 100 or more of that entity including but not limited to whole number values between 1 and 100 and greater than 100.

Unless otherwise indicated all numbers expressing quantities of ingredients reaction conditions and so forth used in the specification and claims are to be understood as being modified in all instances by the term about . The term about as used herein when referring to a measurable value such as an amount of mass weight time volume concentration or percentage is meant to encompass variations of in some embodiments 20 in some embodiments 10 in some embodiments 5 in some embodiments 1 in some embodiments 0.5 and in some embodiments 0.1 from the specified amount as such variations are appropriate to perform the disclosed methods. Accordingly unless indicated to the contrary the numerical parameters set forth in this specification and attached claims are approximations that can vary depending upon the desired properties sought to be obtained by the presently disclosed subject matter.

As used herein the term and or refers to and encompasses any and all possible combinations of one or more of the associated listed items as well as the lack of combinations when interpreted in the alternative or .

As used herein the term allele refers to a variant or an alternative sequence form at a genetic locus. In diploids a single allele is inherited by a progeny individual separately from each parent at each locus. The two alleles of a given locus present in a diploid organism occupy corresponding places on a pair of homologous chromosomes although one of ordinary skill in the art understands that the alleles in any particular individual do not necessarily represent all of the alleles that are present in the species.

As used herein the term anthesis silk interval ASI refers to the difference between when a plant starts shedding pollen anthesis and when it begins producing silk female . Data are collected on a per plot basis. In some embodiments this interval is expressed in days.

As used herein the phrase associated with refers to a recognizable and or assayable relationship between two entities. For example the phrase associated with a water optimization trait refers to a trait locus gene allele marker phenotype etc. or the expression thereof the presence or absence of which can influence an extent degree and or rate at which a plant or a part of interest thereof that has the water optimization trait grows. As such a marker is associated with a trait when it is linked to it and when the presence of the marker is an indicator of whether and or to what extent the desired trait or trait form will occur in a plant germplasm comprising the marker. Similarly a marker is associated with an allele when it is linked to it and when the presence of the marker is an indicator of whether the allele is present in a plant germplasm comprising the marker. For example a marker associated with enhanced drought tolerance refers to a marker whose presence or absence can be used to predict whether and or to what extent a plant will display a drought tolerant phenotype.

As used herein the terms backcross and backcrossing refer to the process whereby a progeny plant is repeatedly crossed back to one of its parents. In a backcrossing scheme the donor parent refers to the parental plant with the desired gene or locus to be introgressed. The recipient parent used one or more times or recurrent parent used two or more times refers to the parental plant into which the gene or locus is being introgressed. For example see Ragot M. et al. TUMMLC Vol. 72 pp. 45 56 1995 and Openshaw et al. PS AMMD pp. 41 43 1994 . The initial cross gives rise to the F1 generation. The term BC1 refers to the second use of the recurrent parent BC2 refers to the third use of the recurrent parent and so on. In some embodiments a backcross is performed repeatedly with a progeny individual of each successive backcross generation being itself backcrossed to the same parental genotype.

A centimorgan cM is a unit of measure of recombination frequency. One cM is equal to a 1 chance that a marker at one genetic locus will be separated from a marker at a second locus due to crossing over in a single generation.

As used herein the term chromosome is used in its art recognized meaning of the self replicating genetic structure in the cellular nucleus containing the cellular DNA and bearing in its nucleotide sequence the linear array of genes. The chromosome numbers disclosed herein refer to those as set forth in Perin et al. 2002 which relates to a reference nomenclature system adopted by L institut National da la Recherch Agronomique INRA Paris France .

As used herein the phrase consensus sequence refers to a sequence of DNA built to identify nucleotide differences e.g. SNP and Indel polymorphisms in alleles at a locus. A consensus sequence can be either strand of DNA at the locus and states the nucleotide s at one or more positions e.g. at one or more SNPs and or at one or more Indels in the locus. In some embodiments a consensus sequence is used to design oligonucleotides and probes for detecting polymorphisms in the locus.

The term comprising which is synonymous with including containing or characterized by is inclusive or open ended and does not exclude additional unrecited elements and or method steps. Comprising is a term of art that means that the named elements and or steps are present but that other elements and or steps can be added and still fall within the scope of the relevant subject matter.

As used herein the phrase consisting of excludes any element step or ingredient not specifically recited. For example when the phrase consists of appears in a clause of the body of a claim rather than immediately following the preamble it limits only the element set forth in that clause other elements are not excluded from the claim as a whole.

As used herein the phrase consisting essentially of limits the scope of the related disclosure or claim to the specified materials and or steps plus those that do not materially affect the basic and novel characteristic s of the disclosed and or claimed subject matter. For example the presently disclosed subject matter relates in some embodiments to introgressing favorable alleles and or haplotypes into maize plants. One locus that comprises certain favorable alleles and or haplotypes is represented by SEQ ID NO 7 which includes nine 9 different polymorphisms as set forth herein with nine different favorable alelles. For any given introgression effort with respect to the genetic locus corresponding to SEQ ID NO 7 the method can consist essentially of introgressing a particular favorable allele selected from among these nine polymorphic locations which means that the recited favorable allele is the only favorable allele introgressed into a progeny genome. It is noted however that additional polymorphic loci will also be introgressed into the genome although the effects thereof might be unknown or not of interest.

With respect to the terms comprising consisting essentially of and consisting of where one of these three terms is used herein the presently disclosed and claimed subject matter can include the use of either of the other two terms. For example the presently disclosed subject matter relates in some embodiments to oligonucleotide primers comprise any of SEQ ID NOs 118 399 and 402 413. It is understood that the presently disclosed subject matter thus also encompasses oligonucleotide primers that in some embodiments consist essentially of any of SEQ ID NOs 118 399 and 402 113 as well as oligonucleotide primers that in some embodiments consist of any of SEQ ID NOs 118 399 and 402 113. Similarly it is also understood that in some embodiments the methods of the presently disclosed subject matter comprise the steps that are disclosed herein in some embodiments the methods of the presently disclosed subject matter consist essentially of the steps that are disclosed and in some embodiments the methods of the presently disclosed subject matter consist of the steps that are disclosed herein.

As used herein the terms cross or crossed refer to the fusion of gametes via pollination to produce progeny e.g. cells seeds or plants . The term encompasses both sexual crosses the pollination of one plant by another and selfing self pollination e.g. when the pollen and ovule are from the same plant . The term crossing refers to the act of fusing gametes via pollination to produce progeny.

As used herein the terms cultivar and variety refer to a group of similar plants that by structural or genetic features and or performance can be distinguished from other varieties within the same species.

As used herein the terms desired allele and allele of interest are used interchangeably to refer to an allele associated with a desired trait. In some embodiments a desired allele and or allele of interest can be associated with either an increase or a decrease of or in a given trait depending on the nature of the desired phenotype. In some embodiments a desired allele and or allele of interest can be associated with a change in morphology color etc.

As used herein the terms drought tolerance and drought tolerant refer to a plant s ability to endure and or thrive under drought stress conditions. When used in reference to germplasm the terms refer to the ability of a plant that arises from that germplasm to endure and or thrive under drought conditions. In general a plant or germplasm is labeled as drought tolerant if it displays enhanced drought tolerance. 

As used herein the term enhanced drought tolerance refers to an improvement enhancement or increase in one or more water optimization phenotypes as compared to one or more control plants e.g. one or both of the parents or a plant lacking a marker associated with enhanced drought tolerance . Exemplary water optimization phenotypes include but are not limited to grain yield at standard moisture percentage YGSMN grain moisture at harvest GMSTP grain weight per plot GWTPN percent yield recovery PYREC yield reduction YRED anthesis silk interval ASI and percent barren PB . Thus a plant that demonstrates higher YGSMN than one or both of its parents when each is grown under drought stress conditions displays enhanced drought tolerance and can be labeled as drought tolerant. 

As used herein the terms elite and elite line refer to any line that is substantially homozygous and has resulted from breeding and selection for desirable agronomic performance.

As used herein the term gene refers to a hereditary unit including a sequence of DNA that occupies a specific location on a chromosome and that contains the genetic instruction for a particular characteristic or trait in an organism.

A genetic map is a description of genetic linkage relationships among loci on one or more chromosomes within a given species generally depicted in a diagrammatic or tabular form. For each genetic map distances between loci are measured by the recombination frequencies between them. Recombinations between loci can be detected using a variety of markers. A genetic map is a product of the mapping population types of markers used and the polymorphic potential of each marker between different populations. The order and genetic distances between loci can differ from one genetic map to another.

As used herein the phrase genetic marker refers to a nucleic acid sequence e.g. a polymorphic nucleic acid sequence that has been identified as associated with a locus or allele of interest and that is indicative of the presence or absence of the locus or allele of interest in a cell or organism. Examples of genetic markers include but are not limited to genes DNA or RNA derived sequences promoters any untranslated regions of a gene microRNAs siRNAs QTLs transgenes mRNAs ds RNAs transcriptional profiles and methylation patterns.

As used herein the term genotype refers to the genetic constitution of an individual or group of individuals at one or more genetic loci as contrasted with the observable and or detectable and or manifested trait the phenotype . Genotype is defined by the allele s and or haplotype s of one or more known loci that the individual has inherited from its parents. The term genotype can be used to refer to an individual s genetic constitution at a single locus at multiple loci or more generally the term genotype can be used to refer to an individual s genetic make up for all the genes in its genome. Genotypes can be indirectly characterized e.g. using markers and or directly characterized by nucleic acid sequencing.

As used herein the term germplasm refers to genetic material of or from an individual e.g. a plant a group of individuals e.g. a plant line variety or family or a clone derived from a line variety species or culture. The germplasm can be part of an organism or cell or can be separate from the organism or cell. In general germplasm provides genetic material with a specific molecular makeup that provides a physical foundation for some or all of the hereditary qualities of an organism or cell culture. As used herein germplasm includes cells seed or tissues from which new plants can be grown as well as plant parts such as leafs stems pollen or cells that can be cultured into a whole plant.

A haplotype is the genotype of an individual at a plurality of genetic loci i.e. a combination of alleles. Typically the genetic loci that define a haplotype are physically and genetically linked i.e. on the same chromosome segment. The term haplotype can refer to polymorphisms at a particular locus such as a single marker locus or polymorphisms at multiple loci along a chromosomal segment.

A heterotic group comprises a set of genotypes that perform well when crossed with genotypes from a different heterotic group. Hallauer et al. CCIp. 463 564 1998 . Inbred lines are classified into heterotic groups and are further subdivided into families within a heterotic group based on several criteria such as pedigree molecular marker based associations and performance in hybrid combinations. Smith et al. 80 833 1990 .

As used herein the term heterozygous refers to a genetic status wherein different alleles reside at corresponding loci on homologous chromosomes. As used herein the term homozygous refers to a genetic status wherein identical alleles reside at corresponding loci on homologous chromosomes. It is noted that both of these terms can refer to single nucleotide positions multiple nucleotide positions whether contiguous or not or entire loci on homologous chromosomes.

As used herein the term hybrid refers to a seed and or plant produced when at least two genetically dissimilar parents are crossed.

As used herein the term hybrid when used in the context of nucleic acids refers to a double stranded nucleic acid molecule or duplex formed by hydrogen bonding between complementary nucleotide bases. The terms hybridize and anneal refer to the process by which single strands of nucleic acid sequences form double helical segments through hydrogen bonding between complementary bases.

As used herein the phrase ILLUMINA GOLDENGATE Assay refers to a high throughput genotyping assay sold by Illumina Inc. of San Diego Calif. United States of America that can generate SNP specific PCR products. This assay is described in detail at the website of Illumina Inc. and in Fan et al. 2006.

As used herein the phrase immediately adjacent when used to describe a nucleic acid molecule that hybridizes to DNA containing a polymorphism refers to a nucleic acid that hybridizes to a DNA sequence that directly abuts the polymorphic nucleotide base position. For example a nucleic acid molecule that can be used in a single base extension assay is immediately adjacent to the polymorphism.

As used herein the term improved and grammatical variants thereof refers to a plant or a part progeny or tissue culture thereof that as a consequence of having or lacking a particular water optimization associated allele such as but not limited to those water optimization associated alleles disclosed herein is characterized by a higher or lower content of a water optimization associated trait depending on whether the higher or lower content is desired for a particular purpose.

As used herein the term inbred refers to a substantially homozygous plant or variety. The term can refer to a plant or variety that is substantially homozygous throughout the entire genome or that is substantially homozygous with respect to a portion of the genome that is of particular interest.

As used herein the term INDEL also spelled indel refers to an insertion or deletion in a pair of nucleotide sequences wherein a first sequence can be referred to as having an insertion relative to a second sequence or the second sequence can be referred to as having a deletion relative to the first sequence.

As used herein the term informative fragment refers to a nucleotide sequence comprising a fragment of a larger nucleotide sequence wherein the fragment allows for the identification of one or more alleles within the larger nucleotide sequence. For example an informative fragment of the nucleotide sequence of SEQ ID NO 1 comprises a fragment of the nucleotide sequence of SEQ ID NO 1 and allows for the identification of one or more alleles e.g. a G nucleotide at position 115 of SEQ ID NO 1 an A nucleotide at the position that corresponds to position 270 of SEQ ID NO 1 a T nucleotide at the position that corresponds to position 301 of SEQ ID NO 1 and or an A nucleotide at the position that corresponds to position 483 .

As used herein the phrase interrogation position refers to a physical position on a solid support that can be queried to obtain genotyping data for one or more predetermined genomic polymorphisms.

As used herein the terms introgression introgressing and introgressed refer to both the natural and artificial transmission of a desired allele or combination of desired alleles of a genetic locus or genetic loci from one genetic background to another. For example a desired allele at a specified locus can be transmitted to at least one progeny via a sexual cross between two parents of the same species where at least one of the parents has the desired allele in its genome. Alternatively for example transmission of an allele can occur by recombination between two donor genomes e.g. in a fused protoplast where at least one of the donor protoplasts has the desired allele in its genome. The desired allele can be a selected allele of a marker a QTL a transgene or the like. Offspring comprising the desired allele can be repeatedly backcrossed to a line having a desired genetic background and selected for the desired allele with the result being that the desired allele becomes fixed in the desired genetic background. For example a marker associated with enhanced drought tolerance can be introgressed from a donor into a recurrent parent that is not drought tolerant or only partially drought tolerant. The resulting offspring could then be repeatedly backcrossed and selected until the progeny possess the drought tolerance allele in the recurrent parent background.

As used herein the term isolated refers to a nucleotide sequence e.g. a genetic marker that is free of sequences that normally flank one or both sides of the nucleotide sequence in a plant genome. As such the phrase isolated and purified genetic marker associated with a water optimization trait in can be for example a recombinant DNA molecule provided one of the nucleic acid sequences normally found flanking that recombinant DNA molecule in a naturally occurring genome is removed or absent. Thus isolated nucleic acids include without limitation a recombinant DNA that exists as a separate molecule including but not limited to genomic DNA fragments produced by PCR or restriction endonuclease treatment with no flanking sequences present as well as a recombinant DNA that is incorporated into a vector an autonomously replicating plasmid or into the genomic DNA of a plant as part of a hybrid or fusion nucleic acid molecule.

As used herein the term linkage refers to a phenomenon wherein alleles on the same chromosome tend to be transmitted together more often than expected by chance if their transmission were independent. Thus two alleles on the same chromosome are said to be linked when they segregate from each other in the next generation in some embodiments less than 50 of the time in some embodiments less than 25 of the time in some embodiments less than 20 of the time in some embodiments less than 15 of the time in some embodiments less than 10 of the time in some embodiments less than 9 of the time in some embodiments less than 8 of the time in some embodiments less than 7 of the time in some embodiments less than 6 of the time in some embodiments less than 5 of the time in some embodiments less than 4 of the time in some embodiments less than 3 of the time in some embodiments less than 2 of the time and in some embodiments less than 1 of the time.

As such linkage typically implies and can also refer to physical proximity on a chromosome. Thus two loci are linked if they are within in some embodiments 20 centiMorgans cM in some embodiments 15 cM in some embodiments 12 cM in some embodiments 10 cM in some embodiments 9 cM in some embodiments 8 cM in some embodiments 7 cM in some embodiments 6 cM in some embodiments 5 cM in some embodiments 4 cM in some embodiments 3 cM in some embodiments 2 cM and in some embodiments 1 cM of each other. Similarly a yield locus of the presently disclosed subject matter is linked to a marker e.g. a genetic marker if it is in some embodiments within 20 15 12 10 9 8 7 6 5 4 3 2 or 1 cM of the marker.

Thus the term linkage refers to the degree with which one marker locus is associated with another marker locus or some other locus for example a drought tolerance locus . The linkage relationship between a molecular marker and a phenotype can be given as a probability or adjusted probability. Linkage can be expressed as a desired limit or range. For example in some embodiments any marker is linked genetically and physically to any other marker when the markers are separated by less than about 50 40 30 25 20 or 15 map units or cM .

In some embodiments of the presently disclosed subject matter it is advantageous to define a bracketed range of linkage for example from about 10 cM and about 20 cM from about 10 cM and about 30 cM or from about 10 cM and about 40 cM. The more closely a marker is linked to a second locus the better an indicator for the second locus that marker becomes. Thus closely linked loci such as a marker locus and a second locus display an inter locus recombination frequency of about 10 9 8 7 6 5 4 3 or 2 or less. In some embodiments the relevant loci display a recombination frequency of about 1 or less e.g. about 0.75 0.5 0.25 or less. Two loci that are localized to the same chromosome and at such a distance that recombination between the two loci occurs at a frequency of less than about 10 e.g. about 9 8 7 6 5 4 3 2 1 0.75 0.5 or 0.25 or less can also be said to be proximal to each other. Since one cM is the distance between two markers that show a 1 recombination frequency any marker is closely linked genetically and physically to any other marker that is in close proximity e.g. at or less than about 10 cM distant. Two closely linked markers on the same chromosome can be positioned about 9 8 7 6 5 4 3 2 1 0.75 0.5 or 0.25 cM or less from each other.

As used herein the term linkage disequilibrium refers to a non random segregation of genetic loci or traits or both . In either case linkage disequilibrium implies that the relevant loci are within sufficient physical proximity along a length of a chromosome so that they segregate together with greater than random i.e. non random frequency in the case of co segregating traits the loci that underlie the traits are in sufficient proximity to each other . Markers that show linkage disequilibrium are considered linked. Linked loci co segregate more than 50 of the time e.g. from about 51 to about 100 of the time. In other words two markers that co segregate have a recombination frequency of less than 50 and by definition are separated by less than 50 cM on the same chromosome . As used herein linkage can be between two markers or alternatively between a marker and a phenotype. A marker locus can be associated with linked to a trait e.g. drought tolerance. The degree of linkage of a molecular marker to a phenotypic trait is measured e.g. as a statistical probability of co segregation of that molecular marker with the phenotype.

Linkage disequilibrium is most commonly assessed using the measure r which is calculated using the formula described by Hill and Robertson 38 226 1968 . When r 1 complete linkage disequilibrium exists between the two marker loci meaning that the markers have not been separated by recombination and have the same allele frequency. Values for rabove indicate sufficiently strong linkage disequilibrium to be useful for mapping. Ardlie et al. 3 299 2002 . Hence alleles are in linkage disequilibrium when rvalues between pairwise marker loci are greater than or equal to about 0.33 0.4 0.5 0.6 0.7 0.8 0.9 or 1.0.

As used herein the term linkage equilibrium describes a situation where two markers independently segregate i.e. sort among progeny randomly. Markers that show linkage equilibrium are considered unlinked whether or not they lie on the same chromosome . As such the phrase linkage disequilibrium is defined as change from the expected relative frequency of gamete types in a population of many individuals in a single generation such that two or more loci act as genetically linked loci. If the frequency in a population of allele S is x s is x B is y and b is y then the expected frequency of genotype SB is xy that of Sb is xy that of sB is x y and that of sb is x y and any deviation from these frequencies is an example of disequilibrium.

As used herein the phrase linkage group refers to all of the genes or genetic traits that are located on the same chromosome. Within the linkage group those loci that are close enough together can exhibit linkage in genetic crosses. Since the probability of crossover increases with the physical distance between loci on a chromosome loci for which the locations are far removed from each other within a linkage group might not exhibit any detectable linkage in direct genetic tests. The term linkage group is mostly used to refer to genetic loci that exhibit linked behavior in genetic systems where chromosomal assignments have not yet been made. Thus in the present context the term linkage group is synonymous with the physical entity of a chromosome although one of ordinary skill in the art will understand that a linkage group can also be defined as corresponding to a region of i.e. less than the entirety of a given chromosome.

A locus is a position on a chromosome where a gene or marker or allele is located. In some embodiments a locus can encompass one or more nucleotides.

As used herein the term maize plant includes whole maize plants maize plant cells maize plant protoplast maize plant cell or maize tissue cultures from which maize plants can be regenerated maize plant calli and maize plant cells that are intact in maize plants or parts of maize plants such as maize seeds maize cobs maize flowers maize cotyledons maize leaves maize stems maize buds maize roots maize root tips and the like.

As used herein the terms marker genetic marker and molecular marker are used interchangeably to refer to an identifiable position on a chromosome the inheritance of which can be monitored and or a reagent that is used in methods for visualizing differences in nucleic acid sequences present at such identifiable positions on chromosomes. Thus in some embodiments a marker comprises a known or detectable nucleic acid sequence. Examples of markers include but are not limited to genetic markers protein composition peptide levels protein levels oil composition oil levels carbohydrate composition carbohydrate levels fatty acid composition fatty acid levels amino acid composition amino acid levels biopolymers starch composition starch levels fermentable starch fermentation yield fermentation efficiency e.g. captured as digestibility at 24 48 and or 72 hours energy yield secondary compounds metabolites morphological characteristics and agronomic characteristics. As such a marker can comprise a nucleotide sequence that has been associated with an allele or alleles of interest and that is indicative of the presence or absence of the allele or alleles of interest in a cell or organism and or to a reagent that is used to visualize differences in the nucleotide sequence at such an identifiable position or positions. A marker can be but is not limited to an allele a gene a haplotype a restriction fragment length polymorphism RFLP a simple sequence repeat SSR random amplified polymorphic DNA RAPD cleaved amplified polymorphic sequences CAPS Rafalski and Tingey 9 275 1993 an amplified fragment length polymorphism AFLP Vos et al. 23 4407 1995 a single nucleotide polymorphism SNP Brookes 234 177 1993 a sequence characterized amplified region SCAR Paran and Michelmore 85 985 1993 a sequence tagged site STS Onozaki et al. 138 255 2004 a single stranded conformation polymorphism SSCP Orita et al. 86 2766 1989 an inter simple sequence repeat ISSR Blair et al. 98 780 1999 an inter retrotransposon amplified polymorphism IRAP a retrotransposon microsatellite amplified polymorphism REMAP Kalendar et al. 98 704 1999 or an RNA cleavage product such as a Lynx tag . A marker can be present in genomic or expressed nucleic acids e.g. ESTs . The term marker can also refer to nucleic acids used as probes or primers e.g. primer pairs for use in amplifying hybridizing to and or detecting nucleic acid molecules according to methods well known in the art. A large number of maize molecular markers are known in the art and are published or available from various sources such as the Maize GDB internet resource and the Arizona Genomics Institute internet resource run by the University of Arizona.

In some embodiments a marker corresponds to an amplification product generated by amplifying a nucleic acid with one or more oligonucleotides for example by the polymerase chain reaction PCR . As used herein the phrase corresponds to an amplification product in the context of a marker refers to a marker that has a nucleotide sequence that is the same allowing for mutations introduced by the amplification reaction itself and or naturally occurring and or artificial alleleic differences as an amplification product that is generated by amplifying genomic DNA with a particular set of oligonucleotides. In some embodiments the amplifying is by PCR and the oligonucleotides are PCR primers that are designed to hybridize to opposite strands of the genomic DNA in order to amplify a genomic DNA sequence present between the sequences to which the PCR primers hybridize in the genomic DNA. The amplified fragment that results from one or more rounds of amplification using such an arrangement of primers is a double stranded nucleic acid one strand of which has a nucleotide sequence that comprises in 5 to 3 order the sequence of one of the primers the sequence of the genomic DNA located between the primers and the reverse complement of the second primer. Typically the forward primer is assigned to be the primer that has the same sequence as a subsequence of the arbitrarily assigned top strand of a double stranded nucleic acid to be amplified such that the top strand of the amplified fragment includes a nucleotide sequence that is in 5 to 3 direction equal to the sequence of the forward primer the sequence located between the forward and reverse primers of the top strand of the genomic fragment the reverse complement of the reverse primer. Accordingly a marker that corresponds to an amplified fragment is a marker that has the same sequence of one of the strands of the amplified fragment.

Markers corresponding to genetic polymorphisms between members of a population can be detected by methods well established in the art. These include e.g. nucleic acid sequencing hybridization methods amplification methods e.g. PCR based sequence specific amplification methods detection of restriction fragment length polymorphisms RFLP detection of isozyme markers detection of polynucleotide polymorphisms by allele specific hybridization ASH detection of amplified variable sequences of the plant genome detection of self sustained sequence replication detection of simple sequence repeats SSRs detection of single nucleotide polymorphisms SNPs and or detection of amplified fragment length polymorphisms AFLPs . Well established methods are also known for the detection of expressed sequence tags ESTs and SSR markers derived from EST sequences and randomly amplified polymorphic DNA RAPD .

A marker allele also described as an allele of a marker locus can refer to one of a plurality of polymorphic nucleotide sequences found at a marker locus in a population that is polymorphic for the marker locus.

As used herein the phrase marker assay refers to a method for detecting a polymorphism at a particular locus using a particular method such as but not limited to measurement of at least one phenotype such as seed color oil content or a visually detectable trait nucleic acid based assays including but not limited to restriction fragment length polymorphism RFLP single base extension electrophoresis sequence alignment allelic specific oligonucleotide hybridization ASO random amplified polymorphic DNA RAPD microarray based technologies TAQMAN Assays ILLUMINA GOLDENGATE Assay analysis nucleic acid sequencing technologies peptide and or polypeptide analyses or any other technique that can be employed to detect a polymorphism in an organism at a locus of interest.

 Marker assisted selection MAS is a process by which phenotypes are selected based on marker genotypes.

 Marker assisted counter selection is a process by which marker genotypes are used to identify plants that will not be selected allowing them to be removed from a breeding program or planting.

As used herein the terms marker locus and marker loci refer to a specific chromosome location or locations in the genome of an organism where a specific marker or markers can be found. A marker locus can be used to track the presence of a second linked locus e.g. a linked locus that encodes or contributes to expression of a phenotypic trait. For example a marker locus can be used to monitor segregation of alleles at a locus such as a QTL or single gene that are genetically or physically linked to the marker locus.

As used herein the terms marker probe and probe refer to a nucleotide sequence or nucleic acid molecule that can be used to detect the presence of one or more particular alleles within a marker locus e.g. a nucleic acid probe that is complementary to all of or a portion of the marker or marker locus through nucleic acid hybridization . Marker probes comprising about 8 10 15 20 30 40 50 60 70 80 90 100 or more contiguous nucleotides can be used for nucleic acid hybridization. Alternatively in some aspects a marker probe refers to a probe of any type that is able to distinguish i.e. genotype the particular allele that is present at a marker locus.

As used herein the term molecular marker can be used to refer to a genetic marker as defined above or an encoded product thereof e.g. a protein used as a point of reference when identifying a linked locus. A molecular marker can be derived from genomic nucleotide sequences or from expressed nucleotide sequences e.g. from a spliced RNA a cDNA etc. . The term also refers to nucleotide sequences complementary to or flanking the marker sequences such as nucleotide sequences used as probes and or primers capable of amplifying the marker sequence. Nucleotide sequences are complementary when they specifically hybridize in solution e.g. according to Watson Crick base pairing rules. Some of the markers described herein are also referred to as hybridization markers when located on an indel region. This is because the insertion region is by definition a polymorphism vis vis a plant without the insertion. Thus the marker need only indicate whether the indel region is present or absent. Any suitable marker detection technology can be used to identify such a hybridization marker e.g. SNP technology is used in the examples provided herein.

The presently disclosed subject matter provides in some embodiments markers for determining the presence of genetic polymorphisms in the maize loci disclosed herein. The loci that can be analyzed using the compositions and methods of the presently disclosed subject matter include but are not limited to the loci referred to herein as ZmAdh1 1s ZmBglcn ZmCat1 ZmDhn1 ZmDhn2 ZmDr1 ZmDr2 ZmDr3 ZmDr3 ZmDr4 ZmDr5 ZmDr6 ZmDr7 ZmDr8 ZmDr9 ZmDr10 ZmDr12 ZmDr16 ZmDr17 ZmH2B1 ZmHsp70 ZmIga4 ZmLOC100276591 ZmMa3 ZmPK4 ZmRIC1 ZmZCN6 Zpu1 ZmDrA ZmDrB ZmDrC ZmDrD ZmDrE ZmDrF ZmDrG ZmDrH ZmDrI ZmDrj ZmDrk ZmDrL and ZmDrM which terms thus refer to genomic regions and or genetic loci that are linked to water optimization associated traits present on chromosomes and as described in more detail hereinbelow. Exemplary genomic nucleotide sequences that are derived from these loci are summarized herein above.

The term ZmAdh1 1s refers to a locus on chromosome 1 that encodes a alcohol dehydrogenase 1 gene Dennis et al. 1984 . Exemplary gene products derived from the ZmAdh1 1s locus can be found in GENBANK Accession Nos. X04049 and P00333.

The term ZmBglcn refers to a locus on chromosome 3 that encodes a maize 1 3 glucanase polypeptide Wu et al. 1994 . Exemplary gene products derived from the ZmBglcn locus can be found in GENBANK Accession Nos. M95407 and AAA74320.

The term ZmCat1 refers to a locus on chromosome 5 that encodes a maize catalast 1 polypeptide Guan Scandalios 1993 . Exemplary gene products derived from the ZmCat1 locus can be found in GENBANK Accession Nos. X60135 and CAA42720.

The term ZmDhn1 refers to a locus on chromosome 6 that encodes a maize dehydrin 1 dhn1 polypeptide Close et al. 1989 . Exemplary gene products derived from the ZmDhn1 locus can be found in GENBANK Accession Nos. X15290 and CAA33364.

The term ZmDhn2 refers to a locus on chromosome 4 that encodes a maize dehydrin 2 dhn2 polypeptide. Exemplary gene products derived from the ZmDhn2 locus can be found in GENBANK Accession Nos. L35913 and AA33480.

The term ZmDr1 refers to a locus that in some embodiments corresponds to GENBANK Accession No. AY105200.

The term ZmDr2 refers to a locus that in some embodiments corresponds to GENBANK Accession No. AF043347.

The term ZmDr3 refers to a locus that in some embodiments corresponds to nucleotides 120 959 121 302 of GENBANK Accession No. AC206638.3 and in some embodiments corresponds to GENBANK Accession No. AF043347.

The term ZmDr4 refers to a locus that in some embodiments corresponds to GENBANK Accession No. AY103545.

The term ZmDr5 refers to a locus that in some embodiments corresponds to GENBANK Accession No. AY109606.

The term ZmDr6 refers to a locus that encodes a maize calmodulin binding protein. Exemplary gene products derived from the ZmDr6 locus can be found in GENBANK Accession Nos. L01497 NM 001158968 AAA33447 and NP 001152440.

The term ZmDr7 refers to a locus that encodes a maize sucrose transporter protein. Exemplary gene products derived from the ZmDr7 locus can be found in GENBANK Accession Nos. AB008464 NM 001111370 BAA83501. and NP 001104840.

The term ZmDr8 refers to a locus that in some embodiments corresponds to GENBANK Accession No. EU976286.

The term ZmDr9 refers to a locus that in some embodiments corresponds to nucleotides 75 481 76 499 of GENBANK Accession No. AC196196.4.

The term ZmDr10 refers to a locus that in some embodiments corresponds to GENBANK Accession No. DQ245017.

The term ZmDr12 refers to a locus that in some embodiments corresponds to GENBANK Accession No. AI770817.

The term ZmDr16 refers to a locus that in some embodiments corresponds to GENBANK Accession No. NM 001156978.

The term ZmDr17 refers to a locus that in some embodiments corresponds to nucleotides 60 463 60 838 of GENBANK Accession No. AC231410.4.

The term ZmDrA refers to a locus on chromosome 7 that encodes a voltage dependent anion channel protein. An exemplary gene product derived from the ZmDrA locus can be found in GENBANK Accession No. BT018647.

The term ZmDrB refers to a locus that encodes a xylan endohydrolase protein. An exemplary gene product derived from the ZmDrB locus can be found in GENBANK Accession No. AI691894.

The term ZmDrC refers to a locus that encodes a trehalose P synthase protein. An exemplary gene product derived from the ZmDrC locus can be found in GENBANK Accession No. AY110270.

The term ZmDrD refers to a locus on chromosome 5 that encodes a subtilisin chymotrypsin inhibitor 2 protein. An exemplary gene product derived from the ZmDrD locus can be found in GENBANK Accession No. BT066886.

The term ZmDrE refers to a locus on chromosome 8 that encodes a legumin like protein cl2 1 protein. Exemplary gene products derived from the ZmDrE locus can be found in GENBANK Accession Nos. NM 001111592 and NP 001105062.

The term ZmDrF refers to a locus on chromosome 9 that encodes a putative cellulose synthase protein. Exemplary gene products derived from the ZmDrF locus can be found in GENBANK Accession Nos. BT067558 and ACN34455.

The term ZmDrG refers to a locus on chromosome 5 that in some embodiments corresponds to GENBANK Accession No. AI691276.

The term ZmDrH refers to a locus on chromosome 5 that in some embodiments corresponds to GENBANK Accession No. AI665888.

The term ZmDrI refers to a locus on chromosome 3 that in some embodiments corresponds to GENBANK Accession No. AI737958.

The term ZmDrJ refers to a locus on chromosome 5 that encodes a mcm5 DNA replication factor protein. An exemplary gene products derived from the ZmDrJ locus can be found in GENBANK Accession No. AI666237.

The term ZmDrK refers to a locus that encodes an inorganic phosphatase protein that in some embodiments corresponds to nucleotides 28 345 29 279 of GENBANK Accession No. AC191554.3.

The term ZmDrL refers to a locus on chromosome 9 that encodes a late embryonic abundant like protein. An exemplary gene product derived from the ZmDrL locus can be found in GENBANK Accession No. AY105938.

The term ZmDrM refers to a locus on chromosome 7 that encodes a hexose transporter protein. Exemplary gene products derived from the ZmDrM locus can be found in GENBANK Accession Nos. NM 001154535 and NP 001148007.

The term ZmH2B1 refers to a locus on chromosome 4 that encodes a histone 2B1. An exemplary gene product derived from the ZmDr6 locus can be found in GENBANK Accession No. AI737900.

The term ZmHsp70 refers to a locus on chromosome 1 that encodes a maize heat shock cognate 70 kDa protein 2 protein. Exemplary gene products derived from the ZmDr7 locus can be found in GENBANK Accession Nos. EU971059 NM 001154726 and NP 001148198.

The term ZmIga4 refers to a locus on chromosome 8 that encodes a liguleless4 lg4 protein. Exemplary gene products derived from the ZmIga4 locus can be found in GENBANK Accession Nos. AF457121 NM 001111614 AAM27190 and NP 001105084.

The term ZmLOC100276591 refers to a locus that in some embodiments corresponds to GENBANK Accession Nos. NM 001150343 and NP 001143815.

The term ZmMa3 refers to a locus on chromosome 2 that encodes a maize topoisomerase like apoptosis protein ma 3. Exemplary gene products derived from the ZmMa3 locus can be found in GENBANK Accession Nos. NM 001154442 and NP 001147914.

The term ZmPK4 refers to a locus on chromosome 8 that encodes a maize protein kinase PK4 protein. Exemplary gene products derived from the ZmPK4 locus can be found in GENBANK Accession Nos. AF141378 NM 001111470 AAF22219 and NP 001104940.

The term ZmRIC1 refers to a locus on chromosome 8 that encodes a maize ras related protein RIC1. Exemplary gene products derived from the ZmRIC1 locus can be found in GENBANK Accession Nos. EU952511 NM 001137272 ACG24629 and NP 001130744.

The term ZmZCN6 refers to a locus on chromosome 4 that encodes a maize ZCN6 protein. Exemplary gene products derived from the ZmZCN6 locus can be found in GENBANK Accession Nos. EU241897 NM 001112774 ABX11008 and NP 001106245.

The term Zpu1 refers to a locus on chromosome 2 that encodes a pullulanase type starch debranching enzyme zpu1 protein. Exemplary gene products derived from the Zpu1 locus can be found in GENBANK Accession Nos. AF080567 NM 001111450 AAD11599 and NP 001104920.

As used herein the phrase native trait refers to any existing monogenic or oligogenic trait in a certain crop s germplasm. When identified through molecular marker s the information obtained can be used for the improvement of germplasm through marker assisted breeding of the water optimization associated traits disclosed herein.

A non naturally occurring variety of maize is any variety of maize that does not naturally exist in nature. A non naturally occurring variety of maize can be produced by any method known in the art including but not limited to transforming a maize plant or germplasm transfecting a maize plant or germplasm and crossing a naturally occurring variety of maize with a non naturally occurring variety of maize. In some embodiments a non naturally occurring variety of maize can comprise one of more heterologous nucleotide sequences. In some embodiments a non naturally occurring variety of maize can comprise one or more non naturally occurring copies of a naturally occurring nucleotide sequence i.e. extraneous copies of a gene that naturally occurs in maize .

The non Stiff Stalk heterotic group represents a major heterotic group in the northern U.S. and Canadian corn growing regions. It can also be referred to as the Lancaster or Lancaster Sure Crop heterotic group.

As used herein the terms nucleotide sequence polynucleotide nucleic acid sequence nucleic acid molecule and nucleic acid fragment refer to a polymer of RNA or DNA that is single or double stranded optionally containing synthetic non natural and or altered nucleotide bases. A nucleotide is a monomeric unit from which DNA or RNA polymers are constructed and consists of a purine or pyrimidine base a pentose and a phosphoric acid group. Nucleotides usually found in their 5 monophosphate form are referred to by their single letter designation as follows A for adenylate or deoxyadenylate for RNA or DNA respectively C for cytidylate or deoxycytidylate G for guanylate or deoxyguanylate U for uridylate T for deoxythymidylate R for purines A or G Y for pyrimidines C or T K for G or T H for A or C or T I for inosine and N for any nucleotide.

As used herein the term nucleotide sequence identity refers to the presence of identical nucleotides at corresponding positions of two polynucleotides. Polynucleotides have identical sequences if the sequence of nucleotides in the two polynucleotides is the same when aligned for maximum correspondence e.g. in a comparison window . Sequence comparison between two or more polynucleotides is generally performed by comparing portions of the two sequences over a comparison window to identify and compare local regions of sequence similarity. The comparison window is generally from about 20 to 200 contiguous nucleotides. The percentage of sequence identity for polynucleotides such as about 50 55 60 65 70 75 80 85 90 95 98 99 or 100 percent sequence identity can be determined by comparing two optimally aligned sequences over a comparison window wherein the portion of the polynucleotide sequence in the comparison window can include additions or deletions i.e. gaps as compared to the reference sequence for optimal alignment of the two sequences. The percentage is calculated by a determining the number of positions at which the identical nucleic acid base occurs in both sequences b dividing the number of matched positions by the total number of positions in the window of comparison and c multiplying the result by 100. Optimal alignment of sequences for comparison can also be conducted by computerized implementations of known algorithms or by visual inspection. Readily available sequence comparison and multiple sequence alignment algorithms are respectively the Basic Local Alignment Search Tool BLAST and ClustalW programs both available on the internet. Other suitable programs include but are not limited to GAP BestFit Plot Similarity and FASTA which are part of the Accelrys GCG Package available from Accelrys Inc. of San Diego Calif. United States of America. In some embodiments a percentage of sequence identity refers to sequence identity over the full length of one of the sequences being compared. In some embodiments a calculation to determine a percentage of sequence identity does not include in the calculation any nucleotide positions in which either of the compared nucleic acids includes an N i.e. where any nucleotide could be present at that position .

As used herein the term percent barren PB refers to the percentage of plants in a given area e.g. plot with no grain. It is typically expressed in terms of the percentage of plants per plot and can be calculated as 

As used herein the term percent yield recovery PYREC refers to the effect an allele and or combination of alleles has on the yield of a plant grown under drought stress conditions as compared to that of a plant that is genetically identical except insofar as it lacks the allele and or combination of alleles. PYREC is calculated as 

As used herein the phrase Grain Yield Well Watered refers to yield from an area that obtained enough irrigation to prevent plants from being water stressed during their growth cycle. In some embodiments this trait is expressed in bushels per acre.

As used herein the phrase Yield Reduction Hybrid refers to a calculated trait obtained from a hybrid yield trial grown under stress and non stress conditions. For a given hybrid it equals 

As used herein the phrase Yield Reduction Inbred refers to a calculated trait obtained from an inbred yield trial grown under stress and non stress conditions. For a given inbred it equals 

As used herein the phrase Anthesis Silk Interval ASI refers to the difference in some embodiments expressed in days between when a plant starts shedding pollen anthesis and it starts producing silk female . Data are collected on a per plot basis for anthesis and silking and the difference is calculated.

As used herein the phrase Percent Barren refers to a percentage of plants in a given area plot with no grain. It is typically expressed in terms of plants per plot and can be calculated as 

As used herein the terms phenotype phenotypic trait or trait refer to one or more traits of an organism. The phenotype can be observable to the naked eye or by any other means of evaluation known in the art e.g. microscopy biochemical analysis or an electromechanical assay. In some cases a phenotype is directly controlled by a single gene or genetic locus i.e. a single gene trait. In other cases a phenotype is the result of several genes. It is noted that as used herein the term water optimization phenotype takes into account environmental conditions that might affect water optimization such that the water optimization effect is real and reproducible.

As used herein the term plant can refer to a whole plant any part thereof or a cell or tissue culture derived from a plant. Thus the term plant can refer to any of whole plants plant components or organs e.g. leaves stems roots etc. plant tissues seeds and or plant cells.

A plant cell is a cell of a plant taken from a plant or derived through culture from a cell taken from a plant. Thus the term plant cell includes without limitation cells within seeds suspension cultures embryos meristematic regions callus tissue leaves shoots gametophytes sporophytes pollen and microspores. The phrase plant part refers to a part of a plant including single cells and cell tissues such as plant cells that are intact in plants cell clumps and tissue cultures from which plants can be regenerated. Examples of plant parts include but are not limited to single cells and tissues from pollen ovules leaves embryos roots root tips anthers flowers fruits stems shoots and seeds as well as scions rootstocks protoplasts calli and the like.

As used herein the term polymorphism refers to a variation in the nucleotide sequence at a locus where said variation is too common to be due merely to a spontaneous mutation. A polymorphism must have a frequency of at least about 1 in a population. A polymorphism can be a single nucleotide polymorphism SNP or an insertion deletion polymorphism also referred to herein as an indel. Additionally the variation can be in a transcriptional profile or a methylation pattern. The polymorphic site or sites of a nucleotide sequence can be determined by comparing the nucleotide sequences at one or more loci in two or more germplasm entries.

As used herein the term population refers to a genetically heterogeneous collection of plants sharing a common genetic derivation.

As used herein the term primer refers to an oligonucleotide which is capable of annealing to a nucleic acid target in some embodiments annealing specifically to a nucleic acid target allowing a DNA polymerase to attach thereby serving as a point of initiation of DNA synthesis when placed under conditions in which synthesis of a primer extension product is induced e.g. in the presence of nucleotides and an agent for polymerization such as DNA polymerase and at a suitable temperature and pH . In some embodiments a plurality of primers are employed to amplify nucleic acids e.g. using the polymerase chain reaction PCR .

As used herein the term probe refers to a nucleic acid e.g. a single stranded nucleic acid or a strand of a double stranded or higher order nucleic acid or a subsequence thereof that can form a hydrogen bonded duplex with a complementary sequence in a target nucleic acid sequence. Typically a probe is of sufficient length to form a stable and sequence specific duplex molecule with its complement and as such can be employed in some embodiments to detect a sequence of interest present in a plurality of nucleic acids.

As used herein the terms progeny and progeny plant refer to a plant generated from a vegetative or sexual reproduction from one or more parent plants. A progeny plant can be obtained by cloning or selfing a single parent plant or by crossing two parental plants. Thus the phrase progeny plant refers to any plant resulting as progeny from a vegetative or sexual reproduction from one or more parent plants or descendants thereof. For instance a progeny plant can be obtained by cloning or selfing of a parent plant or by crossing two parental plants and include selfings as well as the F1 or F2 or still further generations. An F1 is a first generation progeny produced from parents at least one of which is used for the first time as donor of a trait while progeny of second generation F2 or subsequent generations F3 F4 and the like are specimens produced from selfings intercrosses backcrosses or other crosses of F1 s F2s and the like. An F1 can thus be and in some embodiments is a hybrid resulting from a cross between two true breeding parents i.e. parents that are true breeding are each homozygous for a trait of interest or an allele thereof while an F2 can be and in some embodiments is a progeny resulting from self pollination of the F1 hybrids.

As used herein the phrase quantitative trait locus QTL quantitative trait loci QTLs refers to a genetic locus or loci that control to some degree a numerically representable trait that in some embodiments is continuously distributed. In some embodiments a QTL comprises a water optimization associated locus. As used herein the phrase water optimization associated locus is used herein to refer to a chromosomal region containing alleles e.g. in the form of genes or regulatory sequences associated with the expression of a water optimization associated trait. Thus a locus associated with a water optimization trait refers to one or more regions located on one or more chromosomes that includes at least one gene the expression of which influences water optimization and or at least one regulatory region that controls the expression of one or more genes involved in one or more water optimization traits. The loci can be defined by indicating their genetic location in the genome of a given plant using one or more molecular genomic markers. One or more markers in turn indicate a specific locus. Distances between loci are usually measured by the frequency of crossovers between loci on the same chromosome. The farther apart two loci are the more likely that a crossover will occur between them. Conversely if two loci are close together a crossover is less likely to occur between them. Typically one centiMorgan cM is equal to 1 recombination between loci. When a QTL can be indicated by multiple markers the genetic distance between the end point markers is indicative of the size of the QTL.

As used herein the phrase recombination refers to an exchange of DNA fragments between two DNA molecules or chromatids of paired chromosomes a crossover over in a region of similar or identical nucleotide sequences. A recombination event is herein understood to refer to a meiotic crossover.

As used herein the term reference sequence refers to a defined nucleotide sequence used as a basis for nucleotide sequence comparison. The reference sequence for a marker for example is obtained by genotyping a number of lines at the locus or loci of interest aligning the nucleotide sequences in a sequence alignment program and then obtaining the consensus sequence of the alignment. Hence a reference sequence identifies the polymorphisms in alleles at a locus. A reference sequence can not be a copy of an actual nucleic acid sequence from any particular organism however it is useful for designing primers and probes for actual polymorphisms in the locus or loci.

As used herein the term regenerate and grammatical variants thereof refers to the production of a plant from tissue culture.

As used herein the phrases selected allele desired allele and allele of interest are used interchangeably to refer to a nucleic acid sequence that includes a polymorphic allele associated with a desired trait. It is noted that a selected allele desired allele and or allele of interest can be associated with either an increase in a desired trait or a decrease in a desired trait depending on the nature of the phenotype sought to be generated in an introgressed plant.

As used herein the phrase single nucleotide polymorphism or SNP refers to a polymorphism that constitutes a single base pair difference between two nucleotide sequences. As used herein the term SNP also refers to differences between two nucleotide sequences that result from simple alterations of one sequence in view of the other that occurs at a single site in the sequence. For example the term SNP is intended to refer not just to sequences that differ in a single nucleotide as a result of a nucleic acid substitution in one versus the other but is also intended to refer to sequences that differ in 1 2 3 or more nucleotides as a result of a deletion of 1 2 3 or more nucleotides at a single site in one of the sequences versus the other. It would be understood that in the case of two sequences that differ from each other only by virtue of a deletion of 1 2 3 or more nucleotides at a single site in one of the sequences versus the other this same scenario can be considered an addition of 1 2 3 or more nucleotides at a single site in one of the sequences versus the other depending on which of the two sequences is considered the reference sequence. Single site insertions and or deletions are thus also considered to be encompassed by the term SNP .

The Stiff Stalk heterotic group represents a major heterotic group in the northern U.S. and Canadian corn growing regions. It can also be referred to as the Iowa Stiff Stalk Synthetic or BSSS heterotic group.

As used herein the phrase stringent hybridization conditions refers to conditions under which a polynucleotide hybridizes to its target subsequence typically in a complex mixture of nucleic acids but to essentially no other sequences. Stringent conditions are sequence dependent and can be different under different circumstances.

Longer sequences typically hybridize specifically at higher temperatures. An extensive guide to the hybridization of nucleic acids is found in Tijssen 1993. Generally stringent conditions are selected to be about 5 10 C. lower than the thermal melting point Tm for the specific sequence at a defined ionic strength pH. The Tm is the temperature under defined ionic strength pH and nucleic acid concentration at which 50 of the probes complementary to the target hybridize to the target sequence at equilibrium as the target sequences are present in excess at Tm 50 of the probes are occupied at equilibrium . Exemplary stringent conditions are those in which the salt concentration is less than about 1.0 M sodium ion typically about 0.01 to 1.0 M sodium ion concentration or other salts at pH 7.0 to 8.3 and the temperature is at least about 30 C. for short probes e.g. 10 to 50 nucleotides and at least about 60 C. for long probes e.g. greater than 50 nucleotides .

Stringent conditions can also be achieved with the addition of destabilizing agents such as formamide. Additional exemplary stringent hybridization conditions include 50 formamide 5 SSC and 1 SDS incubating at 42 C. or SSC 1 SDS incubating at 65 C. with one or more washes in 0.2 SSC and 0.1 SDS at 65 C. For PCR a temperature of about 36 C. is typical for low stringency amplification although annealing temperatures can vary between about 32 C. and 48 C. or higher depending on primer length. Additional guidelines for determining hybridization parameters are provided in numerous references see e.g. Ausubel et al. 1999 .

As used herein the phrase TAQMAN Assay refers to real time sequence detection using PCR based on the TAQMAN Assay sold by Applied Biosystems Inc. of Foster City Calif. United States of America. For an identified marker a TAQMAN Assay can be developed for application in a breeding program.

As used herein the term tester refers to a line used in a testcross with one or more other lines wherein the tester and the line s tested are genetically dissimilar. A tester can be an isogenic line to the crossed line.

As used herein the term trait refers to a phenotype of interest a gene that contributes to a phenotype of interest as well as a nucleic acid sequence associated with a gene that contributes to a phenotype of interest. For example a water optimization trait refers to a water optimization phenotype as well as a gene that contributes to a water optimization phenotype and a nucleic acid sequence e.g. an SNP or other marker that is associated with a water optimization phenotype.

As used herein the term transgene refers to a nucleic acid molecule introduced into an organism or its ancestors by some form of artificial transfer technique. The artificial transfer technique thus creates a transgenic organism or a transgenic cell . It is understood that the artificial transfer technique can occur in an ancestor organism or a cell therein and or that can develop into the ancestor organism and yet any progeny individual that has the artificially transferred nucleic acid molecule or a fragment thereof is still considered transgenic even if one or more natural and or assisted breedings result in the artificially transferred nucleic acid molecule being present in the progeny individual.

An unfavorable allele of a marker is a marker allele that segregates with the unfavorable plant phenotype therefore providing the benefit of identifying plants that can be removed from a breeding program or planting.

As used herein the term water optimization refers to any measure of a plant its parts or its structure that can be measured and or quantitated in order to assess an extent of or a rate of plant growth and development under conditions of sufficient water availability as compared to conditions of suboptimal water availability e.g. drought . As such a water optimization trait is any trait that can be shown to influence yield in a plant under different sets of growth conditions related to water availability.

Similarly water optimization can be considered a phenotype which as used herein refers to a detectable observable and or measurable characteristic of a cell or organism. In some embodiments a phenotype is based at least in part on the genetic make up of the cell or the organism referred to herein as the cell or the organism s genotype . Exemplary water optimization phenotypes are grain yield at standard moisture percentage YGSMN grain moisture at harvest GMSTP grain weight per plot GWTPN and percent yield recovery PYREC . It is noted that as used herein the term phenotype takes into account how the environment e.g. environmental conditions might affect water optimization such that the water optimization effect is real and reproducible. As used herein the term yield reduction YD refers to the degree to which yield is reduced in plants grown under stress conditions. YD is calculated as 

Molecular markers are used for the visualization of differences in nucleic acid sequences. This visualization can be due to DNA DNA hybridization techniques after digestion with a restriction enzyme e.g. an RFLP and or due to techniques using the polymerase chain reaction e.g. STS SSR microsatellites AFLP and the like . In some embodiments all differences between two parental genotypes segregate in a mapping population based on the cross of these parental genotypes. The segregation of the different markers can be compared and recombination frequencies can be calculated. Methods for mapping markers in plants are disclosed in for example Glick Thompson 1993 Zietkiewicz et al. 1994. The recombination frequencies of molecular markers on different chromosomes are generally 50 . Between molecular markers located on the same chromosome the recombination frequency generally depends on the distance between the markers. A low recombination frequency typically corresponds to a small genetic distance between markers on a chromosome. Comparing all recombination frequencies results in the most logical order of the molecular markers on the chromosomes. This most logical order can be depicted in a linkage map Paterson 1996 . A group of adjacent or contiguous markers on the linkage map that is associated with increased water optimization can provide the position of an MTL associated with increased water optimization. Genetic loci correlating with particular phenotypes such as drought tolerance can be mapped in an organism s genome. By identifying a marker or cluster of markers that co segregate with a trait of interest the breeder is able to rapidly select a desired phenotype by selecting for the proper marker a process called marker assisted selection or MAS . Such markers can also be used by breeders to design genotypes in silico and to practice whole genome selection.

The presently disclosed subject matter provides in some embodiments markers associated with enhanced drought tolerance water optimization. Detection of these markers and or other linked markers can be used to identify select and or produce drought tolerant plants and or to eliminate plants that are not drought tolerant from breeding programs or planting.

The presently disclosed subject matter provides markers associated with improved water optimization traits. A marker of the presently disclosed subject matter can comprise a single allele or a combination of alleles at one or more genetic loci. In some embodiments the one or more alleles are characterized by one or more loci selected from but not limited to the loci represented by SEQ ID NOs 1 117 400 and 401 which are located in the genome as follows 

 i SEQ ID NO 1 is derived from the ZmIga4 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 1 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 118 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 119 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a single nucleic polymorphism at nucleotide positions 115 270 301 and 483 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 1 on chromosome 8 that confers an improved water optimization associated trait 

 ii SEQ ID NO 2 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 2 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 120 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 121 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a single nucleic polymorphism at nucleotide positions 100 and 264 271 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 2 on chromosome 8 that confers an improved water optimization associated trait 

 iii SEQ ID NO 3 is derived from the ZmDr1 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 2 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 122 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 123 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a single nucleic polymorphism at nucleotide position 216 of SEQ ID NO 3 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 3 in a genome that confers an improved water optimization associated trait 

 iv SEQ ID NO 4 is derived from the ZmDrA locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 4 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 124 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 125 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a single nucleic polymorphism at nucleotide position 503 of SEQ ID NO 4 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 4 on chromosome 7 that confers an improved water optimization associated trait 

 v SEQ ID NO 5 is derived from the ZmDr2 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 4 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 126 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 127 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide positions 818 821 of SEQ ID NO 5 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 5 on chromosome 2 that confers an improved water optimization associated trait 

 vi SEQ ID NO 6 is derived from the ZmDr3 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 6 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 128 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 129 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 254 of SEQ ID NO 6 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 6 on chromosome 2 that confers an improved water optimization associated trait 

 vii SEQ ID NO 7 is derived from the ZmDr4 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 7 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 130 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 131 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide positions 4497 4498 4505 4609 4641 4792 4836 4844 4969 and 4979 4981 of SEQ ID NO 7 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 7 on chromosome 8 that confers an improved water optimization associated trait 

 viii SEQ ID NO 8 is derived from the ZmMa3 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 8 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 132 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 133 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by polymorphisms at nucleotide positions 217 390 and 477 of SEQ ID NO 8 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 8 on chromosome 2 that confers an improved water optimization associated trait 

 ix SEQ ID NO 9 is derived from the genome and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 9 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 134 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 135 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 292 of SEQ ID NO 9 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 9 on chromosome 4 that confers an improved water optimization associated trait 

 x SEQ ID NO 10 is derived from the ZmBglcn locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 10 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 136 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 137 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 166 of SEQ ID NO 10 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 10 on chromosome 3 that confers an improved water optimization associated trait 

 xi SEQ ID NO 11 is derived from the ZmLOC100276591 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 11 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 138 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 139 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 148 of SEQ ID NO 11 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 11 in the genome that confers an improved water optimization associated trait 

 xii SEQ ID NO 12 is derived from the ZmDr7 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 12 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 140 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 141 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 94 of SEQ ID NO 12 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 12 on chromosome 1 that confers an improved water optimization associated trait 

 xiii SEQ ID NO 13 is derived from the ZmDr7 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 13 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 140 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 141 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 35 86 and or 89 of SEQ ID NO 13 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 13 on chromosome 1 that confers an improved water optimization associated trait 

 xiv SEQ ID NO 14 is derived from the ZmDr8 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 14 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 142 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 143 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 432 of SEQ ID NO 14 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 14 in the genome that confers an improved water optimization associated trait 

 xv SEQ ID NO 15 is derived from the ZmHsp70 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 15 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 144 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 145 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 753 of SEQ ID NO 15 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 15 on chromosome 1 that confers an improved water optimization associated trait 

 xvi SEQ ID NO 16 is derived from the ZmDr9 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 16 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 146 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 147 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 755 of SEQ ID NO 16 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 16 on chromosome 4 that confers an improved water optimization associated trait 

 xvii SEQ ID NO 17 is derived from the ZmDrB locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 17 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 148 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 149 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 431 of SEQ ID NO 17 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 17 in the genome that confers an improved water optimization associated trait 

 xviii SEQ ID NO 18 is derived from the ZmAdh1 1s locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 18 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 150 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 151 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 518 of SEQ ID NO 18 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 18 on chromosome 1 that confers an improved water optimization associated trait 

 xix SEQ ID NO 19 is derived from the ZmDr10 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 19 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 152 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 153 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 182 309 330 and 463 of SEQ ID NO 19 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 19 on chromosome 8 that confers an improved water optimization associated trait 

 xx SEQ ID NO 20 is derived from the ZmDrC locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 20 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 154 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 155 and further wherein this locus comprises one or more alleles of a water optimization associated trait wherein the one or more alleles are characterized by a polymorphism at nucleotide positions 773 776 of SEQ ID NO 20 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 20 in the genome that confers an improved water optimization associated trait 

 xxi SEQ ID NO 21 is derived from the ZmDr5 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 21 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 156 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 157 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 61 200 and 316 324 of SEQ ID NO 21 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 21 on chromosome 5 that confers an improved water optimization associated trait 

 xxii SEQ ID NO 22 is derived from the ZmDrD locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 22 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 158 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 159 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 211 of SEQ ID NO 22 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 22 on chromosome 5 that confers an improved water optimization associated trait 

 xxiii SEQ ID NO 23 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 23 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 160 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 161 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 116 and 217 of SEQ ID NO 21 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 23 on chromosome 8 that confers an improved water optimization associated trait 

 xxiv SEQ ID NO 24 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 24 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 162 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 163 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 746 of SEQ ID NO 24 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 24 on chromosome 8 that confers an improved water optimization associated trait 

 xxv SEQ ID NO 25 is derived from the ZmDr12 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 25 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 164 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 165 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 562 of SEQ ID NO 25 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 25 on chromosome 8 that confers an improved water optimization associated trait 

 xxvi SEQ ID NO 26 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 26 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 166 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 167 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 1271 of SEQ ID NO 26 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 26 on chromosome 8 that confers an improved water optimization associated trait 

 xxvii SEQ ID NO 27 is derived from the ZmDrE locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 27 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 168 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 169 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 64 and or 254 of SEQ ID NO 27 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 27 on chromosome 8 that confers an improved water optimization associated trait 

 xxviii SEQ ID NO 28 is derived from the ZmDrF locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 28 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 170 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 171 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 98 147 224 and or 496 of SEQ ID NO 28 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 28 on chromosome 9 that confers an improved water optimization associated trait 

 xxix SEQ ID NO 29 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 29 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 172 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 173 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 258 of SEQ ID NO 29 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 29 on chromosome 8 that confers an improved water optimization associated trait 

 xxx SEQ ID NO 30 is derived from the ZmDhn2 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 30 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 174 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 175 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 259 296 398 and or 1057 of SEQ ID NO 30 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 30 on chromosome 4 that confers an improved water optimization associated trait 

 xxxi SEQ ID NO 31 is derived from the ZmDr16 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 31 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 176 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 177 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 239 of SEQ ID NO 31 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 31 on chromosome 8 that confers an improved water optimization associated trait 

 xxxii SEQ ID NO 32 is derived from the ZmDr17 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 32 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 178 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 179 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 208 of SEQ ID NO 32 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 32 in the genome that confers an improved water optimization associated trait 

 xxxiii SEQ ID NO 33 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 33 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 180 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 181 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 391 of SEQ ID NO 33 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 33 on chromosome 8 that confers an improved water optimization associated trait 

 xxxiv SEQ ID NO 34 is derived from the ZmZCN6 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 34 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 182 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 183 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide positions 144 145 169 and or 537 of SEQ ID NO 34 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 34 on chromosome 4 that confers an improved water optimization associated trait 

 xxxv SEQ ID NO 35 is derived from the ZmDrG locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 35 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 184 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 185 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 76 of SEQ ID NO 35 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 35 on chromosome 5 that confers an improved water optimization associated trait 

 xxxvi SEQ ID NO 36 is derived from the ZmDhn1 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 36 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 186 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 187 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 500 568 and or 698 of SEQ ID NO 36 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 36 on chromosome 6 that confers an improved water optimization associated trait 

 xxxvii SEQ ID NO 37 is derived from the ZmDrH locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 37 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 188 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 189 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 375 and or 386 of SEQ ID NO 37 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 37 on chromosome 5 that confers an improved water optimization associated trait 

 xxxviii SEQ ID NO 38 is derived from the ZmDrI locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 38 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 190 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 191 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 309 and or 342 of SEQ ID NO 38 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 38 on chromosome 3 that confers an improved water optimization associated trait 

 xxxix SEQ ID NO 39 is derived from the ZmDrJ locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 39 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 192 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 193 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 445 of SEQ ID NO 39 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 39 on chromosome 5 that confers an improved water optimization associated trait 

 xl SEQ ID NO 40 is derived from the ZmH2B1 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 40 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 194 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 195 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 602 of SEQ ID NO 40 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 40 on chromosome 4 that confers an improved water optimization associated trait 

 xli SEQ ID NO 41 is derived from the ZmDr3 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 41 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 196 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 198 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 190 and or 580 of SEQ ID NO 41 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 41 on chromosome 2 that confers an improved water optimization associated trait 

 xlii SEQ ID NO 42 is derived from the ZmDrK locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 42 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 198 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 199 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide positions 238 266 267 and 808 of SEQ ID NO 42 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 42 in the genome that confers an improved water optimization associated trait 

 xliii SEQ ID NO 43 is derived from the ZmCat1 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 43 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 200 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 201 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 708 of SEQ ID NO 43 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 43 on chromosome 5 that confers an improved water optimization associated trait 

 xliv SEQ ID NO 44 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 44 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 202 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 203 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 266 of SEQ ID NO 44 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 44 on chromosome 8 that confers an improved water optimization associated trait 

 xlv SEQ ID NO 45 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 45 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 202 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 203 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 475 of SEQ ID NO 45 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 45 on chromosome 8 that confers an improved water optimization associated trait 

 xlvi SEQ ID NO 46 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 46 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 204 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 205 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 386 of SEQ ID NO 46 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 46 on chromosome 8 that confers an improved water optimization associated trait 

 xlvii SEQ ID NO 47 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 47 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 206 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 207 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 87 of SEQ ID NO 47 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 47 on chromosome 8 that confers an improved water optimization associated trait 

 xlviii SEQ ID NO 48 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 48 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 208 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 209 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 472 of SEQ ID NO 48 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 48 on chromosome 8 that confers an improved water optimization associated trait 

 xlix SEQ ID NO 49 is derived from the ZmRIC1 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 49 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 210 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 211 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 166 224 650 and or 892 of SEQ ID NO 49 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 49 on chromosome 8 that confers an improved water optimization associated trait 

 l SEQ ID NO 50 is derived from the ZmPK4 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 50 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 212 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 213 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 541 of SEQ ID NO 50 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 50 on chromosome 8 that confers an improved water optimization associated trait 

 li SEQ ID NO 51 is derived from the ZmPK4 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 51 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 212 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 213 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 111 of SEQ ID NO 51 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 51 on chromosome 8 that confers an improved water optimization associated trait 

 lii SEQ ID NO 52 is derived from the Zpu1 locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 52 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 214 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 215 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 442 of SEQ ID NO 52 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 52 on chromosome 2 that confers an improved water optimization associated trait 

 liii SEQ ID NO 53 is derived from the ZmDrL locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 53 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 216 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 217 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 83 428 491 and or 548 of SEQ ID NO 53 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 53 on chromosome 9 that confers an improved water optimization associated trait 

 liv SEQ ID NO 54 is derived from the ZmDrM locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 54 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 218 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 219 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 126 of SEQ ID NO 54 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 54 on chromosome 7 that confers an improved water optimization associated trait 

 lv SEQ ID NO 55 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 55 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 220 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 221 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 193 of SEQ ID NO 55 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 55 on chromosome 8 that confers an improved water optimization associated trait 

 lvi SEQ ID NO 56 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 56 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 222 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 223 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by one or more polymorphisms at nucleotide positions 237 and or 516 of SEQ ID NO 56 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 56 on chromosome 8 that confers an improved water optimization associated trait 

 lvii SEQ ID NO 57 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 57 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 224 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 225 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 173 of SEQ ID NO 57 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 57 on chromosome 8 that confers an improved water optimization associated trait 

 lviii SEQ ID NO 58 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 58 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 226 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 227 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 486 of SEQ ID NO 58 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 58 on chromosome 8 that confers an improved water optimization associated trait 

 lix SEQ ID NO 59 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 59 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 228 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 229 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 729 of SEQ ID NO 59 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 59 on chromosome 8 that confers an improved water optimization associated trait and

 lx SEQ ID NO 60 is derived from a water optimization locus and is defined by a first oligonucleotide and a second oligonucleotide wherein said oligonucleotides can be employed to amplify a subsequence of SEQ ID NO 60 generated by amplifying a nucleic acid with a first oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 230 and a second oligonucleotide comprising a nucleotide sequence as set forth in SEQ ID NO 231 and further wherein this locus comprises alleles of a water optimization associated trait wherein the alleles are characterized by a polymorphism at nucleotide position 267 of SEQ ID NO 60 and comprises any part of a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of SEQ ID NO 60 on chromosome 8 that confers an improved water optimization associated trait and

In some embodiments a DNA sequence within 1 2 3 4 5 6 7 8 9 10 15 20 or 25 cM of a marker of the presently disclosed subject matter displays a genetic recombination frequency of less than about 25 20 15 10 9 8 7 6 5 4 3 2 or 1 with the marker of the presently disclosed subject matter. In some embodiments the germplasm is a line or variety.

DNA fragments associated with the presence of a water optimization associated trait alleles and or haplotypes including but not limited to SEQ ID NOs 1 117 400 and 401 are also provided. In some embodiments the DNA fragments associated with the presence of a water optimization associated trait have a predicted length and or nucleic acid sequence and detecting a DNA fragment having the predicted length and or the predicted nucleic acid sequence is performed such that the amplified DNA fragment has a length that corresponds plus or minus a few bases e.g. a length of one two or three bases more or less to the predicted length. In some embodiments a DNA fragment is an amplified fragment and the amplified fragment has a predicted length and or nucleic acid sequence as does an amplified fragment produced by a similar reaction with the same primers with the DNA from the plant in which the marker was first detected or the nucleic acid sequence that corresponds i.e. as a nucleotide sequence identity of more than 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 or 99 to the expected sequence as based on the sequence of the marker associated with that water optimization associated trait in the plant in which the marker was first detected. Upon a review of the instant disclosure one of ordinary skill in the art would appreciate that markers that are absent in plants while they were present in at least one parent plant so called trans markers can also be useful in assays for detecting a desired trait in an progeny plant although testing for the absence of a marker to detect the presence of a specific trait is not optimal. The detecting of an amplified DNA fragment having the predicted length or the predicted nucleic acid sequence can be performed by any of a number of techniques including but not limited to standard gel electrophoresis techniques and or by using automated DNA sequencers. The methods are not described here in detail as they are well known to the skilled person.

The primer in some embodiments an extension primer and in some embodiments an amplification primer is in some embodiments single stranded for maximum efficiency in extension and or amplification. In some embodiments the primer is an oligodeoxyribonucleotide. A primer is typically sufficiently long to prime the synthesis of extension and or amplification products in the presence of the agent for polymerization. The minimum lengths of the primers can depend on many factors including but not limited to temperature and composition A T vs. G C content of the primer.

In the context of an amplification primer these are typically provided as one or more sets of bidirectional primers that include one or more forward and one or more reverse primers as commonly used in the art of DNA amplification such as in PCR amplification As such it will be understood that the term primer as used herein can refer to more than one primer particularly in the case where there is some ambiguity in the information regarding the terminal sequence s of the target region to be amplified. Hence a primer can include a collection of primer oligonucleotides containing sequences representing the possible variations in the sequence or includes nucleotides which allow a typical base pairing. Primers can be prepared by any suitable method. Methods for preparing oligonucleotides of specific sequence are known in the art and include for example cloning and restriction of appropriate sequences and direct chemical synthesis. Chemical synthesis methods can include for example the phospho di or tri ester method the diethylphosphoramidate method and the solid support method disclosed in U.S. Pat. No. 4 458 068.

Primers can be labeled if desired by incorporating detectable moieties by for instance spectroscopic fluorescence photochemical biochemical immunochemical or chemical moieties.

Template dependent extension of an oligonucleotide primer is catalyzed by a polymerizing agent in the presence of adequate amounts of the four deoxyribonucleotides triphosphates dATP dGTP dCTP and dTTP i.e. dNTPs or analogues in a reaction medium that comprises appropriate salts metal cations and a pH buffering system. Suitable polymerizing agents are enzymes known to catalyze primer and template dependent DNA synthesis. Known DNA polymerases include for example DNA polymerase or its Klenow fragment T4 DNA polymerase and Taq DNA polymerase as well as various modified versions thereof. The reaction conditions for catalyzing DNA synthesis with these DNA polymerases are known in the art. The products of the synthesis are duplex molecules consisting of the template strands and the primer extension strands which include the target sequence. These products in turn can serve as template for another round of replication. In the second round of replication the primer extension strand of the first cycle is annealed with its complementary primer synthesis yields a short product which is bound on both the 5 and the 3 ends by primer sequences or their complements. Repeated cycles of denaturation primer annealing and extension can result in the exponential accumulation of the target region defined by the primers. Sufficient cycles are run to achieve the desired amount of polynucleotide containing the target region of nucleic acid. The desired amount can vary and is determined by the function which the product polynucleotide is to serve.

The PCR method is well described in handbooks and known to the skilled person. After amplification by PCR the target polynucleotides can be detected by hybridization with a probe polynucleotide which forms a stable hybrid with that of the target sequence under stringent to moderately stringent hybridization and wash conditions. If it is expected that the probes will be essentially completely complementary i.e. about 99 or greater to the target sequence stringent conditions can be used. If some mismatching is expected for example if variant strains are expected with the result that the probe will not be completely complementary the stringency of hybridization can be reduced. In some embodiments conditions are chosen to rule out non specific adventitious binding. Conditions that affect hybridization and that select against non specific binding are known in the art and are described in for example Sambrook Russell 2001. Generally lower salt concentration and higher temperature increase the stringency of hybridization conditions.

In order to detect the presence of two water optimization associated alleles on a single chromosome in a plant chromosome painting methods can also be used. In such methods at least a first water optimization associated allele and at least a second water optimization associated allele can be detected in the same chromosome by in situ hybridization or in situ PCR techniques. More conveniently the fact that two water optimization associated alleles are present on a single chromosome can be confirmed by determining that they are in coupling phase i.e. that the traits show reduced segregation when compared to genes residing on separate chromosomes.

The water optimization associated alleles identified herein are located on a number of different chromosomes or linkage groups and their locations can be characterized by a number of otherwise arbitrary markers. In the present investigations single nucleotide polymorphisms SNPs were used although restriction fragment length polymorphism RFLP markers amplified fragment length polymorphism AFLP markers microsatellite markers e.g. SSRs insertion mutation markers sequence characterized amplified region SCAR markers cleaved amplified polymorphic sequence CAPS markers isozyme markers microarray based technologies TAQMAN Assays ILLUMINA GOLDENGATE Assay analysis nucleic acid sequencing technologies or combinations of these markers might also have been used and indeed can be used.

In general providing complete sequence information for a water optimization associated allele and or haplotype is unnecessary as the way in which the water optimization associated allele and or haplotype is first detected through an observed correlation between the presence of one or more single nucleotide polymorphisms and the presence of a particular phenotypic trait allows one to trace among a population of progeny plants those plants that have the genetic potential for exhibiting a particular phenotypic trait. By providing a non limiting list of markers the presently disclosed subject matter thus provides for the effective use of the presently disclosed water optimization associated alleles and or haplotypes in breeding programs. In some embodiments a marker is specific for a particular line of descent. Thus a specific trait can be associated with a particular marker.

The markers as disclosed herein not only indicate the location of the water optimization associated allele they also correlate with the presence of the specific phenotypic trait in a plant. It is noted that single nucleotide polymorphisms that indicate where a water optimization associated allele is present in the genome is non limiting. In general the location of a water optimization associated allele is indicated by a set of single nucleotide polymorphisms that exhibit statistical correlation to the phenotypic trait. Once a marker is found outside a single nucleotide polymorphism i.e. one that has a LOD score below a certain threshold indicating that the marker is so remote that recombination in the region between that marker and the water optimization associated allele occurs so frequently that the presence of the marker does not correlate in a statistically significant manner to the presence of the phenotype the boundaries of the water optimization associated allele can be considered set. Thus it is also possible to indicate the location of the water optimization associated allele by other markers located within that specified region. It is further noted that a single nucleotide polymorphism can also be used to indicate the presence of the water optimization associated allele and thus of the phenotype in an individual plant which in some embodiments means that it can be used in marker assisted selection MAS procedures.

In principle the number of potentially useful markers can be very large. Any marker that is linked to a water optimization associated allele e.g. falling within the physically boundaries of the genomic region spanned by the markers having established LOD scores above a certain threshold thereby indicating that no or very little recombination between the marker and the water optimization associated allele occurs in crosses as well as any marker in linkage disequilibrium to the water optimization associated allele as well as markers that represent the actual causal mutations within the water optimization associated allele can be used in the presently disclosed methods and compositions and are within the scope of the presently disclosed subject matter. This means that the markers identified in the application as associated with the water optimization associated allele e.g. markers that are present in or comprise any of SEQ ID NOs 1 24 are non limiting examples of markers suitable for use in the presently disclosed methods and compositions. Moreover when a water optimization associated allele or the specific trait conferring part thereof is introgressed into another genetic background i.e. into the genome of another maize or another plant species then some markers might no longer be found in the progeny although the trait is present therein indicating that such markers are outside the genomic region that represents the specific trait conferring part of the water optimization associated allele in the original parent line only and that the new genetic background has a different genomic organization. Such markers of which the absence indicates the successful introduction of the genetic element in the progeny are called trans markers and can be equally suitable with respect to the presently disclosed subject matter.

Upon the identification of a water optimization associated allele and or haplotype the water optimization associated allele and or haplotype effect e.g. the trait can for instance be confirmed by assessing trait in progeny segregating for the water optimization associated alleles and or haplotypes under investigation. The assessment of the trait can suitably be performed by using phenotypic assessment as known in the art for water optimization traits. For example field trials under natural and or irrigated conditions can be conducted to assess the traits of hybrid and or inbred maize.

The markers provided by the presently disclosed subject matter can be used for detecting the presence of one or more water optimization trait alleles and or haplotypes at loci of the presently disclosed subject matter in a suspected water optimization trait introgressed maize plant and can therefore be used in methods involving marker assisted breeding and selection of such water optimization trait bearing maize plants. In some embodiments detecting the presence of a water optimization associated allele and or haplotype of the presently disclosed subject matter is performed with at least one of the markers for a water optimization associated allele and or haplotype as defined herein. The presently disclosed subject matter therefore relates in another aspect to a method for detecting the presence of a water optimization associated allele and or haplotype for at least one of the presently disclosed water optimization traits comprising detecting the presence of a nucleic acid sequence of the water optimization associated allele and or haplotype in a trait bearing maize plant which presence can be detected by the use of the disclosed markers.

In some embodiments the detecting comprises determining the nucleotide sequence of a nucleic acid associated with a water optimization associated trait allele and or haplotype. The nucleotide sequence of a water optimization associated allele and or haplotype of the presently disclosed subject matter can for instance be resolved by determining the nucleotide sequence of one or more markers associated with the water optimization associated allele and or haplotype and designing internal primers for the marker sequences that can then be used to further determine the sequence of the water optimization associated allele and or haplotype outside of the marker sequences.

For example the nucleotide sequence of the SNP markers disclosed herein can be obtained by isolating the markers from the electrophoresis gel used in the determination of the presence of the markers in the genome of a subject plant and determining the nucleotide sequence of the markers by for example dideoxy chain termination sequencing methods which are well known in the art. In some embodiments of such methods for detecting the presence of a water optimization associated allele and or haplotype in a trait bearing maize plant the method can also comprise providing a oligonucleotide or polynucleotide capable of hybridizing under stringent hybridization conditions to a nucleic acid sequence of a marker linked to the water optimization associated allele and or haplotype in some embodiments selected from the markers disclosed herein contacting the oligonucleotide or polynucleotide with digested genomic nucleic acid of a trait bearing maize plant and determining the presence of specific hybridization of the oligonucleotide or polynucleotide to the digested genomic nucleic acid. In some embodiments the method is performed on a nucleic acid sample obtained from the trait bearing maize plant although in situ hybridization methods can also be employed. Alternatively one of ordinary skill in the art can once the nucleotide sequence of the water optimization associated allele and or haplotype has been determined design specific hybridization probes or oligonucleotides capable of hybridizing under stringent hybridization conditions to the nucleic acid sequence of the water optimization associated allele and or haplotype and can use such hybridization probes in methods for detecting the presence of a water optimization associated allele and or haplotype disclosed herein in a trait bearing maize plant.

Particular nucleotides that are present at particular locations in the markers and nucleic acids disclosed herein can be determined using standard molecular biology techniques including but not limited to amplification of genomic DNA from plants and subsequent sequencing. Additionally oligonucleotide primers can be designed that would be expected to specifically hybridize to particular sequences that include the polymorphisms disclosed herein. For example oligonucleotides can be designed to distinguish between the A allele and the G allele at a nucleotide position that corresponds to position 270 of SEQ ID NO 1 using oligonucleotides comprising consisting essentially of or consisting of SEQ ID NOs 232 and 233. The relevant difference between SEQ ID NOs 232 and 233 is that the former has a G nucleotide at position 19 and the latter has an A nucleotide at position 19. Thus SEQ ID NO 232 hybridization conditions can be designed that would permit SEQ ID NO 232 to specifically hybridize to the G allele if present but not hybridize to the A allele if present. Thus hybridization using these two primers that differ in only one nucleotide can be employed to assay for the presence of one or the other allele at a nucleotide position that corresponds to position 270 of SEQ ID NO 1.

In some embodiments the alleles comprising the marker associated with enhanced drought tolerance are detected using a plurality of probes selected from the group consisting of 

 i SEQ ID NOs 348 and 349 SEQ ID NOs 350 and 351 SEQ ID NOs 360 and 361 SEQ ID NOs 372 and 373 SEQ ID NOs 382 and 383 SEQ ID NOs 388 and 389 SEQ ID NOs 382 and 383 and SEQ ID NOs 398 and 399 

 ii SEQ ID NOs 350 and 251 SEQ ID NOs 356 and 357 SEQ ID NOs 364 and 365 SEQ ID NOs 366 and 367 SEQ ID NOs 374 and 375 SEQ ID NOs 378 and 379 SEQ ID NOs 382 and 383 and SEQ ID NOs 384 and 385 

 iii SEQ ID NOs 348 and 349 SEQ ID NOs 352 and 353 SEQ ID NOs 358 and 359 SEQ ID NOs 362 and 363 SEQ ID NOs 370 and 371 SEQ ID NOs 374 and 375 SEQ ID NOs 382 and 383 SEQ ID NOs 386 and 387 and SEQ ID NOs 394 and 395 

 iv SEQ ID NOs 346 and 347 SEQ ID NOs 352 and 353 SEQ ID NOs 356 and 357 SEQ ID NOs 372 and 373 SEQ ID NOs 388 and 389 and SEQ ID NOs 394 and 395 

 v SEQ ID NOs 351 and 351 SEQ ID NOs 354 and 355 SEQ ID NOs 368 and 369 SEQ ID NOs 372 and 373 SEQ ID NOs 376 and 377 SEQ ID NOs 386 and 387 SEQ ID NOs 390 and 391 SEQ ID NOs 396 and 397 and SEQ ID NOs 398 and 399 

 vi SEQ ID NOs 352 and 353 SEQ ID NOs 354 and 355 SEQ ID NOs 370 and 371 SEQ ID NOs 386 and 387 SEQ ID NOs 388 and 389 

 vii SEQ ID NOs 350 and 351 SEQ ID NOs 382 and 383 SEQ ID NOs 388 and 389 and SEQ ID NOs 392 and 393 

 viii SEQ ID NOs 350 and 351 SEQ ID NOs 366 and 367 SEQ ID NOs 374 and 375 SEQ ID NOs 378 and 379 SEQ ID NOs 382 and 383 and SEQ ID NOs 384 and 385 

 ix SEQ ID NOs 352 and 353 SEQ ID NOs 370 and 371 SEQ ID NOs 380 and 381 SEQ ID NOs 382 and 383 and SEQ ID NOs 386 and 387 

In some embodiments the marker can comprise consist essentially of or consist of the reverse complement of any of the aforementioned markers. In some embodiments one or more of the alleles that make up a marker haplotype is present as described above whilst one or more of the other alleles that make up the marker haplotype is present as the reverse complement of the allele s described above. In some embodiments each of the alleles that make up a marker haplotype is present as the reverse complement of the allele s described above.

In some embodiments the marker can comprise consist essentially of or consist of an informative fragment of any of the aforementioned markers the reverse complement of any of the aforementioned markers or an informative fragment of the reverse complement of any of the aforementioned markers. In some embodiments one or more of the alleles sequences that make up a marker haplotype is present as described above whilst one or more of the other alleles sequences that make up the marker haplotype is present as the reverse complement of the alleles sequences described above. In some embodiments one or more of the alleles sequences that make up a marker haplotype is present as described above whilst one or more of the other alleles sequences that make up the marker haplotype is present as an informative fragment of the alleles sequences described above. In some embodiments one or more of the alleles sequences that make up a marker haplotype is present as described above whilst one or more of the other alleles sequences that make up the marker haplotype is present as an informative fragment of the reverse complement of the alleles sequences described above. In some embodiments each of the alleles sequences that make up a marker haplotype is present as an informative fragment of the alleles sequences described above the reverse complement of the alleles sequences described above or an informative fragment of the reverse complement of the alleles sequences described above.

In some embodiments the marker can comprise consist essentially of or consist of any marker linked to the aforementioned markers. That is any allele and or haplotype that is in linkage disequilibrium with any of the aforementioned markers can also be used to identify select and or produce a maize plant with enhanced drought tolerance. Linked markers can be determined for example by using resources available on the MaizeGDB website.

Isolated and purified markers associated with enhanced drought tolerance are also provided. Such markers can comprise consist essentially of or consist of a nucleotide sequence as set forth in any of SEQ ID NOs 1 117 400 AND 401 the reverse complement thereof or an informative fragment thereof. In some embodiments the marker comprises a detectable moiety. In some embodiments the marker permits the detection of one or more of the marker alleles identified herein.

Compositions comprising a primer pair capable of amplifying a nucleic acid sample isolated from a maize plant or germplasm to generate a marker associated with enhanced drought tolerance are also provided. In some embodiments the marker comprises a nucleotide sequence as set forth herein the reverse complement thereof or an informative fragment thereof. In some embodiments the marker comprises a nucleotide sequence that is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to a nucleotide sequence set forth herein the reverse complement thereof or an informative fragment thereof. In some embodiments the primer pair is one of the amplification primer pairs identified in Table 1 above. One of ordinary skill in the art will understand how to select alternative primer pairs according to methods well known in the art.

Table 5 provides a summary of favorable alleles and single locus haplotypes that are associated with water optimization.

As used herein the phrase haplotype code refers to a collection of SNPs present in a plant wherein each favorable allele listed in the single locus haplotype column of Table 5 is present in the plant. For example a plant is said to have haplotype A when at least one locus corresponding to SEQ ID NO 1 in the plant has a G at nucleotide position 115 an A at position 270 of SEQ ID NO 1 a T at nucleotide position 301 and an A at position 483. Haplotype A can be alternatively referred to as GATA with respect to these particular nucleotide positions. Similarly a plant is said to have haplotype C when at least one locus correspond to SEQ ID NO 8 in the plant has an A at nucleotide position 217 a G at nucleotide position 290 and an A at nucleotide position 477 of SEQ ID NO 8. Haplotype C can thus be alternatively referred to as AGA with respect to these particular nucleotide positions.

The identification of plants with different alleles and or haplotypes of interest can provide starting materials for combining alleles and or haplotypes in progeny plants via breeding strategies designed to stack the alleles and or haplotypes. As used herein the term stacking and grammatical variants thereof refers to the intentional accumulation by breeding including but not limited to crossing two plants selfing a single plant and or creating a double haploid from a single plant of favorable water optimization haplotypes in plants such that a plant s genome has at least one additional favorable water optimization haplotype than its immediate progenitor s . Stacking includes in some embodiments conveying one or more water optimization traits alleles and or haplotypes into a progeny maize plant such that the progeny maize plant includes higher number of water optimization traits alleles and or haplotypes than does either parent from which it was derived. By way of example and not limitation if Parent 1 has haplotypes A B and C and Parent 2 has haplotypes D E and F stacking refers to the production of a plant that has any of A B and C with any combination of D E and F. Particularly stacking refers in some embodiments to producing a plant that has A B and C as well as one or more of D E and F or producing a plant that has D E and F as well as one or more of A B and C. In some embodiments stacking refers to the production of a plant from a bi parental cross that contains all water optimization associated haplotypes possessed by either parent.

In some embodiments the water optimization trait is Grain Yield Drought and the favorable haplotype comprises a nucleotide sequence comprising a T at nucleotide position 301 a G at nucleotide position 115 an A at nucleotide position 483 and an A at nucleotide position 270 of SEQ ID NO 1 a TCC trinucleotide at nucleotide positions 4979 4981 a G at nucleotide position 4969 an A at nucleotide position 4641 a T at nucleotide position 4609 a deletion of nucleotide positions 4497 4498 a T at nucleotide position 4792 a T at nucleotide position 4836 a G at nucleotide position 4505 and a C at nucleotide position 4844 of SEQ ID NO 7 an A at nucleotide position 217 a G at nucleotide position 390 and an A at nucleotide position 477 of SEQ ID NO 8 a G at nucleotide position 463 a G at nucleotide position 330 a G at nucleotide position 182 and an A at nucleotide position 309 of SEQ ID NO 19 a G at nucleotide position 64 and an A at nucleotide position 254 of SEQ ID NO 27 a C at nucleotide position 98 a T at nucleotide position 147 a C at nucleotide position 224 and a T at nucleotide position 496 of SEQ ID NO 28 a C at nucleotide position 500 a G at nucleotide position 568 and a T at nucleotide position 698 of SEQ ID NO 36 a deletion of nucleotide positions 266 267 a C at nucleotide position 808 and an A at nucleotide position 238 of SEQ ID NO 42 and or a C at nucleotide position 166 an A at nucleotide position 224 a G at nucleotide position 650 and a G at nucleotide position 892 of SEQ ID NO 49.

In some embodiments the water optimization trait is Grain Yield Well Watered and the favorable haplotype comprises a nucleotide sequence comprising an A at nucleotide position 217 a G at nucleotide position 390 and an A at nucleotide position 477 of SEQ ID NO 8 a C at nucleotide position 500 a G at nucleotide position 568 and a T at nucleotide position 698 of SEQ ID NO 36 and or a C at nucleotide position 83 a C at nucleotide position 548 a C at nucleotide position 491 and a C at nucleotide position 428 of SEQ ID NO 53.

In some embodiments the water optimization trait is Yield Reduction Hybrid and the favorable haplotype comprises a nucleotide sequence comprising a C at nucleotide position 98 a T at nucleotide position 147 a C at nucleotide position 224 and a T at nucleotide position 496 of SEQ ID NO 28.

In some embodiments the water optimization trait is Yield Reduction Inbred and the favorable haplotype comprises a nucleotide sequence comprising a TCC trinucleotide at nucleotide positions 4979 4981 a G at nucleotide position 4969 an A at nucleotide position 4641 a T at nucleotide position 4609 a deletion of nucleotide positions 4497 4498 a T at nucleotide position 4792 a T at nucleotide position 4836 a G at nucleotide position 4505 and a C at nucleotide position 4844 of SEQ ID NO 7 an A at nucleotide position 217 a G at nucleotide position 390 and an A at nucleotide position 477 of SEQ ID NO 8 a G at nucleotide position 64 and an A at nucleotide position 254 of SEQ ID NO 27 and or a C at nucleotide position 83 a C at nucleotide position 548 a C at nucleotide position 491 and a C at nucleotide position 428 of SEQ ID NO 53.

In some embodiments the water optimization trait is ASI and the favorable haplotype comprises a nucleotide sequence comprising a TCC trinucleotide at nucleotide positions 4979 4981 a G at nucleotide position 4969 an A at nucleotide position 4641 a T at nucleotide position 4609 a deletion of nucleotide positions 4497 4498 a T at nucleotide position 4792 a T at nucleotide position 4836 a G at nucleotide position 4505 and a C at nucleotide position 4844 of SEQ ID NO 7.

In some embodiments the water optimization trait is Percent Barren and the favorable haplotype comprises a nucleotide sequence comprising a TCC trinucleotide at nucleotide positions 4979 4981 a G at nucleotide position 4969 an A at nucleotide position 4641 a T at nucleotide position 4609 a deletion of nucleotide positions 4497 4498 a T at nucleotide position 4792 a T at nucleotide position 4836 a G at nucleotide position 4505 and a C at nucleotide position 4844 of SEQ ID NO 7 a G at nucleotide position 463 a G at nucleotide position 330 a G at nucleotide position 182 and an A at nucleotide position 309 of SEQ ID NO 19 a C at nucleotide position 61 a C at nucleotide position 200 and a deletion of nucleotide positions 316 324 of SEQ ID NO 21 and or an A at nucleotide position 398 a T at nucleotide position 296 a T at nucleotide position 259 and a C at nucleotide position 1057 of SEQ ID NO 30.

In some embodiments of the presently disclosed subject matter the genomes of inbred or hybrid plants comprise at least three four five six seven eight or nine of haplotypes A M wherein haplotypes A M are associated with water optimization and are defined herein. In some embodiments the inbred or hybrid plant comprises a genome comprising Haplotypes C D and G Haplotypes C D and L Haplotypes C G and H Haplotypes C G and I Haplotypes C I and L Haplotypes E G and I Haplotypes F G and H Haplotypes A C F and G Haplotypes C E H and I Haplotypes C G H and I Haplotypes C H I and K Haplotypes C H I and L Haplotypes E F G and H Haplotypes A C G H and I Haplotypes B C D G and L Haplotypes C E G H and I Haplotypes C G H I and L Haplotypes A C G H I and K Haplotypes C E F G H I J K and L Haplotypes C D G and M Haplotypes C D L and M Haplotypes C G H and M Haplotypes C G I and M Haplotypes C I L and M Haplotypes E G I and M Haplotypes F G H and M Haplotypes A C F G and M Haplotypes C E H I and M Haplotypes C G H I and M Haplotypes C H I K and M Haplotypes C H I L and M Haplotypes E F G H and M Haplotypes A C G H I and M Haplotypes B C D G L and M Haplotypes C E G H I and M Haplotypes C G H I L and M Haplotypes A C G H I K and M and Haplotypes C E F G H I J K L and M. In some embodiments the inbred or hybrid plant is a hybrid plant that is homozygous for at least one of Haplotypes A M.

In some embodiments the inbred or hybrid plant comprises a genome comprising Haplotypes A C E G H and I optionally further comprising Haplotype M Haplotypes B C D E F G H I and L optionally further comprising Haplotype M Haplotypes C D E F G H and L optionally further comprising Haplotype M Haplotypes B C D G I and L optionally further comprising Haplotype M Haplotypes B C D E G H I and L optionally further comprising Haplotype M Haplotypes C D E F G H I J K and L optionally further comprising Haplotype M Haplotypes A C G H and I optionally further comprising Haplotype M Haplotypes C E F G H and I optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M Haplotypes C D E F G and H optionally further comprising Haplotype M Haplotypes D E F G and H optionally further comprising Haplotype M Haplotypes A C G H and I optionally further comprising Haplotype M Haplotypes A C E G H I and K optionally further comprising Haplotype M Haplotype C E G H I and L optionally further comprising Haplotype M Haplotypes C D E G H I and L optionally further comprising Haplotype M Haplotypes B C D E G H I and L optionally further comprising Haplotype M Haplotypes A C G H and I optionally further comprising Haplotype M Haplotypes A C G H I and K optionally further comprising Haplotype M Haplotypes C G H I and L optionally further comprising Haplotype M Haplotypes C D G H I and L optionally further comprising Haplotype M Haplotypes B C D G H I and L optionally further comprising Haplotype M Haplotypes A C E F G H and I optionally further comprising Haplotype M Haplotypes A C E F G H I and K optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M Haplotypes C D E F G H I and L optionally further comprising Haplotype M Haplotypes A C E F G H I J K and L optionally further comprising Haplotype M Haplotypes A C E F G H I J K and L optionally further comprising Haplotype M Haplotypes C E F G H I J K and L optionally further comprising Haplotype M Haplotypes C D E F G H I J K and L optionally further comprising Haplotype M Haplotypes B C D E F G H I J K and L optionally further comprising Haplotype M Haplotypes A C E F G H and I optionally further comprising Haplotype M Haplotypes A C E F G H I and K optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M Haplotypes B C D E F G H and L optionally further comprising Haplotype M Haplotypes C E F G H I J K and L optionally further comprising Haplotype M Haplotypes C D G H and L optionally further comprising Haplotype M Haplotypes C E F G H I and L optionally further comprising Haplotype M and or Haplotypes B C D E G I and L optionally further comprising Haplotype M.

As used herein a plant that comprises multiple haplotypes can also be referred to by a code designating the haplotypes its posseses. Thus for example a plant that comprises at least one copy of Haplotyes C D E F G H I J K and L in its genome can be referred to as CDEFGHIJKL a plant that comprises at least one copy of Haplotypes B C D E F G H I J K and L in its genome can be referred to as BCDEFGHIJKL etc. In some embodiments uppercase and lowercase letters are employed to further delinate those haplotypes for which a plant or a cell thereof is either homozygous e.g. uppercase or heterozygous e.g. lowercase . By way of example and not limitation a plant or a cell that is referred to as CDEFGHIJKL has at least one of Haplotypes C D E F G H I J K and L. In some embodiments this designation would indicate that the plant or cell is homozygous for each of these haplotypes. Similarly the designation cdefghijkl indicates that the plant or cell is heterozygous for Haplotypes C D E F G H I J K and L. And finally the designation CdeFGhijKL indicates that the plant or cell is homozygous for Haplotypes C F G K and L is homozygous for Haplotypes D E H I and J. In some embodiments this designation further indicates that plant or cell lacks Haplotypes A and B although in some embodiments it indicates that the status of the plant or cell with respect to these Haplotypes is unknown or untested.

Markers can be used in a variety of plant breeding applications. See e.g. Staub et al. 31 729 1996 Tanksley 1 3 1983 . One of the main areas of interest is to increase the efficiency of backcrossing and introgressing genes using marker assisted selection MAS . In general MAS takes advantage of genetic markers that have been identified as having a significant likelihood of co segregation with a desired trait. Such markers are presumed to be in near the gene s that give rise to the desired phenotype and their presence indicates that the plant will possess the desired trait. Plants which possess the marker are expected to transfer the desired phenotype to their progeny.

A marker that demonstrates linkage with a locus affecting a desired phenotypic trait provides a useful tool for the selection of the trait in a plant population. This is particularly true where the phenotype is hard to assay or occurs at a late stage in plant development. Since DNA marker assays are less laborious and take up less physical space than field phenotyping much larger populations can be assayed increasing the chances of finding a recombinant with the target segment from the donor line moved to the recipient line. The closer the linkage the more useful the marker as recombination is less likely to occur between the marker and the gene causing or imparting the trait. Having flanking markers decreases the chances that false positive selection will occur. The ideal situation is to have a marker in the gene itself so that recombination cannot occur between the marker and the gene. Such a marker is called a perfect marker. 

When a gene is introgressed by MAS it is not only the gene that is introduced but also the flanking regions. Gepts 42 1780 2002 . This is referred to as linkage drag. In the case where the donor plant is highly unrelated to the recipient plant these flanking regions carry additional genes that can code for agronomically undesirable traits. This linkage drag can also result in reduced yield or other negative agronomic characteristics even after multiple cycles of backcrossing into the elite maize line. This is also sometimes referred to as yield drag. The size of the flanking region can be decreased by additional backcrossing although this is not always successful as breeders do not have control over the size of the region or the recombination breakpoints. Young et al. 120 579 1998 . In classical breeding it is usually only by chance that recombinations which contribute to a reduction in the size of the donor segment are selected. Tanksley et al. 7 257 1989 . Even after 20 backcrosses one can expect to find a sizeable piece of the donor chromosome still linked to the gene being selected. With markers however it is possible to select those rare individuals that have experienced recombination near the gene of interest. In 150 backcross plants there is a 95 chance that at least one plant will have experienced a crossover within 1 cM of the gene based on a single meiosis map distance. Markers allow for unequivocal identification of those individuals. With one additional backcross of 300 plants there would be a 95 chance of a crossover within 1 cM single meiosis map distance of the other side of the gene generating a segment around the target gene of less than 2 cM based on a single meiosis map distance. This can be accomplished in two generations with markers while it would have required on average 100 generations without markers. See Tanksley et al. supra. When the exact location of a gene is known flanking markers surrounding the gene can be utilized to select for recombinations in different population sizes. For example in smaller population sizes recombinations can be expected further away from the gene so more distal flanking markers would be required to detect the recombination.

The availability of integrated linkage maps of the maize genome containing increasing densities of public maize markers has facilitated maize genetic mapping and MAS. See e.g. the IBM2 Neighbors maps which are available online on the MaizeGDB website.

Of all the molecular marker types SNPs are the most abundant and have the potential to provide the highest genetic map resolution. Bhattramakki et al. 48 539 2002 . SNPs can be assayed in a so called ultra high throughput fashion because they do not require large amounts of nucleic acid and automation of the assay is straight forward. SNPs also have the benefit of being relatively low cost systems. These three factors together make SNPs highly attractive for use in MAS. Several methods are available for SNP genotyping including but not limited to hybridization primer extension oligonucleotide ligation nuclease cleavage minisequencing and coded spheres. Such methods have been reviewed in various publications Gut 17 475 2001 Shi 47 164 2001 Kwok 1 95 2000 Bhattramakki and Rafalski PG TDNA FP CABI Publishing Wallingford 2001 . A wide range of commercially available technologies utilize these and other methods to interrogate SNPs including Masscode Qiagen Germantown Md. Invader Hologic Madison Wis. SnapShot Applied Biosystems Foster City Calif. Taqman Applied Biosystems Foster City Calif. and Beadarrays Illumina San Diego Calif. .

A number of SNPs together within a sequence or across linked sequences can be used to describe a haplotype for any particular genotype. Ching et al. 3 19 2002 Gupta et al. 2001 Rafalski 162 329 2002b . Haplotypes can be more informative than single SNPs and can be more descriptive of any particular genotype. For example a single SNP can be allele T for a specific drought tolerant line or variety but the allele T might also occur in the maize breeding population being utilized for recurrent parents. In this case a combination of alleles at linked SNPs can be more informative. Once a unique haplotype has been assigned to a donor chromosomal region that haplotype can be used in that population or any subset thereof to determine whether an individual has a particular gene. The use of automated high throughput marker detection platforms known to those of ordinary skill in the art makes this process highly efficient and effective.

The markers of the presently disclosed subject matter can be used in marker assisted selection protocols to identify and or select progeny with enhanced drought tolerance. Such methods can comprise consist essentially of or consist of crossing a first maize plant or germplasm with a second maize plant or germplasm wherein the first maize plant or germplasm comprises a marker associated with enhanced drought tolerance and selecting a progeny plant that possesses the marker. Either of the first and second maize plants or both can be of a non naturally occurring variety of maize. In some embodiments the first maize plant or germplasm is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the first maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the second maize plant or germplasm is of an elite variety of maize. In some embodiments the genome of the second maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of an elite variety of maize. In some embodiments the second maize plant is of the NP2391 variety. In some embodiments the genome of the second maize plant is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391.

Thus in some embodiments the presently disclosed subject matter provides methods for introgressing an allele associated with enhanced drought tolerance into a genetic background lacking said allele. In some embodiments the methods comprise crossing a donor comprising said allele with a recurrent parent that lacks said allele and repeatedly backcrossing progeny comprising said allele with the recurrent parent wherein said progeny are identified by detecting in their genomes the presence of a haplotype associated with enhanced drought tolerance wherein said haplotype is selected from the group consisting of 

In some embodiments of the presently disclosed methods the genome of said donor is at least 95 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments said donor is selected from the group consisting of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 and Tuxpeno VEN 692. In some embodiments the genome of said recurrent parent plant or germplasm is at least 95 identical to that of an elite variety of maize. In some embodiments said recurrent parent is of an elite variety of maize. In some embodiments 23.

The presently disclosed subject matter relates in some embodiments to stacking of haplotypes associated with water optimization in order to produce plants and parts thereof that have multiple favorable water optimization haplotypes. By way of example and not limitation the presently disclosed subject matter relates in some embodiments to the identification and characterization of loci that are each associated with one or more water optimization traits. These loci correspond to SEQ ID NOs 1 413.

For each of these loci favorable haplotypes have been identified that are associated with water optimization traits. These favorable haplotypes are summarized herein. The presently disclosed subject matter provides exemplary haplotypes that are associated with increases and decreases of various water optimization traits as defined herein. The phrase favorable haplotype refers to a haplotype that when present results in a quantitatively higher water optimization versus the case when an unfavorable haplotype is present. It is noted however then in the case where a lower water optimization is desirable the haplotypes disclosed herein as favorable could be unfavorable haplotypes. As such as used herein favorable is employed in the context of increased water optimization and would be reversed in the context of decreased water optimization.

Methods for identifying a drought tolerant maize plant or germplasm can comprise detecting the presence of a marker associated with enhanced drought tolerance. The marker can be detected in any sample taken from the plant or germplasm including but not limited to the whole plant or germplasm a portion of said plant or germplasm e.g. a cell from said plant or germplasm or a nucleotide sequence from said plant or germplasm. The maize plant can be of a non naturally occurring variety of maize. In some embodiments the genome of the maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of an elite variety of maize. In some embodiments the genome of the maize plant is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391.

In some embodiments the presently disclosed subject matter provides methods for introgressing an allele of interest of a locus associated with a water optimization trait into germplasm. In some embodiments the methods comprise 

 a selecting a plant that comprises an allele of interest of a locus associated with a water optimization trait which allele is defined by at least one marker allele comprising a polymorphic site identifiable by PCR amplification of a nucleic acid with a pair of oligonucleotides primers selected from among primer pair 1 represented by a primer comprising SEQ ID NO 118 and a primer comprising SEQ ID NO 119 primer pair 2 represented by a primer comprising SEQ ID NO 120 and a primer comprising SEQ ID NO 121 primer pair 3 represented by a primer comprising SEQ ID NO 122 and a primer comprising SEQ ID NO 123 primer pair 4 represented by a primer comprising SEQ ID NO 124 and a primer comprising SEQ ID NO 125 primer pair 5 represented by a primer comprising SEQ ID NO 126 and a primer comprising SEQ ID NO 127 primer pair 6 represented by a primer comprising SEQ ID NO 128 and a primer comprising SEQ ID NO 129 primer pair 7 represented by a primer comprising SEQ ID NO 130 and a primer comprising SEQ ID NO 131 primer pair 8 represented by a primer comprising SEQ ID NO 132 and a primer comprising SEQ ID NO 133 primer pair 9 represented by a primer comprising SEQ ID NO 134 and a primer comprising SEQ ID NO 135 primer pair 10 represented by a primer comprising SEQ ID NO 136 and a primer comprising SEQ ID NO 137 primer pair 11 represented by a primer comprising SEQ ID NO 138 and a primer comprising SEQ ID NO 139 primer pair 12 represented by a primer comprising SEQ ID NO 140 and a primer comprising SEQ ID NO 141 primer pair 13 represented by a primer comprising SEQ ID NO 142 and a primer comprising SEQ ID NO 143 primer pair 14 represented by a primer comprising SEQ ID NO 144 and a primer comprising SEQ ID NO 145 primer pair 15 represented by a primer comprising SEQ ID NO 146 and a primer comprising SEQ ID NO 147 primer pair 16 represented by a primer comprising SEQ ID NO 148 and a primer comprising SEQ ID NO 149 primer pair 17 represented by a primer comprising SEQ ID NO 150 and a primer comprising SEQ ID NO 151 primer pair 18 represented by a primer comprising SEQ ID NO 152 and a primer comprising SEQ ID NO 153 primer pair 19 represented by a primer comprising SEQ ID NO 154 and a primer comprising SEQ ID NO 155 primer pair 20 represented by a primer comprising SEQ ID NO 156 and a primer comprising SEQ ID NO 157 primer pair 21 represented by a primer comprising SEQ ID NO 158 and a primer comprising SEQ ID NO 159 primer pair 22 represented by a primer comprising SEQ ID NO 160 and a primer comprising SEQ ID NO 161 primer pair 23 represented by a primer comprising SEQ ID NO 162 and a primer comprising SEQ ID NO 163 primer pair 24 represented by a primer comprising SEQ ID NO 164 and a primer comprising SEQ ID NO 165 primer pair 25 represented by a primer comprising SEQ ID NO 166 and a primer comprising SEQ ID NO 167 primer pair 26 represented by a primer comprising SEQ ID NO 168 and a primer comprising SEQ ID NO 169 primer pair 27 represented by a primer comprising SEQ ID NO 170 and a primer comprising SEQ ID NO 171 primer pair 28 represented by a primer comprising SEQ ID NO 172 and a primer comprising SEQ ID NO 173 primer pair 29 represented by a primer comprising SEQ ID NO 174 and a primer comprising SEQ ID NO 175 primer pair 30 represented by a primer comprising SEQ ID NO 176 and a primer comprising SEQ ID NO 177 primer pair 31 represented by a primer comprising SEQ ID NO 178 and a primer comprising SEQ ID NO 179 primer pair 32 represented by a primer comprising SEQ ID NO 180 and a primer comprising SEQ ID NO 181 primer pair 33 represented by a primer comprising SEQ ID NO 182 and a primer comprising SEQ ID NO 183 primer pair 34 represented by a primer comprising SEQ ID NO 184 and a primer comprising SEQ ID NO 185 primer pair 35 represented by a primer comprising SEQ ID NO 186 and a primer comprising SEQ ID NO 187 primer pair 36 represented by a primer comprising SEQ ID NO 188 and a primer comprising SEQ ID NO 189 primer pair 37 represented by a primer comprising SEQ ID NO 190 and a primer comprising SEQ ID NO 191 primer pair 38 represented by a primer comprising SEQ ID NO 192 and a primer comprising SEQ ID NO 193 primer pair 39 represented by a primer comprising SEQ ID NO 194 and a primer comprising SEQ ID NO 195 primer pair 40 represented by a primer comprising SEQ ID NO 196 and a primer comprising SEQ ID NO 197 primer pair 41 represented by a primer comprising SEQ ID NO 198 and a primer comprising SEQ ID NO 199 primer pair 42 represented by a primer comprising SEQ ID NO 200 and a primer comprising SEQ ID NO 201 primer pair 43 represented by a primer comprising SEQ ID NO 202 and a primer comprising SEQ ID NO 203 primer pair 44 represented by a primer comprising SEQ ID NO 204 and a primer comprising SEQ ID NO 205 primer pair 45 represented by a primer comprising SEQ ID NO 206 and a primer comprising SEQ ID NO 207 primer pair 46 represented by a primer comprising SEQ ID NO 208 and a primer comprising SEQ ID NO 209 primer pair 47 represented by a primer comprising SEQ ID NO 210 and a primer comprising SEQ ID NO 211 primer pair 48 represented by a primer comprising SEQ ID NO 212 and a primer comprising SEQ ID NO 213 primer pair 49 represented by a primer comprising SEQ ID NO 214 and a primer comprising SEQ ID NO 215 primer pair 50 represented by a primer comprising SEQ ID NO 216 and a primer comprising SEQ ID NO 217 primer pair 51 represented by a primer comprising SEQ ID NO 218 and a primer comprising SEQ ID NO 219 primer pair 52 represented by a primer comprising SEQ ID NO 220 and a primer comprising SEQ ID NO 221 primer pair 53 represented by a primer comprising SEQ ID NO 222 and a primer comprising SEQ ID NO 223 primer pair 54 represented by a primer comprising SEQ ID NO 224 and a primer comprising SEQ ID NO 225 primer pair 55 represented by a primer comprising SEQ ID NO 226 and a primer comprising SEQ ID NO 227 primer pair 56 represented by a primer comprising SEQ ID NO 228 and a primer comprising SEQ ID NO 229 and primer pair 57 represented by a primer comprising SEQ ID NO 230 and a primer comprising SEQ ID NO 231 and

 b introgressing the allele of interest into germplasm that lacks the allele. In some embodiments the allele of interest comprises one of SEQ ID NOs 1 117 400 and 401 or a nucleotide sequence that is at least 85 90 or 95 identical thereto over the full length of the one of SEQ ID NOs 1 117 400 and 401. In some embodiments the allele of interest is a favorable allele and or a favorable haplotype that positively correlates with a water optimization trait.

In some embodiments the favorable allele comprises a nucleotide sequence at least 90 identical to one or more of SEQ ID NOs 1 117 400 and 401 and further comprises one or more of the particular nucleotide and position combinations disclosed herein. By way of example and not limitation in some embodiments the favorable allele comprises a nucleotide sequence at least 90 identical to 

SEQ ID NO 1 and further comprises a G nucleotide at the position that corresponds to position 115 of SEQ ID NO 1 an A nucleotide at the position that corresponds to position 270 of SEQ ID NO 1 a T nucleotide at the position that corresponds to position 301 of SEQ ID NO 1 an A nucleotide at the position that corresponds to position 483 of SEQ ID NO 1 or any combination thereof 

SEQ ID NO 2 and further comprises a G nucleotide at the position that corresponds to position 100 and a deletion at the position that corresponds to positions 264 271 of SEQ ID NO 2 or a combination thereof 

SEQ ID NO 3 and further comprises a G nucleotide at the position that corresponds to position 216 of SEQ ID NO 3 

SEQ ID NO 4 and further comprises an A nucleotide at the position that corresponds to position 503 of SEQ ID NO 4 

SEQ ID NO 5 and further comprises a CGCG tetranucleotide at the position that corresponds to positions 818 821 of SEQ ID NO 5 

SEQ ID NO 6 and further comprises a G or an A nucleotide at the position that corresponds to position 254 of SEQ ID NO 6 

SEQ ID NO 7 and further comprises a GA dinucleotide at the position that corresponds to positions 4497 4498 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 a C or a T nucleotide at the position that corresponds to position 4792 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4836 of SEQ ID NO 7 an ACT or a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 or any combination thereof or further comprises a deletion at positions 4497 4498 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 4505 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4609 of SEQ ID NO 7 an A nucleotide at the position that corresponds to position 4641 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4792 of SEQ ID NO 7 a T nucleotide at the position that corresponds to position 4836 of SEQ ID NO 7 a C nucleotide at the position that corresponds to position 4844 of SEQ ID NO 7 a G nucleotide at the position that corresponds to position 4969 of SEQ ID NO 7 and a TCC trinucleotide at the position that corresponds to positions 4979 4981 of SEQ ID NO 7 

SEQ ID NO 8 and further comprises an A nucleotide at the position that corresponds to position 217 of SEQ ID NO 8 and optionally further comprises a G nucleotide at the position that corresponds to position 390 of SEQ ID NO 8 and an A nucleotide at the position that corresponds to position 477 of SEQ ID NO 8 or any combination thereof 

SEQ ID NO 9 and further comprises a C nucleotide at the position that corresponds to position 292 of SEQ ID NO 9 

SEQ ID NO 10 and further comprises an A nucleotide at the position that corresponds to position 166 of SEQ ID NO 10 

SEQ ID NO 11 and further comprises a G nucleotide at the position that corresponds to position 148 of SEQ ID NO 11 

SEQ ID NO 12 and further comprises a C nucleotide at the position that corresponds to position 94 of SEQ ID NO 12 

SEQ ID NO 13 and further comprises an A nucleotide at the position that corresponds to position 35 of SEQ ID NO 13 a C nucleotide at the position that corresponds to position 148 of SEQ ID NO 13 or a G nucleotide at the position that corresponds to position 89 of SEQ ID NO 13 or any combination thereof 

SEQ ID NO 14 and further comprises a G nucleotide at the position that corresponds to position 432 of SEQ ID NO 14 

SEQ ID NO 15 and further comprises an A nucleotide at the position that corresponds to position 753 of SEQ ID NO 15 

SEQ ID NO 16 and further comprises a G nucleotide at the position that corresponds to position 755 of SEQ ID NO 16 

SEQ ID NO 17 and further comprises a G nucleotide at the position that corresponds to position 431 of SEQ ID NO 17 

SEQ ID NO 18 and further comprises a G or a T nucleotide at the position that corresponds to position 518 of SEQ ID NO 18 

SEQ ID NO 19 and further comprises an A nucleotide at the position that corresponds to position 182 of SEQ ID NO 19 an A nucleotide at the position that corresponds to position 309 of SEQ ID NO 19 or a G or a C nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 or any combination thereof or that further comprises a G nucleotide at the position that corresponds to position 182 of SEQ ID NO 19 an A nucleotide at the position that corresponds to position 309 of SEQ ID NO 19 a G nucleotide at the position that corresponds to position 330 of SEQ ID NO 19 and a G nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 

SEQ ID NO 20 and further comprises a CTGG tetranucleotide at the position that corresponds to positions 773 776 of SEQ ID NO 20 

SEQ ID NO 21 and further comprises a deletion of nucleotide at the positions that correspond to positions 316 324 of SEQ ID NO 21 or that further comprises a C nucleotide at the position that corresponds to position 61 of SEQ ID NO 21 a C nucleotide at the position that corresponds to position 200 of SEQ ID NO 21 and a deletion at the positions that correspond to positions 316 324 of SEQ ID NO 21 

SEQ ID NO 22 and further comprises a G nucleotide at the position that corresponds to position 211 of SEQ ID NO 22 

SEQ ID NO 23 and further comprises a G nucleotide at the position that corresponds to position 116 of SEQ ID NO 23 an A nucleotide at the position that corresponds to position 217 of SEQ ID NO 23 or a combination thereof 

SEQ ID NO 24 and further comprises a C nucleotide at the position that corresponds to position 746 of SEQ ID NO 24 

SEQ ID NO 25 and further comprises a G nucleotide at the position that corresponds to position 562 of SEQ ID NO 25 

SEQ ID NO 26 and further comprises a C nucleotide at the position that corresponds to position 1271 of SEQ ID NO 26 

SEQ ID NO 27 and further comprises a C or a T nucleotide at the position that corresponds to position 254 of SEQ ID NO 27 or that optionally further comprises a G nucleotide at the position that corresponds to position 64 of SEQ ID NO 27 and a T nucleotide at the position that corresponds to position 254 of SEQ ID NO 27 

SEQ ID NO 28 and further comprises a T nucleotide at the position that corresponds to position 496 of SEQ ID NO 28 or further comprises a C nucleotide at the position that corresponds to position 98 of SEQ ID NO 28 a T nucleotide at the position that corresponds to position 147 of SEQ ID NO 28 a C nucleotide at the position that corresponds to position 224 of SEQ ID NO 28 and a T nucleotide at the position that corresponds to position 496 of SEQ ID NO 28 

SEQ ID NO 29 and further comprises a C nucleotide at the position that corresponds to position 258 of SEQ ID NO 29 

SEQ ID NO 30 and further comprises a T nucleotide at the position that corresponds to position 259 of SEQ ID NO 30 a G nucleotide at the position that corresponds to position 398 of SEQ ID NO 30 or a combination thereof or that further comprises a T nucleotide at the position that corresponds to position 259 of SEQ ID NO 30 a T nucleotide at the position that corresponds to position 296 of SEQ ID NO 30 an A nucleotide at the position that corresponds to position 398 of SEQ ID NO 30 and a C nucleotide at the position that corresponds to position 1057 of SEQ ID NO 30 

SEQ ID NO 31 and further comprises a G nucleotide at the position that corresponds to position 239 of SEQ ID NO 31 

SEQ ID NO 32 and further comprises a G nucleotide at the position that corresponds to position 208 of SEQ ID NO 32 

SEQ ID NO 33 and further comprises an A nucleotide at the position that corresponds to position 391 of SEQ ID NO 33 

SEQ ID NO 34 and further comprises a CA dinucleotide at the position that corresponds to positions 144 145 of SEQ ID NO 34 a T nucleotide at the position that corresponds to position 169 of SEQ ID NO 34 an A nucleotide at the position that corresponds to position 537 of SEQ ID NO 34 or any combination thereof 

SEQ ID NO 35 and further comprises a G nucleotide at the position that corresponds to position 76 of SEQ ID NO 35 

SEQ ID NO 36 and further comprises a T nucleotide at the position that corresponds to position 698 of SEQ ID NO 36 or that further comprises a C nucleotide at the position that corresponds to position 500 of SEQ ID NO 36 a G nucleotide at the position that corresponds to position 568 of SEQ ID NO 36 and a T nucleotide at the position that corresponds to position 698 of SEQ ID NO 36 

SEQ ID NO 37 and further comprises an A nucleotide at the position that corresponds to position 375 of SEQ ID NO 37 an A or a G nucleotide at the position that corresponds to position 386 of SEQ ID NO 37 or a combination thereof 

SEQ ID NO 38 and further comprises a C nucleotide at the position that corresponds to position 309 of SEQ ID NO 38 an A nucleotide at the position that corresponds to position 342 of SEQ ID NO 38 or a combination thereof 

SEQ ID NO 39 and further comprises a G or a C nucleotide at the position that corresponds to position 445 of SEQ ID NO 39 

SEQ ID NO 40 and further comprises an A nucleotide at the position that corresponds to position 602 of SEQ ID NO 40 

SEQ ID NO 41 and further comprises a G or an A nucleotide at the position that corresponds to position 190 of SEQ ID NO 41 a C nucleotide at the position that corresponds to position 580 of SEQ ID NO 41 or a combination thereof 

SEQ ID NO 42 and further comprises a TTG trinucleotide at the position that corresponds to positions 266 268 of SEQ ID NO 42 or that further comprises an A nucleotide at the position that corresponds to position 238 of SEQ ID NO 42 a deletion of the nucleotides that corresponds to positions 266 268 of SEQ ID NO 42 and a C nucleotide at the position that corresponds to position 808 of SEQ ID NO 42 

SEQ ID NO 43 and further comprises a C or an A nucleotide at the position that corresponds to position 708 of SEQ ID NO 43 

SEQ ID NO 44 and further comprises an A nucleotide at the position that corresponds to position 266 of SEQ ID NO 44 

SEQ ID NO 45 and further comprises a T nucleotide at the position that corresponds to position 475 of SEQ ID NO 45 

SEQ ID NO 46 and further comprises a C nucleotide at the position that corresponds to position 386 of SEQ ID NO 46 

SEQ ID NO 47 and further comprises a G nucleotide at the position that corresponds to position 87 of SEQ ID NO 47 

SEQ ID NO 48 and further comprises an A or a G nucleotide at the position that corresponds to position 472 of SEQ ID NO 48 

SEQ ID NO 49 and further comprises a G nucleotide at the position that corresponds to position 650 of SEQ ID NO 49 and optionally also further comprises a C nucleotide at the position that corresponds to position 166 of SEQ ID NO 49 and A nucleotide at the position that corresponds to position 224 of SEQ ID NO 49 and a G nucleotide at the position that corresponds to position 892 of SEQ ID NO 49 

SEQ ID NO 50 and further comprises a T or an A nucleotide at the position that corresponds to position 541 of SEQ ID NO 50 

SEQ ID NO 51 and further comprises a G nucleotide at the position that corresponds to position 111 of SEQ ID NO 51 

SEQ ID NO 52 and further comprises a C or a G nucleotide at the position that corresponds to position 442 of SEQ ID NO 52 

SEQ ID NO 53 and further comprises a C or a T nucleotide at the position that corresponds to position 428 of SEQ ID NO 53 a C nucleotide at the position that corresponds to position 491 of SEQ ID NO 53 or a combination thereof or that further comprises a C nucleotide at the positions that correspond to at positions 83 428 491 and 548 of SEQ ID NO 53 

SEQ ID NO 54 and further comprises an A nucleotide at the position that corresponds to position 126 of SEQ ID NO 54 

SEQ ID NO 55 and further comprises a G nucleotide at the position that corresponds to position 193 of SEQ ID NO 55 

SEQ ID NO 56 and further comprises and A or a G nucleotide at the position that corresponds to position 237 of SEQ ID NO 56 a C nucleotide at the position that corresponds to position 516 of SEQ ID NO 56 or a combination thereof 

SEQ ID NO 57 and further comprises a T nucleotide at the position that corresponds to position 173 of SEQ ID NO 57 

SEQ ID NO 58 and further comprises a C nucleotide at the position that corresponds to position 486 of SEQ ID NO 58 

SEQ ID NO 59 and further comprises a G nucleotide at the position that corresponds to position 729 of SEQ ID NO 59 and or SEQ ID NO 60 and further comprises a G nucleotide at the position that corresponds to position 267 of SEQ ID NO 60.

Methods for producing a drought tolerant maize plant can comprise detecting in a germplasm a marker associated with enhanced drought tolerance and producing a maize plant from said germplasm. The germplasm can be of a non naturally occurring variety of maize. In some embodiments the genome of the germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of an elite variety of maize. In some embodiments the genome of the germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391.

In some embodiments the alleles comprising the marker associated with enhanced drought tolerance are detected using a plurality of probes selected from the group consisting of 

In some embodiments the alleles comprising the marker associated with enhanced drought tolerance are detected using a plurality of probes selected from the group consisting of 

In some embodiments the allele s comprising the marker associated with enhanced drought tolerance is are detected using a probe or probes selected from the group consisting of 

In some embodiments the alleles comprising the marker associated with enhanced drought tolerance are detected in amplification products from a nucleic acid sample isolated from a maize plant or germplasm wherein the amplification products are produced using pairs of amplification primers selected from the group consisting of 

In some embodiments the alleles comprising the marker associated with enhanced drought tolerance are detected in amplification products from a nucleic acid sample isolated from a maize plant or germplasm wherein the amplification products are produced using pairs of amplification primers selected from the group consisting of 

In some embodiments the allele s comprising the marker associated with enhanced drought tolerance is are detected in an amplification product or products from a nucleic acid sample isolated from a maize plant or germplasm 

wherein the amplification product s is are produced using a pair or pairs of amplification primers selected from the group consisting of 

Methods for introgressing an allele associated with enhanced drought tolerance into a maize plant or germplasm can comprise crossing a first maize plant or germplasm comprising said allele the donor with a second maize plant or germplasm that lacks said allele the recurrent parent and repeatedly backcrossing progeny comprising said allele with the recurrent parent. Progeny comprising said allele can be identified by detecting in their genomes the presence of a marker associated with enhanced drought tolerance. Either the donor or the recurrent parent or both can be of a non naturally occurring variety of maize. In some embodiments the donor is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the donor is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the recurrent parent is of an elite variety of maize. In some embodiments the genome of the recurrent parent is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of an elite variety of maize. In some embodiments the recurrent parent is of the NP2391 variety. In some embodiments the genome of the recurrent parent is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391.

In some embodiments the presently disclosed subject matter also provides methods of producing a drought tolerant maize plant. In some embodiments the presently disclosed methods comprise detecting in a maize germplasm the presence of a marker associated with enhanced drought tolerance wherein said marker is selected from the group consisting of 

The presently disclosed subject matter also provides methods for identifying and or selecting a drought tolerant maize plant or germplasm. Methods for selecting a drought tolerant maize plant or germplasm can comprise crossing a first maize plant or germplasm with a second maize plant or germplasm wherein said first maize plant or germplasm comprises a marker associated with enhanced drought tolerance and selecting a progeny plant or germplasm comprising said marker associated with enhanced drought tolerance. Either the first or second maize plant or germplasm or both can be of a non naturally occurring variety of maize. In some embodiments the first maize plant or germplasm is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the first maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the second maize plant or germplasm is of an elite variety of maize. In some embodiments the genome of the second maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of an elite variety of maize. In some embodiments the second maize plant or germplasm is of the NP2391 variety. In some embodiments the genome of the second maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391.

Thus in some embodiments the methods comprise detecting in said maize plant or germplasm the presence of a marker associated with enhanced drought tolerance wherein said marker comprises a plurality of alleles which are detected in amplification products from a nucleic acid sample isolated from said maize plant or germplasm said amplification products having been produced using pairs of amplification primers selected from the group consisting of 

The presently disclosed subject matter also provides methods for producing a drought tolerant maize plant comprising detecting in a maize germplasm the presence of a marker associated with enhanced drought tolerance wherein said marker comprises a plurality of alleles which are detected in amplification products from a nucleic acid sample isolated from said maize plant or germplasm said amplification product having been produced using pairs of amplification primers selected from the group consisting of 

The presently disclosed subject matter also provides methods for identifying and or selecting a drought tolerant maize plant or germplasm comprising detecting in said maize plant or germplasm the presence of a marker associated with enhanced drought tolerance wherein said marker comprises a plurality of alleles which are detected using a plurality of probes selected from the group consisting of 

The presently disclosed subject matter also provides methods for producing a drought tolerant maize plant comprising detecting in a maize germplasm the presence of a marker associated with enhanced drought tolerance wherein said marker comprises a plurality of alleles which are detected using a plurality of probes selected from the group consisting of 

In some embodiments of the presently disclosed methods the maize plant or germplasm is of a non naturally occurring variety of maize. In some embodiments the genome of said maize plant or germplasm is at least 95 identical to that of NP2391.

The presently disclosed subject matter also provides in some embodiments methods for selecting a drought tolerant maize plant or germplasm. In some embodiments the methods comprise crossing a first maize plant or germplasm with a second maize plant or germplasm wherein said first maize plant or germplasm comprises within its genome a haplotype associated with enhanced drought tolerance wherein said haplotype is selected from the group consisting of 

In some embodiments of the presently disclosed methods the genome of said second maize plant or germplasm is at least 95 identical to that of an elite variety of maize. In some embodiments the second maize plant or germplasm is of an elite variety of maize. In some embodiments the elite variety of maize is NP2391.

In some embodiments the presently disclosed subject matter relates to the use of polymorphisms including but not limited to SNPs or trait conferring parts for producing a trait carrying maize plant by introducing a nucleic acid sequence comprising a trait associated allele and or haplotype of the polymorphism into a recipient plant.

A donor plant with the nucleic acid sequence that comprises a water optimization trait allele and or haplotype can be transferred to the recipient plant lacking the allele and or the haplotype. The nucleic acid sequence can be transferred by crossing a water optimization trait carrying donor plant with a non trait carrying recipient plant e.g. by introgression by transformation by protoplast transformation or fusion by a doubled haploid technique by embryo rescue or by any other nucleic acid transfer system. Then if desired progeny plants comprising one or more of the presently disclosed water optimization trait alleles and or haplotypes can be selected. A nucleic acid sequence comprising an water optimization trait allele and or haplotype can be isolated from the donor plant using methods known in the art and the isolated nucleic acid sequence can transform the recipient plant by transgenic methods. This can occur with a vector in a gamete or other suitable transfer element such as a ballistic particle coated with the nucleic acid sequence.

Plant transformation generally involves the construction of an expression vector that will function in plant cells and includes nucleic acid sequence that comprises an allele and or haplotype associated with the water optimization trait which vector can comprise a water optimization trait conferring gene. This gene usually is controlled or operatively linked to one or more regulatory element such as a promoter. The expression vector can contain one or more such operably linked gene regulatory element combinations provided that at least one of the genes contained in the combinations encodes water optimization trait. The vector s can be in the form of a plasmid and can be used alone or in combination with other plasmids to provide transgenic plants that are better water optimization plants using transformation methods known in the art such as the transformation system.

Transformed cells often contain a selectable marker to allow transformation identification. The selectable marker is typically adapted to be recovered by negative selection by inhibiting the growth of cells that do not contain the selectable marker gene or by positive selection by screening for the product encoded by the selectable marker gene . Many commonly used selectable marker genes for plant transformation are known in the art and include for example genes that code for enzymes that metabolically detoxify a selective chemical agent that can be an antibiotic or a herbicide or genes that encode an altered target which is insensitive to the inhibitor. Several positive selection methods are known in the art such as mannose selection. Alternatively marker less transformation can be used to obtain plants without the aforementioned marker genes the techniques for which are also known in the art.

The presently disclosed subject matter provides drought tolerant maize plants and germplasms. As discussed above the methods of the presently disclosed subject matter can be utilized to identify produce and or select a drought tolerant maize plant or germplasm. In addition to the methods described above a drought tolerant maize plant or germplasm can be produced by any method whereby a marker associated with enhanced drought tolerance is introduced into the maize plant or germplasm including but not limited to transformation protoplast transformation or fusion a double haploid technique embryo rescue or by any other nucleic acid transfer system.

In some embodiments the maize plant or germplasm comprises a non naturally occurring variety of maize. In some embodiments the maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of an elite variety of maize. In some embodiments the genome of said maize plant or germplasm is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391.

The maize plant or germplasm can be the progeny of a cross between an elite variety of maize and a variety of maize that comprises an allele associated with enhanced drought tolerance. In some embodiments the elite variety of maize is NP2391. In some embodiments the genome of the elite variety of maize is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391. In some embodiments the variety comprising an allele associated with enhanced drought tolerance is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the variety comprising an allele associated with enhanced drought tolerance is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692.

The maize plant or germplasm can be the progeny of an introgression wherein the recurrent parent is an elite variety of maize and the donor comprises an allele associated with enhanced drought tolerance. In some embodiments the recurrent parent is NP2391. In some embodiments the genome of the recurrent parent is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391. In some embodiments the donor is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the donor is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692.

The maize plant or germplasm can be the progeny of a cross between a first elite variety of maize e.g. a tester line and the progeny of a cross between a second elite variety of maize e.g. a recurrent parent and a variety of maize that comprises an allele associated with enhanced drought tolerance e.g. a donor . In some embodiments the first elite variety of maize is NP2460. In some embodiments the genome of the first elite variety of maize is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2460. In some embodiments the second elite variety of maize is NP2391. In some embodiments the genome of the second elite variety of maize is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391. In some embodiments the variety comprising an allele associated with enhanced drought tolerance is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the variety comprising an allele associated with enhanced drought tolerance is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692.

The maize plant or germplasm can be the progeny of a cross between a first elite variety of maize and the progeny of an introgression wherein the recurrent parent is a second elite variety of maize and the donor comprises an allele associated with enhanced drought tolerance. In some embodiments the first elite variety of maize is NP2460. In some embodiments the genome of the first elite variety of maize is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2460. In some embodiments the recurrent parent is NP2391. In some embodiments the genome of the recurrent parent is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of NP2391. In some embodiments the donor is CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments the genome of the donor is at least about 50 55 60 65 70 75 80 85 90 95 97 99 or 100 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692.

Thus the presently disclosed subject matter provides in some embodiments inbred plants comprising one or more alleles associated with a desired water optimization trait. In some embodiments 

 i the water optimization trait is grain yield at standard moisture percentage YGSMN and the favorable allele comprises a nucleotide sequence comprising an A at nucleotide position 270 of SEQ ID NO 1 a G at nucleotide position 216 of SEQ ID NO 3 an A at nucleotide position 503 of SEQ ID NO 4 a CGCG tetranucleotide at nucleotide positions 818 821 of SEQ ID NO 5 a G at nucleotide position 254 of SEQ ID NO 6 a GA dinucleotide at nucleotide positions 4497 4498 of SEQ ID NO 7 an A at nucleotide position 4641 of SEQ ID NO 7 a C or a T at nucleotide position 4792 of SEQ ID NO 7 a T at nucleotide position 4836 of SEQ ID NO 7 an ACT or a TCC trinucleotide at nucleotide positions 4979 4981 of SEQ ID NO 7 a C at nucleotide position 292 of SEQ ID NO 9 an A at nucleotide position 166 of SEQ ID NO 10 a C at nucleotide position 94 of SEQ ID NO 12 a C at nucleotide position 86 of SEQ ID NO 13 a G at nucleotide position 89 of SEQ ID NO 13 an A at nucleotide position 753 of SEQ ID NO 15 a G at nucleotide position 755 of SEQ ID NO 16 a G at nucleotide position 431 of SEQ ID NO 17 an A at nucleotide position 309 of SEQ ID NO 19 a CTGG tetranucleotide at nucleotide positions 773 776 of SEQ ID NO 20 a deletion of nucleotide positions 316 324 of SEQ ID NO 21 a G at nucleotide position 562 of SEQ ID NO 25 a T at nucleotide position 254 of SEQ ID NO 27 a T at nucleotide position 496 of SEQ ID NO 28 a G at nucleotide position 398 of SEQ ID NO 30 a G at nucleotide position 239 of SEQ ID NO 31 a G at nucleotide position 208 of SEQ ID NO 32 a CA dinucleotide at the position that corresponds to positions 144 145 of SEQ ID NO 34 a T nucleotide at the position that corresponds to position 169 of SEQ ID NO 34 a G nucleotide at the position that corresponds to position 76 of SEQ ID NO 35 a T nucleotide at the position that corresponds to position 698 of SEQ ID NO 36 an A or a G nucleotide at the position that corresponds to position 386 of SEQ ID NO 37 a G nucleotide at the position that corresponds to position 445 of SEQ ID NO 39 an A nucleotide at the position that corresponds to position 602 of SEQ ID NO 40 a G nucleotide at the position that corresponds to position 190 of SEQ ID NO 41 a C nucleotide at the position that corresponds to position 580 of SEQ ID NO 41 a TTG trinucleotide at the position that corresponds to positions 266 268 of SEQ ID NO 42 an A nucleotide at the position that corresponds to position 708 of SEQ ID NO 43 a G nucleotide at the position that corresponds to position 650 of SEQ ID NO 49 an A or a T nucleotide at the position that corresponds to position 541 of SEQ ID NO 50 a T nucleotide at the position that corresponds to position 428 of SEQ ID NO 53 a C nucleotide at the position that corresponds to position 491 of SEQ ID NO 53 and or an A nucleotide at the position that corresponds to position 126 of SEQ ID NO 54 and or

 ii the water optimization trait is grain moisture at harvest GMSTP and the favorable allele comprises a nucleotide sequence comprising an A nucleotide at the position that corresponds to position 254 of SEQ ID NO 6 an A nucleotide at the position that corresponds to position 217 of SEQ ID NO 8 a C nucleotide at the position that corresponds to position 292 of SEQ ID NO 9 an A nucleotide at the position that corresponds to position 166 of SEQ ID NO 10 a G nucleotide at the position that corresponds to position 148 of SEQ ID NO 11 an A nucleotide at the position that corresponds to position 35 of SEQ ID NO 13 a G nucleotide at the position that corresponds to position 432 of SEQ ID NO 14 a G nucleotide at the position that corresponds to position 518 of SEQ ID NO 18 an A nucleotide at the position that corresponds to position 182 of SEQ ID NO 19 a C nucleotide at the position that corresponds to position 463 of SEQ ID NO 19 a CTGG tetranucleotide at the position that corresponds to positions 773 776 of SEQ ID NO 20 a G nucleotide at the position that corresponds to position 211 of SEQ ID NO 22 a G nucleotide at the position that corresponds to position 562 of SEQ ID NO 25 a C nucleotide at the position that corresponds to position 254 of SEQ ID NO 27 a G nucleotide at the position that corresponds to position 239 of SEQ ID NO 31 a CA dinucleotide at the position that corresponds to positions 144 145 of SEQ ID NO 34 an A nucleotide at the position that corresponds to position 537 of SEQ ID NO 34 an A nucleotide at the position that corresponds to position 386 of SEQ ID NO 37 a C nucleotide at the position that corresponds to position 309 of SEQ ID NO 38 an A nucleotide at the position that corresponds to position 342 of SEQ ID NO 38 a C nucleotide at the position that corresponds to position 445 of SEQ ID NO 39 an A nucleotide at the position that corresponds to position 190 of SEQ ID NO 41 a C nucleotide at the position that corresponds to position 708 of SEQ ID NO 43 a G nucleotide at the position that corresponds to position 650 of SEQ ID NO 49 a C nucleotide at the position that corresponds to position 428 of SEQ ID NO 53 or a C nucleotide at the position that corresponds to position 491 of SEQ ID NO 53 and or

 iii the water optimization trait is grain weight per plot GWTPN and the favorable allele comprises a nucleotide sequence comprising a T at nucleotide position 518 of SEQ ID NO 18.

The presently disclosed subject matter also provides in some embodiments plants comprising at least one favorable allele contributing to water optimization which allele is defined by at least one marker allele comprising a polymorphic site and characterized by a PCR amplification product obtainable in a PCR reaction using a PCR oligonucleotide primer or a plurality of oligonucleotide primers particularly a pair of PCR oligonucleotide primers or a plurality of primer pairs but especially a primer pair selected from the group consisting of primer pair 1 represented by a primer comprising SEQ ID NO 118 and a primer comprising SEQ ID NO 119 primer pair 2 represented by a primer comprising SEQ ID NO 120 and a primer comprising SEQ ID NO 121 primer pair 3 represented by a primer comprising SEQ ID NO 122 and a primer comprising SEQ ID NO 123 primer pair 4 represented by a primer comprising SEQ ID NO 124 and a primer comprising SEQ ID NO 125 primer pair 5 represented by a primer comprising SEQ ID NO 126 and a primer comprising SEQ ID NO 127 primer pair 6 represented by a primer comprising SEQ ID NO 128 and a primer comprising SEQ ID NO 129 primer pair 7 represented by a primer comprising SEQ ID NO 130 and a primer comprising SEQ ID NO 131 primer pair 8 represented by a primer comprising SEQ ID NO 132 and a primer comprising SEQ ID NO 133 primer pair 9 represented by a primer comprising SEQ ID NO 134 and a primer comprising SEQ ID NO 135 primer pair 10 represented by a primer comprising SEQ ID NO 136 and a primer comprising SEQ ID NO 137 primer pair 11 represented by a primer comprising SEQ ID NO 138 and a primer comprising SEQ ID NO 139 primer pair 12 represented by a primer comprising SEQ ID NO 140 and a primer comprising SEQ ID NO 141 primer pair 13 represented by a primer comprising SEQ ID NO 142 and a primer comprising SEQ ID NO 143 primer pair 14 represented by a primer comprising SEQ ID NO 144 and a primer comprising SEQ ID NO 145 primer pair 15 represented by a primer comprising SEQ ID NO 146 and a primer comprising SEQ ID NO 147 primer pair 16 represented by a primer comprising SEQ ID NO 148 and a primer comprising SEQ ID NO 149 primer pair 17 represented by a primer comprising SEQ ID NO 150 and a primer comprising SEQ ID NO 151 primer pair 18 represented by a primer comprising SEQ ID NO 152 and a primer comprising SEQ ID NO 153 primer pair 19 represented by a primer comprising SEQ ID NO 154 and a primer comprising SEQ ID NO 155 primer pair 20 represented by a primer comprising SEQ ID NO 156 and a primer comprising SEQ ID NO 157 primer pair 21 represented by a primer comprising SEQ ID NO 158 and a primer comprising SEQ ID NO 159 primer pair 22 represented by a primer comprising SEQ ID NO 160 and a primer comprising SEQ ID NO 161 primer pair 23 represented by a primer comprising SEQ ID NO 162 and a primer comprising SEQ ID NO 163 primer pair 24 represented by a primer comprising SEQ ID NO 164 and a primer comprising SEQ ID NO 165 primer pair 25 represented by a primer comprising SEQ ID NO 166 and a primer comprising SEQ ID NO 167 primer pair 26 represented by a primer comprising SEQ ID NO 168 and a primer comprising SEQ ID NO 169 primer pair 27 represented by a primer comprising SEQ ID NO 170 and a primer comprising SEQ ID NO 171 primer pair 28 represented by a primer comprising SEQ ID NO 172 and a primer comprising SEQ ID NO 173 primer pair 29 represented by a primer comprising SEQ ID NO 174 and a primer comprising SEQ ID NO 175 primer pair 30 represented by a primer comprising SEQ ID NO 176 and a primer comprising SEQ ID NO 177 primer pair 31 represented by a primer comprising SEQ ID NO 178 and a primer comprising SEQ ID NO 179 primer pair 32 represented by a primer comprising SEQ ID NO 180 and a primer comprising SEQ ID NO 181 primer pair 33 represented by a primer comprising SEQ ID NO 182 and a primer comprising SEQ ID NO 183 primer pair 34 represented by a primer comprising SEQ ID NO 184 and a primer comprising SEQ ID NO 185 primer pair 35 represented by a primer comprising SEQ ID NO 186 and a primer comprising SEQ ID NO 187 primer pair 36 represented by a primer comprising SEQ ID NO 188 and a primer comprising SEQ ID NO 189 primer pair 37 represented by a primer comprising SEQ ID NO 190 and a primer comprising SEQ ID NO 191 primer pair 38 represented by a primer comprising SEQ ID NO 192 and a primer comprising SEQ ID NO 193 primer pair 39 represented by a primer comprising SEQ ID NO 194 and a primer comprising SEQ ID NO 195 primer pair 40 represented by a primer comprising SEQ ID NO 196 and a primer comprising SEQ ID NO 197 primer pair 41 represented by a primer comprising SEQ ID NO 198 and a primer comprising SEQ ID NO 199 primer pair 42 represented by a primer comprising SEQ ID NO 200 and a primer comprising SEQ ID NO 201 primer pair 43 represented by a primer comprising SEQ ID NO 202 and a primer comprising SEQ ID NO 203 primer pair 44 represented by a primer comprising SEQ ID NO 204 and a primer comprising SEQ ID NO 205 primer pair 45 represented by a primer comprising SEQ ID NO 206 and a primer comprising SEQ ID NO 207 primer pair 46 represented by a primer comprising SEQ ID NO 208 and a primer comprising SEQ ID NO 209 primer pair 47 represented by a primer comprising SEQ ID NO 210 and a primer comprising SEQ ID NO 211 primer pair 48 represented by a primer comprising SEQ ID NO 212 and a primer comprising SEQ ID NO 213 primer pair 49 represented by a primer comprising SEQ ID NO 214 and a primer comprising SEQ ID NO 215 primer pair 50 represented by a primer comprising SEQ ID NO 216 and a primer comprising SEQ ID NO 217 primer pair 51 represented by a primer comprising SEQ ID NO 218 and a primer comprising SEQ ID NO 219 primer pair 52 represented by a primer comprising SEQ ID NO 220 and a primer comprising SEQ ID NO 221 primer pair 53 represented by a primer comprising SEQ ID NO 222 and a primer comprising SEQ ID NO 223 primer pair 54 represented by a primer comprising SEQ ID NO 224 and a primer comprising SEQ ID NO 225 primer pair 55 represented by a primer comprising SEQ ID NO 226 and a primer comprising SEQ ID NO 227 primer pair 56 represented by a primer comprising SEQ ID NO 228 and a primer comprising SEQ ID NO 229 and primer pair 57 represented by a primer comprising SEQ ID NO 230 and a primer comprising SEQ ID NO 231.

In some embodiments the presently disclosed subject matter provides a non naturally occurring maize plant or germplasm having in its genome a haplotype associated with enhanced drought tolerance. In some embodiments wherein said haplotype is selected from the group consisting of 

In some embodiments the genome of said maize plant or germplasm is at least 95 identical to that of an elite variety of maize. In some embodiments the elite variety of maize is NP2391. In some embodiments the maize plant or germplasm is derived from crossing an elite variety of maize with an exotic variety of maize. In some embodiments the genome of said elite variety of maize is at least 95 identical to that of NP2391. In some embodiments the elite variety of maize is NP2391.

In some embodiments the genome of said exotic variety of maize is at least 95 identical to that of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments 32. In some embodiments said maize plant or germplasm is derived from crossing a first elite variety of maize with the progeny of a cross between a second elite variety of maize and an exotic variety of maize.

In some embodiments the genome of said first elite variety of maize is at least 95 identical to that of NP2460. In some embodiments the first elite variety of maize is NP2460.

In some embodiments the genome of said second elite variety of maize is at least 95 identical to that of NP2391. In some embodiments the second elite variety of maize is NP2391.

In some embodiments the genome of said exotic variety of maize is at least 95 identical to CML333 CML322 Cateto SP VII Confite Morocho AYA 38 or Tuxpeno VEN 692. In some embodiments said exotic variety of maize is selected from the group consisting of CML333 CML322 Cateto SP VII Confite Morocho AYA 38 and Tuxpeno VEN 692.

The presently disclosed subject matter also provides in some embodiments grain and or kernels produced by from a maize plant described herein.

In some embodiments the presently disclosed subject matter also provides pairs of primers consisting of a forward primer and a reverse primer which primers are capable of amplifying in a PCR reaction a fragment of the marker allele which is genetically linked to or identical with the favorable allele contributing to a water optimization phenotype wherein said marker allele comprises a polymorphism which polymorphism is diagnostic for the favorable allele. In some embodiments the primer pairs are selected from the group consisting of primer pair 1 represented by a primer comprising SEQ ID NO 118 and a primer comprising SEQ ID NO 119 primer pair 2 represented by a primer comprising SEQ ID NO 120 and a primer comprising SEQ ID NO 121 primer pair 3 represented by a primer comprising SEQ ID NO 122 and a primer comprising SEQ ID NO 123 primer pair 4 represented by a primer comprising SEQ ID NO 124 and a primer comprising SEQ ID NO 125 primer pair 5 represented by a primer comprising SEQ ID NO 126 and a primer comprising SEQ ID NO 127 primer pair 6 represented by a primer comprising SEQ ID NO 128 and a primer comprising SEQ ID NO 129 primer pair 7 represented by a primer comprising SEQ ID NO 130 and a primer comprising SEQ ID NO 131 primer pair 8 represented by a primer comprising SEQ ID NO 132 and a primer comprising SEQ ID NO 133 primer pair 9 represented by a primer comprising SEQ ID NO 134 and a primer comprising SEQ ID NO 135 primer pair 10 represented by a primer comprising SEQ ID NO 136 and a primer comprising SEQ ID NO 137 primer pair 11 represented by a primer comprising SEQ ID NO 138 and a primer comprising SEQ ID NO 139 primer pair 12 represented by a primer comprising SEQ ID NO 140 and a primer comprising SEQ ID NO 141 primer pair 13 represented by a primer comprising SEQ ID NO 142 and a primer comprising SEQ ID NO 143 primer pair 14 represented by a primer comprising SEQ ID NO 144 and a primer comprising SEQ ID NO 145 primer pair 15 represented by a primer comprising SEQ ID NO 146 and a primer comprising SEQ ID NO 147 primer pair 16 represented by a primer comprising SEQ ID NO 148 and a primer comprising SEQ ID NO 149 primer pair 17 represented by a primer comprising SEQ ID NO 150 and a primer comprising SEQ ID NO 151 primer pair 18 represented by a primer comprising SEQ ID NO 152 and a primer comprising SEQ ID NO 153 primer pair 19 represented by a primer comprising SEQ ID NO 154 and a primer comprising SEQ ID NO 155 primer pair 20 represented by a primer comprising SEQ ID NO 156 and a primer comprising SEQ ID NO 157 primer pair 21 represented by a primer comprising SEQ ID NO 158 and a primer comprising SEQ ID NO 159 primer pair 22 represented by a primer comprising SEQ ID NO 160 and a primer comprising SEQ ID NO 161 primer pair 23 represented by a primer comprising SEQ ID NO 162 and a primer comprising SEQ ID NO 163 primer pair 24 represented by a primer comprising SEQ ID NO 164 and a primer comprising SEQ ID NO 165 primer pair 25 represented by a primer comprising SEQ ID NO 166 and a primer comprising SEQ ID NO 167 primer pair 26 represented by a primer comprising SEQ ID NO 168 and a primer comprising SEQ ID NO 169 primer pair 27 represented by a primer comprising SEQ ID NO 170 and a primer comprising SEQ ID NO 171 primer pair 28 represented by a primer comprising SEQ ID NO 172 and a primer comprising SEQ ID NO 173 primer pair 29 represented by a primer comprising SEQ ID NO 174 and a primer comprising SEQ ID NO 175 primer pair 30 represented by a primer comprising SEQ ID NO 176 and a primer comprising SEQ ID NO 177 primer pair 31 represented by a primer comprising SEQ ID NO 178 and a primer comprising SEQ ID NO 179 primer pair 32 represented by a primer comprising SEQ ID NO 180 and a primer comprising SEQ ID NO 181 primer pair 33 represented by a primer comprising SEQ ID NO 182 and a primer comprising SEQ ID NO 183 primer pair 34 represented by a primer comprising SEQ ID NO 184 and a primer comprising SEQ ID NO 185 primer pair 35 represented by a primer comprising SEQ ID NO 186 and a primer comprising SEQ ID NO 187 primer pair 36 represented by a primer comprising SEQ ID NO 188 and a primer comprising SEQ ID NO 189 primer pair 37 represented by a primer comprising SEQ ID NO 190 and a primer comprising SEQ ID NO 191 primer pair 38 represented by a primer comprising SEQ ID NO 192 and a primer comprising SEQ ID NO 193 primer pair 39 represented by a primer comprising SEQ ID NO 194 and a primer comprising SEQ ID NO 195 primer pair 40 represented by a primer comprising SEQ ID NO 196 and a primer comprising SEQ ID NO 197 primer pair 41 represented by a primer comprising SEQ ID NO 198 and a primer comprising SEQ ID NO 199 primer pair 42 represented by a primer comprising SEQ ID NO 200 and a primer comprising SEQ ID NO 201 primer pair 43 represented by a primer comprising SEQ ID NO 202 and a primer comprising SEQ ID NO 203 primer pair 44 represented by a primer comprising SEQ ID NO 204 and a primer comprising SEQ ID NO 205 primer pair 45 represented by a primer comprising SEQ ID NO 206 and a primer comprising SEQ ID NO 207 primer pair 46 represented by a primer comprising SEQ ID NO 208 and a primer comprising SEQ ID NO 209 primer pair 47 represented by a primer comprising SEQ ID NO 210 and a primer comprising SEQ ID NO 211 primer pair 48 represented by a primer comprising SEQ ID NO 212 and a primer comprising SEQ ID NO 213 primer pair 49 represented by a primer comprising SEQ ID NO 214 and a primer comprising SEQ ID NO 215 primer pair 50 represented by a primer comprising SEQ ID NO 216 and a primer comprising SEQ ID NO 217 primer pair 51 represented by a primer comprising SEQ ID NO 218 and a primer comprising SEQ ID NO 219 primer pair 52 represented by a primer comprising SEQ ID NO 220 and a primer comprising SEQ ID NO 221 primer pair 53 represented by a primer comprising SEQ ID NO 222 and a primer comprising SEQ ID NO 223 primer pair 54 represented by a primer comprising SEQ ID NO 224 and a primer comprising SEQ ID NO 225 primer pair 55 represented by a primer comprising SEQ ID NO 226 and a primer comprising SEQ ID NO 227 primer pair 56 represented by a primer comprising SEQ ID NO 228 and a primer comprising SEQ ID NO 229 and primer pair 57 represented by a primer comprising SEQ ID NO 230 and a primer comprising SEQ ID NO 231.

The following Examples provide illustrative embodiments. In light of the present disclosure and the general level of skill in the art those of skill will appreciate that the following Examples are intended to be exemplary only and that numerous changes modifications and alterations can be employed without departing from the scope of the presently disclosed subject matter.

To assess the value of alleles under drought stress diverse germplasm was screened in controlled field experiments comprising a full irrigation control treatment and a limited irrigation treatment. The goal of the full irrigation treatment is to ensure water does not limit the productivity of the crop. In contrast the goal of the limited irrigation treatment is to ensure that water becomes the major limiting constraint to grain yield. Main effects e.g. treatment and genotype and interactions e.g. genotype treatment can be determined when the two treatments are applied adjacent to one another in the field. Moreover drought related phenotypes can be quantified for each genotype in the panel thereby allowing for marker trait associations to be conducted.

In practice the method for the limited irrigation treatment can vary widely depending upon the germplasm being screened the soil type climatic conditions at the site pre season water supply and in season water supply to name just a few. Initially a site is identified where in season precipitation is low to minimize the chance of unintended water application and is suitable for cropping. In addition determining the timing of the stress can be important such that a target is defined to ensure that year to year or location to location screening consistency is in place. An understanding of the treatment intensity or in some cases the yield loss desired from the limited irrigation treatment can also be considered. Selection of a treatment intensity that is too light can fail to reveal genotypic variation. Selection of a treatment intensity that is too heavy can create large experimental error. Once the timing of stress is identified and treatment intensity is described irrigation can be managed in a manner that is consistent with these targets.

General methods for assessing and assessing drought tolerance can be found in Salekdeh et al. 2009 and in U.S. Pat. Nos. 6 635 803 7 314 757 7 332 651 and 7 432 416.

In order to identify alleles that were associated with water optimization hybrids were grown in different stages at multiple locations and evaluated for water optimization. In this analysis four traits were tested in stage 2 3 YGSMN grain yield at standard moisture GMSTP grain moisture at harvest GWTPN grain weight per plot and PYREC percentage yield recovery . The distribution of the phenotypic data of hybrids of the lines across locations and testers for YGSMN GMSTP GWTPN and PYREC was determined. The mean values for YGSMN GMSTP and GWTPN were 165.41 bushels acre 18.94 and 20.0 bushels plot respectively. The phenotypic data for the selected trials included information from 4 locations. The number of observations in these locations ranged from 311 to 1456. A total of 575 inbreds were evaluated in crosses with up to 47 different inbred testers. The number of observations for inbred lines crossed to a particular tester ranged from 242 to 575 across all locations.

The testing for associations between potential markers and these three traits employed two analytical approaches a Mixed Linear Models TASSEL and a Quantitative Inbred Pedigree Disequilibrium Test referred to herein as QIPDT2 .

The use of stage 2 3 data for association mapping is not a traditional approach and there are several aspects of its analysis that needed to be considered. Moreover hybrids with various testers instead of the lines per se were employed for phenotyping while both of the statistical approaches TASSEL and QIPDT2 were designed for data on inbred lines which require a unique trait value for each line. To obtain a unique trait value for each inbred line that could be compared against its genotype it was necessary to make phenotypic adjustments that help to control the effect of tester and or location. Additional factors e.g. maturity group were not considered to avoid the further reduction of degrees of freedom or subsets sample sizes.

To do the phenotypic adjustments mixed linear model analyses were performed in two different statistical packages SAS JMP and R which were intended to ensure that the mixed model approaches for the large data set were implemented correctly. Since approaches gave very close results the SAS JMP results were used for the downstream data analysis.

The full model analysis included effects of both locations and testers in the model as follows Phenotype Location effect random Line effect random Tester effect fixed error term

The by Location model was used for each of the 4 selected locations as follows Phenotype Line effect random Tester effect fixed error term

The by Tester model was used for each of the 4 selected subsets of lines crossed to a particular tester as follows Phenotype Location effect random Line effect random error term

The models were evaluated for convergence estimation of covariance estimates significance of fixed effects etc. Best linear unbiased predictors BLUPs for line effects were used as adjusted genotypes. In some cases the proposed mixed models did not converge or there was a problem with the estimation of line effects due to the lack of replications. For each such case the effect of the line was removed from the model and the residuals were used as a rough method to capture line effects additional replication was obtained later in the association analysis where each biallelic locus was represented by the total number of inbred lines of each group .

A total of 2189 lines for which phenotypic data was collected in any of the selected trials were also genotyped. A total of 95 polymorphisms corresponding to about 57 candidate genes were scored in the inbred lines. After eliminating monomorphic assays and SNPs with allele frequencies less than 0.01 85 candidate polymorphisms were tested for association in TASSEL. Besides 153 random polymorphisms were genotyped in the inbred lines. After filtering 149 random polymorphisms were also analyzed for association in TASSEL as anonymous candidates.

Association mapping often referred as linkage disequilibrium mapping has become a powerful tool to unveil the genetic control of complex traits. Association mapping relies on the large number of generations and therefore recombination opportunities in the history of a species that allow the removal of association between a QTL and any marker not tightly linked to it Jannink Walsh 2001 . One of the most important steps in association mapping analysis is the control for population structure. Population structure can cause spurious correlations between markers and phenotypes increasing the false positive rate.

The method implemented in TASSEL uses a kinship matrix in the mixed model approach for controlling genetic correlations among lines. Kinship analysis was done using genotypic data on the 153 random SNP assays. A method to estimate kinship relationships based on Zhao et al. 2007 was adopted. Scripts were created to calculate Kinship coefficients that were defined simply as the proportion of shared alleles for each pair of individuals K pShared . Zhao et al. used the proportion of shared haplotypes as their kinship coefficients. The matrix of K coefficients was included for some association models in TASSEL to assess the control for spurious associations due to close interrelatedness of the lines in the panel.

The K matrix was calculated for a set of inbred lines. The kinship coefficient kij was calculated as proportion of shared alleles for all loci between two lines i and j and kij kji kii 1.

Analysis with the software program Structure Pritchard et al. 2000 was done using genotypic data of the 153 random SNP assays.

A linkage model that incorporated population admixture and linkage between the markers was employed. The likelihoods of population structures ranging from k 1 to 15 subpopulations were determined using a burnin period of 50 000 followed by 50 000 MCMC reps. Four replications were run for each value of k. The estimated log probability of data Pr X K for each value of k was plotted to choose an appropriate number of subpopulations to include in the covariance matrix.

The probability for a determinate k increased along with the number of k tested. k 10 was used as the number of subpopulations for association analysis. The inferred ancestry table containing the fraction of each subpopulation contributing to the ancestry of each inbred was used as a series of covariates in the association testing model.

Principal Component Analysis PCA or Eigen analysis was used as an alternative to Structure for inferring population structure from genotypic data. PCA has some advantages over Structure such as the ability to handle large datasets in much shorter periods of time and avoiding the need of selecting a specific number of sub populations. PCA was performed using the software SMARTPCA that is part of Eigenstrat Price et al. 2006 . Ten Eigenvectors and their corresponding Eigenvalues for each of the lines were used as another covariate series for the association models of TASSEL.

The different models employed in TASSEL are shown in the Table 6. For the YGSMN and GMSTP phenotypes adjusted across locations and testers the six 6 models were run and compared. Only Model 4 was run for all the sub sets by location and by tester.

The GLM procedure in TASSEL employed an option to perform permutations to find out the experiment wise error rate that corrected for accumulation of false positives when doing multiple comparisons. A total of 10 000 permutations were used for the water optimization data. The MLM procedure did not include correction for multiple testing. The Bonferroni correction was used a posteriori to avoid accumulation of false positives.

QIPDT2 Quantitative Inbred Pedigree Disequilibrium Test 2 was used for association mapping that takes advantage of inbred pedigree information which can give higher statistical power and lower false positive rates with a better control of population structure issue Stich et al. 2006 TAG 113 1121 1130 . This is an extension of QIPDT originally developed for mapping human disease genes Zhang et al. 2001. Genetic Epidemiol 21 370 375 see reference in Stich et al. 2006 . An advantage of QIPDT2 is that this method can be more easily applied to materials from early breeding stages e.g. stage 2 and 3 because phenotypic data on these materials have been collected for breeding purposes. Generally speaking the materials from early breeding stages are similar to the lines in the well known nested association populations NAM which was designed to use both linkage and linkage disequilibrium for mapping QTL.

Thus the general QIPDT approach was improved using a regression model which is referred to herein as QIPDT2 the original method is then called QIPDT1. The model for QIPDT2 can be written as where yis adjusted phenotypic value for individual i in pedigree k xis coded marker genotypic value is intercept is regression coefficient or genetic effect of the SNP in question. Note that the methods for adjusting phenotypic values and coding marker genotypes are the same as used by Stich et al. 2006. With this model both the genetic effect and Rfor each SNP can be estimated. It is important to note that the phenotypic data were pre adjusted for excluding effects from testers and or locations before being further adjusted for pedigree structure this adjustment was necessary to implement the complex model in QIPDT2. The methods for pre adjustment were the same as described previously for the TASSEL analysis.

Association results from both QIPDT1 and QIPDT2 for the whole data set and split subsets for locations and testers were generated. Like the analysis with TASSEL the phenotypic data were adjusted for locations and or testers depending on which subset was used. This resulted in one adjusted phenotypic value either BLUP line values or model residuals for each inbred which contains a combination of all genetic effects for the inbred and random residual only.

Before QIPDT analysis all inbreds were grouped into different nuclear families according to their parental lines. The use of nuclear families was expected to give better control of population structure than extended pedigrees that were used in Stich et al. 2006. For QIPDT1 a test statistic Z value and corresponding p value were estimated for each SNP for QIPDT2 a test statistic T value and corresponding p value were derived from the simple regression model along with R square for each SNP. QIPDT2 was more powerful than QIPDT1 in terms of p values. Since QIPDT2 also gave estimates R for relative contribution for each SNP QIPDT2 was used for reporting association results from the QIPDT approach.

P values and contributions that each favorable allele was observed to have on the water optimization phenotypes YGSMN GMSTP and GWTPN were calculated. These values are summarized in Tables 7 9. In Tables 7 9 the term contribution refers to the contribution that the favorable allele was calculated to have with respect to the phenotype observed in view of the mean values of 201.68 bushels acre 18.95 and 25.29 bushels plot for YGSMN GMSTP and GWTPN respectively. In Tables 7 9 the contribution is expressed in bushels acre percent and bushels plot for YGSMN GMSTP and GWTPN respectively.

NP2391 is an elite non Stiff Stalk variety of maize. NP2391 is described in U.S. Pat. No. 7 166 783. NP2391 comprises a G allele at position 87 of SEQ ID NO 47 a G allele at position 386 of SEQ ID NO 46 a G allele at positions 4979 4981 of SEQ ID NO 7 a C allele at position 4641 of SEQ ID NO 7 an A allele at position 472 of SEQ ID NO 48 a G allele at position 237 of SEQ ID NO 56 an A allele at position 516 of SEQ ID NO 56 an A allele at position 266 of SEQ ID NO 44 a T allele at position 475 of SEQ ID NO 45 a T allele at position 173 of SEQ ID NO 57 a C allele at position 746 of SEQ ID NO 24 an A allele at position 391 of SEQ ID NO 33 a C allele at position 258 of SEQ ID NO 29 an A allele at position 217 of SEQ ID NO 23 a G allele at position 116 of SEQ ID NO 23 a G allele at position 463 of SEQ ID NO 19 a T allele at position 309 of SEQ ID NO 19 a D allele at positions 264 271 of SEQ ID NO 2 an G allele at position 100 of SEQ ID NO 2 a C allele at position 486 of SEQ ID NO 58 a G allele at position 111 of SEQ ID NO 51 a G allele at position 254 of SEQ ID NO 27 a G allele at position 729 of SEQ ID NO 59 a G allele at position 267 of SEQ ID NO 60 a G allele at position 562 of SEQ ID NO 25 a C allele at position 1271 of SEQ ID NO 26 and a G allele at position 193 of SEQ ID NO 55. See .

NP2460 is an elite Stiff Stalk variety of maize. NP2460 is described in U.S. Pat. No. 7 122 726. NP2460 comprises a C allele at position 386 of SEQ ID NO 46 an A allele at positions 4979 4981 of SEQ ID NO 7 an A allele at position 4641 of SEQ ID NO 7 a G allele at position 472 of SEQ ID NO 48 a G allele at position 237 of SEQ ID NO 56 a C allele at position 516 of SEQ ID NO 56 a C allele at position 266 of SEQ ID NO 44 a C allele at position 475 of SEQ ID NO 45 a G allele at position 173 of SEQ ID NO 57 a C allele at position 746 of SEQ ID NO 24 an A allele at position 391 of SEQ ID NO 33 a T allele at position 258 of SEQ ID NO 29 an A allele at position 217 of SEQ ID NO 23 a G allele at position 116 of SEQ ID NO 23 a C allele at position 463 of SEQ ID NO 19 a C allele at position 309 of SEQ ID NO 19 a D allele at positions 264 271 of SEQ ID NO 2 an G allele at position 100 of SEQ ID NO 2 a C allele at position 486 of SEQ ID NO 58 a G allele at position 254 of SEQ ID NO 27 a G allele at position 729 of SEQ ID NO 59 a G allele at position 267 of SEQ ID NO 60 a G allele at position 562 of SEQ ID NO 25 a C allele at position 1271 of SEQ ID NO 26 and an A allele at position 193 of SEQ ID NO 55. See .

CML333 is an exotic inbred variety of maize from the International Maize and Wheat Improvement Center CIMMYT in Mexico. CML333 is known to be resistant to both southwestern corn borer and fall armyworm. CML333 comprises an A allele at positions 4979 4981 of SEQ ID NO 7 a G allele at position 472 of SEQ ID NO 48 a G allele at position 237 of SEQ ID NO 56 a G allele at position 173 of SEQ ID NO 57 a G allele at 0172A an A allele at position 116 of SEQ ID NO 23 an A allele at position 100 of SEQ ID NO 2 and an A allele at position 267 of SEQ ID NO 60. See .

CML322 is an exotic inbred variety of maize from the International Maize and Wheat Improvement Center CIMMYT in Mexico. CML322 comprises a C allele at position 386 of SEQ ID NO 46 an A allele at positions 4979 4981 of SEQ ID NO 7 a G allele at position 472 of SEQ ID NO 48 a C allele at position 266 of SEQ ID NO 44 a C allele at position 309 of SEQ ID NO 19 a C allele at position 111 of SEQ ID NO 51 an A allele at position 562 of SEQ ID NO 25 and an A allele at position 1271 of SEQ ID NO 26. See .

Cateto SP VII is an exotic variety of maize that is native to Brazil. Although it demonstrates a high combining ability with many varieties of maize it produces a relatively low yield. Cateto SP VII comprises an A allele at position 87 of SEQ ID NO 47 an A allele at position 4641 of SEQ ID NO 7 a G allele at position 472 of SEQ ID NO 48 a C allele at position 266 of SEQ ID NO 44 an A allele at position 746 of SEQ ID NO 24 a T allele at position 258 of SEQ ID NO 29 a G allele at position 217 of SEQ ID NO 23 an A allele at position 100 of SEQ ID NO 2 an A allele at position 486 of SEQ ID NO 58 and an A allele at position 193 of SEQ ID NO 55. See .

Confite Morocho AYA 38 is an exotic variety of maize that is native to Peru. Although it is resistant to it is susceptible to rust. Confite morocho AYA 38 comprises an A allele at position 4641 of SEQ ID NO 7 a C allele at position 237 of SEQ ID NO 56 a G allele at position 391 of SEQ ID NO 33 a C allele at position 309 of SEQ ID NO 19 an insertion at positions 264 271 of SEQ ID NO 2 and an A allele at position 486 of SEQ ID NO 58. See .

Tuxpeno VEN 692 is an exotic variety of maize that is native to Venezuela. It is highly resistant to both and rust. Tuxpeno VEN 692 comprises an A allele at position 4641 of SEQ ID NO 7 a C allele at position 237 of SEQ ID NO 56 a G allele at position 258 of SEQ ID NO 29 a C allele at position 463 of SEQ ID NO 19 and an A allele at position 193 of SEQ ID NO 55. See .

Maize plants of the NP2391 variety were crossed with 134 exotic varieties of maize including CML333 CML322 Cateto SP VII Confite Morocho AYA 38 and Tuxpeno VEN 692. The progeny of these crosses were backcrossed with NP2391 for five generations to create NP2391 exotic hybrids. Segmental introgressions of the shaggy kinase gene GENBANK Database Accession No. AY103545 incorporated by reference herein were identified in NP2391 exotic hybrids representing 42 of the exotic donors CML333 CML322 Cateto SP VII Confite Morocho AYA 38 Tuxpeno VEN 692 CML69 HH5982 TLT0766 CML103 M37W TLZ0845 AGG742 NC358 P39 Serrano GUA 3 Mochero LBQ 17 KXI0970 6B209 Cholito BOV 705 CML228 Coroico Amarillo 8B006 EE8001 Enano M.D.3 Perola BOV 711 Puya Grande SAN XPRR001 B97 Cacao SAS 327 Tx303 Clavito ECU 366 Early Caribbean MAR 10 Patillo BOV 502 Rabo De Zorro ANC 325 Shajatu ANC 120 Shoe Peg PI269743 St. Croix IVC 2 Oh7B Polio VEN 336 Tzi8 and Oh43 . Each of the NP2391 exotic hybrids containing a segmental introgression in the shaggy kinase gene was selfed for two generations selecting for progeny comprising the exotic donor genotype at positions 4979 4981 of SEQ ID NO 7 position 4641 of SEQ ID NO 7. Two lines were selected from the progeny of each NP2391 exotic hybrid selfing one homozygous for the exotic donor genotype at positions 4979 4981 of SEQ ID NO 7 position 4641 of SEQ ID NO 7 and the other homozygous for the NP2391 genotype at positions 4979 4981 of SEQ ID NO 7 position 4641 of SEQ ID NO 7. Each of these lines was crossed with NP2460 to create an F1 hybrid line. In addition NP2460 was crossed directly with NP2391 to create an F1 control hybrid.

The F1 hybrids were evaluated in four drought stress locations La Salle Colo. United States of America Gilroy Calif. United States of America Los Andes Chile and Granaros Chile with six treatment replications at each location using a restricted randomized block design as well as in twelve single rep cornbelt locations. Drought stress treatments were imposed around the time of pollination and consisted of a period of water deficiency capable of decreasing grain yield by about 40 60 . The timing of each drought stress treatment was determined by soil type and local climate from which water holding capacity and the evapotranspiration rate ET were estimated or measured . Normal irrigation was stopped about 3 4 weeks before mid shed to allow the soil moisture level to drop to a critical level which level was reached approximately 7 days prior to pollination. Once the soil dried to the critical level deficit irrigation was commenced approximately 40 of ET . Drought stress treatments were continued for two weeks following mid shed at which time normal irrigation was resumed.

Drought tolerance was evaluated by measuring grain yield at standard moisture percentage YGSMN and grain moisture at harvest GMSTP . The statistical data quality control was done by plotting the distribution of the data. Residuals were generated using the following model location replications location family allele family wherein Y dependent phenotype and phenotypic average Location and replications location were random. Family and allele family were fixed. Residuals were analyzed across replications within a family and flagged according to specified criteria which criteria depended upon the trait being evaluated. The final analysis was performed for both individual locations and for locations combined using the following model replications family allele family replications family wherein Y dependent phenotype and phenotypic average. Replications were random. Family and allele family were fixed. Least squares means were calculated and student s T test was run for pairwise comparisons.

As shown in Table 10 six of the F1 hybrids derived from crossing NP2460 with a line homozygous for an exotic donor genotype at positions 4979 4981 of SEQ ID NO 7 position 4641 of SEQ ID NO 7 demonstrated enhanced drought tolerance.

NP2391 was crossed with CML333 and progeny derived from that cross were backcrossed with NP2391 for five generations to create an NP2391 CML333 shaggy hybrid. The NP2391 CML333 shaggy hybrid was selfed for two generations and two lines were selected based upon their genotype at positions 4979 4981 of SEQ ID NO 7 one line homozygous for the CML333 genotype AA CML333 homozygous and the other homozygous for the NP2931 genotype GG CML333 homozygous . See . Each of these lines was crossed with NP2460 to create an F1 hybrid. See .

As shown in Table 2 the F1 hybrid line created by crossing NP2460 with the line comprising AA at positions 4979 4981 of SEQ ID NO 7 CML333 demonstrated enhanced drought tolerance as compared to both the control hybrid and the F1 hybrid derived by crossing NP2460 with the line comprising GG at positions 4979 4981 of SEQ ID NO 7 CML333 . Under drought stress treatment the CML333 hybrid demonstrated a significantly higher grain yield at standard moisture percentage 176 bu ac than both the control hybrid 162 bu ac and the CML333 hybrid 165 bu ac .

NP2391 was crossed with CML322 and progeny derived from that cross were backcrossed with NP2391 for five generations to create an NP2391 CML322 shaggy hybrid. The NP2391 CML322 shaggy hybrid was selfed for two generations and two lines were selected based upon their genotype at positions 4979 4981 of SEQ ID NO 7 one line homozygous for the CML322 genotype AA CML322 homozygous and the other homozygous for the NP2931 genotype GG CML322 homozygous . See . Each of these lines was crossed with NP2460 to create an F1 hybrid. See .

As shown in Table 2 the F1 hybrid line created by crossing NP2460 with the line comprising AA at positions 4979 4981 of SEQ ID NO 7 CML322 demonstrated enhanced drought tolerance as compared to both the control hybrid and the F1 hybrid derived by crossing NP2460 with the line comprising GG at positions 4979 4981 of SEQ ID NO 7 CML322 . Under drought stress treatment the CML322 hybrid demonstrated a significantly higher grain yield at standard moisture percentage 182 bu ac than both the control hybrid 162 bu ac and the CML322 hybrid 159 bu ac . Notably the grain yield of the CML322 hybrid under drought stress conditions was nearly identical to its yield under cornbelt conditions.

NP2391 was crossed with Cateto SP VII and progeny derived from that cross were backcrossed with NP2391 for five generations to create an NP2391 Cateto SP VII shaggy hybrid. The NP2391 Cateto SP VII shaggy hybrid was selfed for two generations and two lines were selected based upon their genotype at position 4641 of SEQ ID NO 7 one line homozygous for the Cateto SP VII genotype AA Cateto homozygous and the other homozygous for the NP2931 genotype CC Cateto homozygous . See . Each of these lines was crossed with NP2460 to create an F1 hybrid. See .

As shown in Table 2 the F1 hybrid line created by crossing NP2460 with the line comprising AA at position 4641 of SEQ ID NO 7 Cateto demonstrated enhanced drought tolerance as compared to both the control hybrid and the F1 hybrid derived by crossing NP2460 with the line comprising CC at position 4641 of SEQ ID NO 7 Cateto . Under drought stress treatment the Cateto hybrid demonstrated a significantly higher grain yield at standard moisture percentage 168 bu ac than both the control hybrid 162 bu ac and the Cateto hybrid 150 bu ac .

NP2391 was crossed with Confite Morocho AYA 38 and progeny derived from that cross were backcrossed with NP2391 for five generations to create an NP2391 Confite Morocho AYA 38 shaggy hybrid. The NP2391 Confite Morocho AYA 38 shaggy hybrid was selfed for two generations and two lines were selected based upon their genotype at position 4641 of SEQ ID NO 7 one line homozygous for the Confite Morocho AYA 38 genotype AA Confite homozygous and the other homozygous for the NP2931 genotype CC Confite homozygous . See . Each of these lines was crossed with NP2460 to create an F1 hybrid. See .

As shown in Table 2 the F1 hybrid line created by crossing NP2460 with the line comprising AA at position 4641 of SEQ ID NO 7 Confite demonstrated enhanced drought tolerance as compared to both the control hybrid and the F1 hybrid derived by crossing NP2460 with the line comprising CC at position 4641 of SEQ ID NO 7 Confite . Under drought stress treatment the Confite hybrid demonstrated a significantly higher grain yield at standard moisture percentage 170 bu ac than both the control hybrid 162 bu ac and the Cateto hybrid 157 bu ac .

NP2391 was crossed with Tuxpeno VEN 692 and progeny derived from that cross were backcrossed with NP2391 for five generations to create an NP2391 Tuxpeno VEN 692 shaggy hybrid. The NP2391 Tuxpeno VEN 692 shaggy hybrid was selfed for two generations and two lines were selected based upon their genotype at position 4641 of SEQ ID NO 7 one line homozygous for the Tuxpeno VEN 692 genotype AA Tuxpeno homozygous and the other homozygous for the NP2931 genotype CC Tuxpeno homozygous . See . Each of these lines was crossed with NP2460 to create an F1 hybrid. See .

As shown in Table 2 the F1 hybrid line created by crossing NP2460 with the line comprising AA at position 4641 of SEQ ID NO 7 Tuxpeno demonstrated enhanced drought tolerance as compared to both the control hybrid and the F1 hybrid derived by crossing NP2460 with the line comprising CC at position 4641 of SEQ ID NO 7 Tuxpeno . Under drought stress treatment the Tuxpeno hybrid demonstrated a significantly higher grain yield at standard moisture percentage 175 bu ac than both the control hybrid 162 bu ac and the Tuxpeno hybrid 165 bu ac .

157 water optimized hybrids from 36 chasis were compared with base genetic control plant in 9 21 locations each under four environment types full irrigation limited irrigation non irrigated non stress receiving adequate rainfall non irrigated stress dryland low plant density non stress receiving adequate rainfall and dryland stress . As shown in Tables 11 and 12 water optimized hybrids outperformed closely related base hybrids under all conditions tested.

All references listed below as well as all references cited in the instant disclosure including but not limited to all patents patent applications and publications thereof scientific journal articles and database entries e.g. GENBANK database entries and all annotations available therein are incorporated herein by reference in their entireties to the extent that they supplement explain provide a background for or teach methodology techniques and or compositions employed herein.

It will be understood that various details of the presently disclosed subject matter can be changed without departing from the scope of the presently disclosed subject matter. Furthermore the foregoing description is for the purpose of illustration only and not for the purpose of limitation.

