---

title: Cell free DNA diagnostic testing standards
abstract: Embodiments of the invention include methods and compositions for producing proficiency testing standards for noninvasive prenatal genetic diagnostics and for the detection and monitoring of cancer. The compositions can include a plurality of different nucleosomal DNA fragments derived from either primary cells or cell lines. The amount of the different nucleosomal DNA fragments can be varied so as to simulate naturally occurring cell free DNA samples obtained from the blood of the pregnant woman or naturally occurring cell free DNA samples obtained from the blood of cancer patients.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09499870&OS=09499870&RS=09499870
owner: Natera, Inc.
number: 09499870
owner_city: San Carlos
owner_country: US
publication_date: 20140926
---
Cell free DNA found in the blood and other bodily tissues can be used to detect and diagnose many genetic disorders. Numerous methods exist for non invasive prenatal genetic diagnostics. Non invasive prenatal genetic diagnoses can be performed on cell free DNA e.g. obtained from blood from a patient. Cell free DNA can also be used to detect or monitor the presence of tumor cells in patient. Such methods are complex to carry out and are subject to numerous errors resulting in imprecision and inaccuracy. It is important for commercial laboratories to demonstrate proficiency in testing in order to obtain regulatory approval for carrying out such tests. Accordingly it is necessary for laboratories carrying out such procedures to engage in proficiency testing using standards for analysis. Such standard testing can be problematic given the relative scarcity of naturally occurring samples and the variability between such samples. Provided herein are methods and compositions for addressing this problem.

An embodiment of the invention is a prenatal nucleic acid proficiency testing standard composition comprising a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation from a second cell source wherein the quantity of the first nucleic acid preparation is greater than the quantity of the second nucleic acid preparation. In another embodiment the invention is a prenatal nucleic acid proficiency testing standard composition comprising a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation from a second cell source wherein the quantity of the first nucleic acid preparation is approximately equal to the quantity of the second nucleic acid preparation.

In some embodiments the prenatal nucleic acid proficiency testing standard composition the first nucleosomal nucleic acid preparation is derived from a primary cell source. In some embodiments the first nucleosomal nucleic acid preparation is derived from a cell line. In some embodiments the first cell source and the second cell source are cell lines. In some embodiments the first cell source and the second cell source are primary cell sources. In some embodiments primary cell source is blood cells from a buffy coat layer.

In some embodiments of the subject compositions nucleosomal nucleic acid preparation has been prepared with an endonuclease. The endonuclease can be a micrococcal endonuclease. In some embodiments the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid are one or more nucleosomal ladder components. In some embodiments the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid comprise a mononucleosomal ladder fraction. In some embodiments the first nucleosomal nucleic acid preparation comprises a dinucleosomal ladder fraction. In some embodiments the first nucleosomal nucleic acid preparation comprises a trinucleosomal ladder fraction. In some embodiments the second nucleosomal nucleic acid preparation comprises a dinucleosomal ladder fraction. In some embodiments the second nucleosomal nucleic acid preparation comprises a trinucleosomal ladder fraction. In some embodiments second nucleosomal nucleic acid preparation comprises a dinucleosomal ladder fraction. In some embodiments the second nucleosomal nucleic acid preparation comprises a trinucleosomal ladder fraction.

In some embodiments the amount of the second nucleosomal nucleic acid preparation is less than 40 of the total nucleic acid in the composition. In some embodiments the amount of the second nucleosomal nucleic acid preparation is less than 30 of the total nucleic acid in the composition. In some embodiments the second nucleosomal nucleic acid preparation is less than 20 of the total nucleic acid in the composition. In some embodiments the amount of the second nucleosomal nucleic acid preparation is less than 10 of the total nucleic acid in the composition.

The first cell source and the second cell source may be from genetically related individuals including embodiments for use in the analysis of fetal DNA. In some embodiments the first cell source is the mother of the second cell source. In some embodiments the first cell source is the father of the second cell source. In some embodiments the first cell source is a sibling of the second cell source.

An embodiment of the invention is a composition that cancer cell nucleic acid proficiency testing standard for diagnostics that detect cell free cancer DNA comprising a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation from a second cell source wherein the quantity of the first nucleic acid preparation is greater than the quantity of the second nucleic acid preparation. In another embodiment the invention is a prenatal nucleic acid proficiency testing standard composition comprising a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation from a second cell source wherein the quantity of the first nucleic acid preparation is approximately equal to the quantity of the second nucleic acid preparation. In some embodiments the cancer cell nucleic acid proficiency testing standard composition comprises a first nucleosomal nucleic acid preparation that is derived from a primary cell source. In some embodiments the first nucleosomal nucleic acid preparation is derived from a cell line. In some embodiments the first cell source and the second cell source are cell lines. In some embodiments the first cell source and the second cell source are primary cell sources.

In some embodiments of the subject compositions nucleosomal nucleic acid preparation can be been prepared with an endonuclease. The endonuclease can be a micrococcal endonuclease. In some embodiments the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid are nucleosomal ladder fractions. In some embodiments the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid comprise mononucleosomal ladder fractions. In some embodiments the first nucleosomal nucleic acid preparation comprises a dinucleosomal ladder fraction. In some embodiments the first nucleosomal nucleic acid preparation comprises a trinucleosomal ladder fraction. In some embodiments the second nucleosomal nucleic acid preparation comprises a dinucleosomal ladder fraction. In some embodiments the second nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions. In some embodiments second nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions. In some embodiments the second nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

In some embodiments the amount of the second nucleosomal nucleic acid preparation is less than 40 of the total nucleic acid in the composition. In some embodiments the amount of the second nucleosomal nucleic acid preparation is less than 30 of the total nucleic acid in the composition. In some embodiments the second nucleosomal nucleic acid preparation is less than 20 of the total nucleic acid in the composition. In some embodiments the amount of the second nucleosomal nucleic acid preparation is less than 10 of the total nucleic acid in the composition.

The first cell source and the second cell source may be from genetically related individuals from the same individual or from genetically unrelated individuals. In some embodiments the first cell source is the mother of the second cell source. In some embodiments the first cell source is non cancerous tissue and the second cell source is a corresponding cancer cell culture.

The invention also includes sets of the subject cell free DNA diagnostic testing standards wherein the set comprise at least two cell free DNA diagnostic testing standards. In some embodiments the sets can comprise cell free DNA diagnostic testing standards that are the same as one another with respect to the identity of the cell sources but differ with respect to one another with respect to the ratios of the different nucleosomal nucleic acid components of the mixture.

The invention also includes methods of making the subject prenatal nucleic acid proficiency testing standard compositions and nucleic acid proficiency testing standard compositions made by the methods. Embodiments of such methods include mixing a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation from a second cell source wherein the quantity of the first nucleic acid preparation is greater than the quantity of the second nucleic acid preparation. Embodiments of the subject methods include methods of making all of the compositions described herein.

The invention also includes methods of making the subject cell free nucleic acid diagnostic proficiency testing standard compositions prepared by the subject methods. Embodiments of such methods include mixing a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation from a second cell source wherein the quantity of the first nucleic acid preparation is greater than or in some embodiment equal to the quantity of the second nucleic acid preparation. Embodiments of the subject methods include methods of making all of the compositions described herein. The cell free nucleic acid diagnostic proficiency testing standard compositions prepared by the subject methods can be used for testing proficiency to perform diagnosis or detection of a wide range of genetic disorders such as cancer or fetal chromosomal abnormalities.

The present invention provides nucleic acid standards that are compositions useful for proficiency testing of laboratories engaging in the analysis of circulating cell free DNA samples including cell free DNA that is used for prenatal genetic analysis or cell free DNA that is used for the detection or analysis of cancerous cells. These standards are designed to simulate naturally occurring cell free circulating DNA found in the bloodstream of a test subject e.g. a pregnant woman or suspected cancer patient. It was unexpected that artificially created standards could produce results that were sufficiently close to results obtained from actual patient data so as to provide a useful substitute for a naturally occurring cell free DNA sample. These artificially created standards can be used to simulate cell free DNA samples obtained directly from a natural source. Many commercial testing laboratories are regulated such laboratories have need to develop standardized testing procedures in order to obtain approval or accreditation. The development of such standardized testing procedures can be facilitated by using standards for analysis. A problem with such biological standards is there limited availability. This problem may be addressed using the subject composition and related methods which can be used to produce large quantities of genetic testing standards thereby facilitating the commercialization of the tests of interest.

A non invasive prenatal diagnostic assay can detect and analyze cell free DNA that is a mixture of maternally derived DNA and DNA derived from the fetus carried by the mother. In some embodiments the mother may be carrying more than one fetus e.g. twins and the subject cell free DNA standard is designed to simulate such cases of multiple births.

Some embodiments of the invention are compositions comprising at least two nucleosomal nucleic acid preparations wherein each nucleosomal nucleic acid preparation is derived from a different cell source. In some embodiments of the invention the compositions can comprise more than two nucleosomal nucleic acid preparations derived from different cell sources. In some embodiments of such standards comprise a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid derived from a second cell source. The different cell sources in a given preparation are different from one another. In various embodiments different ratios of the nucleosomal nucleic acid preparation components of the subject compositions are provided for thereby enabling the creation of proficiency testing standards that simulate a given fetal fraction of interest. In some embodiments different ratios of the nucleosomal nucleic acid preparation components of the subject compositions are provided for thereby enabling the creation of proficiency testing standards that simulate different stages of cancer.

The subject prenatal nucleic acid proficiency testing standard compositions may be created so as to simulate a wide variety of potential patient samples. The patient samples can vary with respect to the relative amounts of maternally derived cell free nucleic acid to fetal he derived cell free nucleic acid. An additional source of potential variation is chromosomal abnormalities or genetic alleles associated with a genetic disease that are present in the fetus or the mother. Examples of chromosomal abnormalities include various aneuploidies deletions copy number variations translocations and the like. Examples of aneuploidies include trisomy 21 trisomy 18 trisomy 13 Turner s syndrome Klinefelter s syndrome XYY XXX and the like. Additionally in some embodiments the source of variation may be a genetic allele associated with a genetic disease or carrier state such as cystic fibrosis sickle cell anemia thalassemias Tay Sachs disease Canavan disease and the like. Similarly various cancer cells genomes can comprise aneuploidies deletions copy number variations translocations and the like. The patient samples can vary with respect to the relative amounts cell free nucleic acid derived from the cancer cell of interest and from other non cancerous cells in the body of the patient.

The ratio of total fetal DNA to total maternal DNA maternal DNA plus fetal DNA can for the sake of convenience be referred to as the fetal fraction. Embodiments of cell free DNA diagnostic testing standards for prenatal nucleic acid proficiency testing can be produced to mimic a wide variety of potential fetal fractions present in actual maternal cell free circulating DNA samples obtained from pregnant women. Fetal fractions in the range of 1 to 70 or even higher as well as all increments within this range can be simulated in various embodiments of the subject compositions. In some embodiments the subject compositions comprise a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation derived from a second cell source where in the quantity of the first nucleic acid preparation is greater than the quantity of the second nucleic acid preparation. In some embodiments the first cell source will be representative of the mother and the second cell source will be representative of the fetus. In some embodiments the second size fractionate nucleic acid preparation will be less than 40 of the total nucleic acid amount in the final preparation. In some embodiments the second size fractionate nucleic acid preparation will be less than 30 of the total nucleic acid amount in the final preparation. In some embodiments the second size fractionate nucleic acid preparation will be less than 20 of the total nucleic acid amount in the final preparation. In some embodiments the second size fractionate nucleic acid preparation will be less than 10 of the total nucleic acid amount in the final preparation.

In embodiments of the invention for use with the analysis of cell free derived from cancer cells the ratio of total cell free cancer cell derived DNA to total cell free DNA cell free cancer cell derived DNA plus other cell free DNA found in the sample can for the sake of convenience be referred to as the cancer cell fraction. Cell free cancer nucleic acid analysis proficiency testing standards can be produced to mimic a wide variety of cancer cell fractions present in cell free circulating DNA samples obtained from patients or suspected patients. Cancer fractions in the range of 1 to 90 or even higher as well as all increments within this range can be simulated in various embodiments of the subject compositions. In some embodiments the subject compositions comprise a first nucleosomal nucleic acid preparation derived from a first cell source and a second nucleosomal nucleic acid preparation derived from a second cell source where in the quantity of the first nucleic acid preparation is greater than the quantity of the second nucleic acid preparation. In some embodiments the first cell source will be representative of the mother and the second cell source will be representative of the non cancerous cells. In some embodiments the second size fractionate nucleic acid preparation will be less than 40 of the total nucleic acid amount in the final preparation. In some embodiments the second nucleosomal nucleic acid preparation will be less than 30 of the total nucleic acid amount in the final preparation. In some embodiments the second size fractionate nucleic acid preparation will be less than 20 of the total nucleic acid amount in the final preparation. In some embodiments the second size fractionate nucleic acid preparation will be less than 10 of the total nucleic acid amount in the final preparation. It will be understood by person skilled in the art that although the previous description refers to a first cell source and a second cell source embodiments of the invention also provided for that include more than two cell sources for example the sample may be prepared from one tumor cell line and 3 separate non tumor cell lines.

The nucleosomal fractions derived from nucleosomal ladders are said to be fractions because they do not contain all sizes of the DNA fragments in the nucleosomal preparation derived from the first cell source or the second cell source. By employing nucleosomal nucleic acid preparations a practical upper size limit is applied the specific size limit depending on whether mononucleosomal dinucleosomal or trinucleosomal fraction containing preparations are used in the particular embodiment.

The compositions include multiple possible combinations of nucleosomal fractions from the first cell source and the second cell source. In some embodiments the nucleosomal nucleic acid preparation prepared from the first cell source comprises 1 the mononucleosomal fraction the mononucleosomal fraction and the dinucleosomal fraction or the mononucleosomal fraction and the dinucleosomal fraction and the trinucleosomal fraction. In some embodiments the nucleosomal nucleic acid preparation prepared from the second cell source comprises 1 the mononucleosomal fraction the mononucleosomal fraction and the dinucleosomal fraction or the mononucleosomal fraction and the dinucleosomal fraction and the trinucleosomal fraction. The provide embodiments included all possible combinations of the nucleosomal fractions 1 the mononucleosomal fraction from the first cell source in combination with the mononucleosomal fraction from the second cell source 2 the mononucleosomal fraction from the first cell source in combination with the mononucleosomal and dinucleosomal fractions from the second cell source 3 the mononucleosomal fraction from the first cell source in combination with the mononucleosomal dinucleosomal and trinucleosomal fractions from the second cell source 4 the mononucleosomal and dinucleosomal fractions from the first cell source in combination with the mononucleosomal fraction from the second cell source 5 the mononucleosomal and dinucleosomal fractions from the first cell source in combination with the mononucleosomal and dinucleosomal fractions from the second cell source 6 the mononucleosomal and dinucleosomal fractions from the first cell source in combination with the mononucleosomal dinucleosomal and trinucleosomal fractions from the second cell source 7 the mononucleosomal dinucleosomal and trinucleosomal fractions from the first cell source in combination with the mononucleosomal fraction from the second cell source 8 the mononucleosomal dinucleosomal and trinucleosomal fractions from the first cell source in combination with the mononucleosomal and dinucleosomal fractions from the second cell source 9 the mononucleosomal dinucleosomal and trinucleosomal fractions from the first cell source in combination with the mononucleosomal dinucleosomal and trinucleosomal fractions from the second cell source.

The nucleosomal nucleic acid preparations used to create the subject prenatal nucleic acid proficiency testing standard compositions can be derived from a variety of cell types. Suitable cell types can be primary cells obtained directly from a human subject or can be cell lines that can be propagated in in vitro cell culture. A wide variety of primary cells can be used. Typically primary cells from an easily removable tissue source are used e.g. blood or a cellular blood fraction such as a buffy coat layer. Similarly a wide variety of cell lines may be used. Examples of such cell lines include cell lines obtained from the Corriell Institute or the American Type Culture Collection ATTC .

In some embodiments the cell sources are from genetically related individuals. Examples of such genetically related individuals are 1 mother and child 2 mother and multiple children and 3 mother father and child. In other embodiments the cell sources are from genetically unrelated individuals. In some embodiments the primary cells are from genetically related individuals. In other embodiments the cell lines are obtained from genetically related individuals.

In some embodiments the cell sources are from cells from the same tissue type wherein one of the cell types is a cancer cell line and the other cell source is a cell line from the same tissue but not a cancerous cell line.

In some embodiments the first cell source may be from a primary cells and the second cell source may be from a cell line. In some embodiments the first cell source may be from a cell line and the second cell source may be from primary cells.

The nucleic acids may be isolated from the cell sources by a variety of methods well known to the person of ordinary skill in molecular biology. Typically such methods will involve lysing the cell thereby liberating nucleic acids so as to leave chromatin structure sufficiently intact to allow the preparation nucleosomal ladders i.e. nucleosomal preparations. Suitable cell lysis methods include methods in which the nucleus is separately released for subsequent isolation and methods in which the nuclear membrane is dissolved. In some embodiments the cells may be permeabilized e.g. using a detergent such as lysolecithin so as to retain chromatin structure. In some embodiments the cell membrane may be disrupted by inducing apoptosis in the cells of the cell source.

It is of interest to prepare nucleic acid that are of free of other cellular components so as to enable the biochemical manipulation of the nucleosomal ladders for use in subsequent procedures e.g. DNA sequencing. In an embodiment of the invention the commercially available nucleic acid system called AMPURE can be used to both purify DNA and isolate nucleosomal fractions of the desired size.

In human cells as well as other eukaryotic cells nuclear DNA is organized in the chromatin in nucleosome s in which the chromosomal DNA is organized in approximately 147 base pair units of DNA wrapping around a histone core. The DNA is close proximity to the histone core is relatively resistant to cleavage as compared to the DNA that is present between the nucleosomes. The nucleosomes form a regular pattern in the chromatin such that exposure of the nucleosomal structures in chromatin to an endonuclease e.g. micrococcal endonuclease results in a reproducible pattern of a DNA fragments of approximately defined length. This pattern can be visualized by separating the nucleic acid digest fragment based on length e.g. by electrophoresis. The histone component of the nucleosome serves to protect the DNA wrapped around the histone core from endonuclease digestion. Fragmenting genomic DNA with a nuclease or fragmenting with a non enzymatic method e.g. a chemical digestion with a hydroxyl radical based reaction electromagnetic radiation or sonication are well known to persons of ordinary skill in the art. Subjecting the chromatin to a digestion reaction results in the formation of a set of nucleic acid fragments approximately 147 base pairs in length and multiples thereof for the sake of convenience such a set of fragments can be can be referred to as a nucleosomal ladder. A nucleosomal ladder would for example appear as a series of bands of different molecular weight when separated by gel electrophoresis. The nucleosomal ladder comprises the approximately 147 base pair fragment and the multiples thereof obtained by digesting the chromatin. The 147 base pair fragment is referred to as the mononucleosomal fraction of the nuclear ladder. The two fold multiple of the mononucleosomal fraction is referred to as the dinucleosomal fraction and is formed by nucleases or other DNA cleavage agents cleaving DNA adjacent to two nucleosomes but leaving the internucleosomal region intact . The three fold multiple of the mononucleosomal fraction is referred to as the trinucleosomal fraction and is formed by nucleases or other DNA cleavage agents cleaving DNA adjacent to three nucleosomes but leaving the internucleosomal region intact It will be understood by person skilled in the art of molecular biology that nuclease cleavage or other DNA cleavage agents is imprecise and can give rise a set of nucleic acid fragments of similar but not identical size.

In some embodiments of the invention the nucleosomal ladders may be produced by inducing apoptosis in cells. As a part of apoptosis process endogenous endonucleases cleave the DNA component of the chromatin so as to form nucleosomal ladders. In some embodiments of the invention the nucleosomal ladders may be produced by digesting the chromatin with an endonuclease e.g. micrococcal endonuclease. In other embodiments of the invention the nucleosomal ladder may be produced by exposing the chromatin to digestion with non enzymatic agents.

AMPURE can be used to both purify DNA and isolate nucleosomal fractions of the desired size. In other embodiments nucleosomal fractions of the desired size can be obtained by gel electrophoresis separated fragment purification purification from HPLC or purification through ultracentrifugation.

In some embodiments the nucleosomal fractions mononucleosomal dinucleosomal trinucleosomal and various combinations thereof may be manipulated so as simulate one or more genetic abnormalities such as a duplication deletion or point mutation. For example a deletion may be simulated by exposing nucleosomal preparations to a solid support comprising nucleic acids or analogs thereof to selectively bind to the region to be deleted thereby producing a preparation containing a greatly reduced amount of the region to be deleted. Similarly point mutations may be introduced by techniques such as PCR performed on the nucleosomal fractions.

The subject compositions for prenatal nucleic acid proficiency testing can be used in a wide variety of prenatal genetic testing methods. The proficiency testing standards are used essentially the same as a sample obtained from a test subject thereby providing a meaningful standard from the specific test being evaluated. Such methods of noninvasive prenatal genetic testing typically involve the analysis of cell free nucleic acids found in the bloodstream of a pregnant woman. In some embodiments the prenatal genetic testing method involves non directed sequencing of the cell free nucleic acids such as in U.S. Pat. No. 8 296 076 B2 U.S. Pat. No. 8 008 018 B2 U.S. Pat. No. 7 888 017 B2 U.S. Pat. No. 8 467 976 B2. In other embodiments the directed analysis of specific polymorphic regions or specific non polymorphic regions such as in patent applications US 2013 0143213 A1 US 2013 0172211 A1 US 2012 0270212 A1 US 2012 0122701 A1 US 2013 0123120 A1 US 2011 0178719 A1 can be employed.

Various protocols are known to the person or ordinary skill in the art for analyzing cell free DNA circulating in the blood stream or other tissue but ultimately derived from cancerous cells for example see publications such as Circulating Cell Free DNA in Plasma Serum of Lung Cancer Patients as a Potential Screening and Prognostic Tool Pathak et al Clinical Chemistry October 2006 vol. 52 no. 10 1833 1842 Cell free Tumor DNA in Blood Plasma As a Marker for Circulating Tumor Cells in Prostate Cancer Schwarzenbach et al Clin Cancer Res Feb. 1 2009 15 1032 Cell free DNA measurement in various carcinomas and establishment of normal reference range Wua et al Clinica Chimica Acta Volume 321 Issues 1 2 July 2002 Pages 77 87 Detection of Circulating Tumour DNA in the Blood Plasma Serum of Cancer Patients Anker et al Cancer and Metastasis Reviews 1999 Volume 18 Issue 1 pp 65 73 Cell free nucleic acids as biomarkers in cancer patients Schwarzenbach et al Nature Reviews Cancer 11 426 437 June 2011 Circulating Tumor Specific DNA A Marker for Monitoring Efficacy of Adjuvant Therapy in Cancer Patients Fiegl et al Cancer Res Feb. 15 2005 65 1141.

The following examples are offered for purposes of illustration only and should not be construed as limiting the claimed inventions.

Cell free DNA cfDNA based non invasive prenatal testing NIPT allows for the identification of fetal aneuploidies from the mixture of maternal and fetal cfDNA cell free DNA in maternal circulation using next generation sequencing based approaches. Such tests are revolutionizing prenatal screening and fetal aneuploidy detection. However cfDNA is a mixture of maternal and fetal cfDNA and both the overall amount of cfDNA as well as the fraction of cfDNA from the fetus can be limiting. This limits the number of analyses that can be performed on a single sample e.g. for development and proficiency testing . Additionally validating NIPT performance on rare disorders is challenging as patient recruitment is limiting. To overcome these challenges a novel method for creating an artificial pregnancy plasma DNA plasmART was invented.

DNA was isolated from primary cells or cultured immortalized cells and treated to generate nucleosomal size ladders mono di and tri nucleosome size fragments . These ladders mimicked observed cfDNA fragment lengths which are derived from genomic DNA digested by apoptotically activated nucleases. This includes shorter fetal fragments and a combination of shorter and longer maternal fragments. To simulate pregnancy plasma the maternal and child cfDNAs were mixed at various ratios to mimic a range of fetal fractions. These mixtures were then examined using the Natera Panorama NIPT which employs the advanced Next generation Aneuploidy Test Using SNPs NATUS algorithm. The NATUS algorithm reports copy number for each chromosome with an associated confidence.

This approach allowed for the identification of the fetal fraction influence on test accuracy on the same mother child pair rather than comparing accuracy over fetal fractions encountered from distinct pregnancies in the population. The performance of the Natera Panorama NIPT was examined on mixtures of maternal and child samples. NIPT correctly distinguished affected and unaffected pregnancy plasmARTs suggesting that these mixtures behave similarly to cfDNA isolated from maternal plasma. The ability to call chromosome copy number with high confidence at fetal fractions of below 5 correlated well with true pregnancy plasma samples

Non invasive Prenatal Screening NIPS to conditions that are rare and not routinely screened for in pregnancy is challenging. The collection of sufficient samples to confidently validate test performance is essentially impossible. Further the samples that are collected are almost always identified after an invasive procedure and therefore of later gestational age and higher fetal fraction. As fetal fraction is a crucial parameter affecting performance of all NIPS using exclusively higher fetal fraction samples may result in inflated claims of test sensitivity. Therefore an alternative approach to validating NIPS for rare disorders is needed to adequately estimate test performance. A method to generate artificial cfDNA samples plasmArt i.e. cell free DNA DNA diagnostic testing standards for use in testing and validation that mimic the size distribution of natural cfDNA. PlasmArt can be generated from lymphoblastoid cell lines or white blood cells i.e. buffy coat of normal or affected individuals. Once prepared plasmArt from two individuals such as a mother and her child can be combined to simulate pregnancy cfDNA at any desired ratio enabling simulation of the fetal fractions observed in real populations. To generate artificial samples that mimic natural cfDNA we sought to replicate the mechanism of cfDNA fragmentation in vitro. An individual s cfDNA predominately arises from apoptosis of cells in the hematopoietic system Lui Y Y Chik K W Chiu R W Ho C Y Lam C W Lo Y M. Clin Chem. 2002 48 421 7 . During apoptosis the Caspase Activated DNase CAD is activated by Caspase 3 cleavage of the CAD inhibitor. The activated nuclease preferentially cleaves DNA between nucleosomes Widlak P. Acta Biochim Pol. 2000 47 1037 44 resulting in the characteristic mono di and tri nucleosomal sized DNA fragments observed in cfDNA Li Y Zimmermann B Rusterholz C Kang A Holzgreve W Hahn S. Clin Chem. 2004 50 1002 11 Fan H C Blumenfeld Y J Chitkara U Hudgins L Quake S R. Clin Chem. 2010 56 1279 86. . Each nucleosome coordinates approximately 146 nucleotides of DNA Luger K Mader A W Richmond R K Sargent D F Richmond T J. Nature. 1997 389 251 60. . Based on the intranucleosomal nuclease activity that generates cfDNA we used micrococcal nuclease MNase which has a similar biochemical activity of cleaving preferentially between nucleosomes Widlak P Li P Wang X Garrard W T J Biol Chem. 2000 275 8226 32 Allan J Fraser R M Owen Hughes T Keszenman Pereyra D. J Mol Biol. 2012 417 152 64. . Previous methods to generate artificial cfDNA from pregnancy samples have relied on sonicated DNA Srinivasan A Bianchi D W Huang H Sehnert A J Rava R P. Am J Hum Genet. 2013 92 167 76 . However sonication results in broad fragment size distributions peak size 200 nucleotides 100 See http www.diagenode.com en applications dna Shearing.php for a description of sonication sizes and distributions and start sites are not constrained by nucleosome position. By employing an enzyme with a similar biochemical activity to the in vivo nuclease involved in fragmentation the cell free DNA DNA diagnostic testing standards e.g. plasmArt preparation method described herein approximates the size and cleavage biases observed in natural cfDNA.

Artificial samples should approximate the size distribution of cfDNA observed in vivo to capture potential biases introduced during library construction. Library preparation PCR typically favors short fragments over long thus post amplification only short fragments will be represented. We first confirmed the nucleosomal ladder pattern observed in natural cfDNA from samples purified in the Natera clinical laboratory. To overcome the low concentration of natural cfDNA the cfDNA from 96 pregnant individuals was mixed in equal volumes concentrated approximately 50 fold and examined on a Bioanalyzer . The mononucleosomal peak is present at 180 nucleotides. A dinucleosomal peak is present at 382 nucleotides. PlasmArt was prepared from cell lines and white blood cells . PlasmArt from cell lines displays a mononucleosomal peak at 148 nucleotides and the dinucleosomal peak at 349 nucleotides. For white blood cells the mononucleosomal peak is at 146 nucleotides and the dinucleosomal peak is at 359 nucleotides. Thus the method for creating plasmArt results in a DNA fragment profile similar to natural cfDNA. The location of the peaks suggests that natural cfDNA are larger than artificial cfDNA fragments but this is consistent with the observation that CAD releases larger fragments than MNase likely due to a higher activity of MNase in vitro. Overall plasmArt is more similar than other methods used to create artificial cfDNA and we expect that size difference has a minimal effect. In fact the small fragments produce a more challenging sample type for PCR assay methods that require that target SNPs be flanked by two intact primer binding sites a less likely occurrence as DNA fragments become smaller.

In addition to mimicking the size distribution of natural cfDNA mixtures of mother and child plasmArt samples must have similar NIPS performance to pregnancy cfDNA samples. PlasmArt was generated from cell lines purchased from the Coriell Cell Repository GM11388 child and GM11389 mother . Four independent mixtures of mother and child were made such that the molar ratios were 3 6 9 and 12 child. These samples were used as input into Panorama NIPS. The fetal fraction calculated by the algorithm for these samples were 3.5 6.3 9.1 and 12.0 respectively R2 0.99 slope 0.94 indicating a near perfect correlation between input child amount and measured fetal fraction in Panorama .

Having demonstrated the ability to make predictable mixtures of mother and child we examined the ability to detect a paternally contributed 22q11.2 microdeletion. If a microdeletion originates in the father the lack of paternally contributed SNPs can be visualized on allele frequency plots . In this case there are no paternally contributed SNPs in the 22q11.2 region while there are paternally contributed SNPs in other genomic regions. This 22q11.2 deletion can be observed from low fetal fraction 4 to a relatively high fetal fraction 14 and all intermediate fetal fractions. Taken together these data show that child DNA can be mixed into mother DNA in predictable amounts and mixtures can be analyzed to identify known microdeletion syndromes.

 A allele ratios from individual binary SNPs A allele reads A B reads are shown in ascending order on the X axis by genomic region then by SNP chromosome location. Data points are colored according to maternal genotype AA red BB blue AB green . In pregnancy cfDNA and plasmArt the father s contribution to the mixture can be most readily observed as blue or red points offset from the 0 or 100 A allele fractions maternal BB and AA points respectively . These are SNPs for which the mother is homozygous AA or BB but has the fetus or mixed in child sample contributes a B or A allele respectively. For instance at 10 fetal fraction the mixed in child sample s contribution can be visualized as points centered at 5 and 95 since one half of the sample mixed in at 10 corresponds to the A or B allele respectively. The absence of any contribution from the paternal SNPs observed at all fetal fractions for the 22q11.2 region is consistent with a paternally contributed microdeletion of this region. This titration demonstrates the ability to detect a paternally contributed microdeletion over a wide range of fetal fractions down to 4 .

We next examined the ability to detect maternally contributed microdeletions. Fifteen plasmArt samples were made over a broad range of fetal fractions from a Coriell Cell Repository Angelman Syndrome family GM11517 mother and GM11516 child . Angelman Syndrome is caused by a maternally contributed deletion of 15q11.2 q13. The calculated fetal fractions were 6.8 7.8 8.4 10.0 10.8 11.8 13.0 14.4 14.6 15.2 16.7 18.6 20.4 21.3 and 24.6 . Unlike paternal deletions maternally inherited deletions result in subtler changes to the allele ratios and are difficult to detect visually on Allele Ratio Plots. Thus the Panorama NATUS algorithm modified to detection segmental deletions was employed to examine maternal deletions. The algorithm correctly identified the deletion in all 15 of the plasmArt samples. Importantly in these samples the algorithm also evaluated copy number of the 22q11.2 Cri du chat and 1p36 regions. The algorithm correctly identified 44 of 45 regions 3 regions by 15 samples as normal no deletion detected. The algorithm did not return a high confidence result for the 22q11.2 region of the 6.8 sample. The observed sensitivity and specificity of these initial tests indicate that plasmArt can be used for developing and validating NIPS for rare syndromes.

It was demonstrated that the cell free DNA standard prepared using MNase in vitro more faithfully recapitulates the size distribution of natural cfDNA than sonication. MNase treatment of either cell lines or white blood cells gave similar results that were comparable to natural cfDNA size distributions. Next we showed that mixtures of various amounts of mother and child plasmArt samples correlated very well with the fetal fractions measured by the NATUS algorithm. Finally we demonstrated that plasmArt mixtures could be used to simulate pregnancy cfDNA samples at various fetal fractions having a paternally inherited 22q11.2 deletion and a maternally inherited Angelman deletion. For each of these simulated groups the NATUS algorithm correctly identified the deletions and the unaffected regions.

These results suggest that plasmArt can be used as a tool for validating rare disorders in the context of NIPS.

Assay validation was performed using five human breast cancer cell lines HCC38 HCC1143 HCC1395 HCC1954 and HCC2218 along with matched normal cell lines these cell lines and matched genomic DNA gDNA samples were obtained from American Type Culture Collection ATCC . Paired father and child cell lines GM10383 and GM10382 respectively for producing cell free nucleic acid standards details below were obtained from the Coriell Cell Repository Camden N.J. . The child of this cell line is a DiGeorge Syndrome DGS proband with a maternal deletion and thus the child cell line has only the paternal DGS 22q11 region the parental origin of the deletion was determined by our SNP based mmPCR assay data not shown .

Tumor tissues from 14 breast cancer patients were obtained from Geneticist Glendale Calif. and North Shore LIJ Manhasset N.Y. In addition matched buffy coat 4 patients and matched plasma samples 9 patients were obtained. Blood from each subject was collected into EDTA tubes and cfDNA was isolated from 1 ml plasma using the QIAamp Circulating Nucleic Acid Kit catalog no. 55114 Qiagen Valencia Calif. according to the manufacturer s instructions.

All cell culture reagents culture media and fetal bovine serum FBS were obtained from Life Technologies Foster City Calif. . ATCC cell lines were cultured according to the ATCC cell culturing passaging and cryogenic storage guidelines. Cells were cultured in 10 FBS RPMI 1640 high glucose with pyruvate with 2 mM L Glutamine at 37 C. with 5 CO2. Seed stocks were made of each cell line after one passage and a cut off of five passages was chosen in order to preserve the genetic stability of each cell line. Cells from the Coriell Cell Repository were grown according to manufacturer s instructions GM10382 in 15 FBS DMEM and GM10383 in 15 FBS RPMI 1640. Cells were washed twice in DPBS to remove FBS and culture media before DNA isolation.

Single cells were isolated from cultures manually using an inverted phase contrast microscope. A serial dilution method was implemented involving pipet transfers of single media droplets containing cells in suspension onto the surface of a petri dish. Subsequently small volumes of the original cell suspension droplet were mixed into droplets of phosphate buffered saline in a serial dilution until visualization of a single intact cell was achieved. Single cells were transferred to a PCR plate 1 cell per well and lysed using a lysis buffer consisting of Salt Mix 1M KCl 25 mM MgCl2 0.1M Tris HCl 0.1M DTT and the Arcturus PicoPure DNA Extraction Kit from Applied BioSystems. After the lysis buffer is added to each well the plate is run on the following thermal cycler protocol 56 C. for 1 hr 95 C. for 10 min 25 C. for 15 min 4 C. hold. The single genomic copies were then used as templates for a PCR reaction.

Genomic DNA from fresh frozen FF tissue was extracted using the DNeasy Blood and Tissue Kit catalog no. 69506 Qiagen according to the manufacturer s spin column protocol for purification of total DNA from animal tissues. DNA was extracted from FFPE samples with the QIAamp DNA FFPE Tissue Kit catalog no. 56404 Qiagen according to the manufacturer s instructions.

A proof of concept plasma model system was established by generating fragmented DNA mixtures for use as cell free nucleic acid size standards that resemble the size profiles of the cell free DNA cfDNA naturally found in plasma. To start 9 106 cells were lysed in hypotonic lysis buffer 20 mM Tris Cl pH 7.5 10 mM NaCl 3 mM MgCl2 for 15 minutes on ice before 10 Igepal CA 630 Sigma St. Louis Mo. was added to a final concentration of 0.5 . Nuclei were pelleted by centrifugation at 3 000 g for 10 minutes at 4 C. and then resuspended in 1 MNase Buffer New England BioLabs Ipswich Mass. before 1000 U of MNase New England BioLabs was added. Resuspended nuclei were incubated for 5 minutes at 37 C. to facilitate MNase digestion. Reactions were stopped by the addition of EDTA to a final concentration of 15 mM. Undigested chromatin was removed by centrifugation at 2 000 g for 1 minute. Fragmented DNA was purified using the DNA Clean Concentrator 500 kit catalog no. D4032 Zymo Research Irvine Calif. according to manufacturer s instructions. Fragmentation was confirmed by running the purified samples on a Bioanalyzer DNA 1000 chip Agilent Santa Clara Calif. . Mononucleosomes were purified by a 2 step purification strategy using AMPure XP Beckman Coulter Brea Calif. . First to remove large fragments 0.9 AMPure XP beads were added and allowed to bind before magnetic removal. Next the supernatant was transferred to a fresh tube additional AMPure XP beads were added to 2 and DNA was purified according to manufacturer s instructions. Mononucleosomal DNA fragment size approximately 150 nt was confirmed by running the samples on a Bioanalyzer DNA 1000 chip Agilent .

Child DNA was titrated into the corresponding father DNA to achieve artificial mixtures with different child DNA fractions. This method generates cell free size standards with 22q11 region CNVs which mimic cancer plasma samples with variable imbalance between copies of the two 22q11 homologs. Pure father samples were run as controls. Cell free size standards from cancer cell lines HCC1954 and HCC2218 were also generated by titrating with the corresponding matched normal cell line HCC1954BL and HCC2218BL respectively .

Chromosomal microarray analysis on fresh frozen tissue samples was performed using the Illumina CytoSNP 12 97 genotyping microarray platform as previously described 1 . Analysis of FFPE tissue samples using the Affymetrix OncoScan microarray platform was carried out according to the manufacturer s protocol.

For the 27 744 plex protocol samples were pre amplified for 15 cycles using PCR and 27 744 target specific assays an aliquot was then transferred to a second nested 15 cycle PCR reaction. Amplified samples were prepared for sequencing by adding barcoded tags in a 12 cycle PCR reaction. Thus for the 28 000 plex protocol 27 744 targets were amplified in a single reaction targets included SNPs from chromosomes 1 2 13 18 21 and X and regions 1p36 4p16 5p15 7q11 15q11 17p13 17p11 22q11 and 22q13. A modified version of this protocol was used for the 3 000 plex approach where 3 168 target specific assays were amplified using a 25 cycle PCR reaction allowing a focused analysis of SNPs from chromosomes 1 and 2 and the 22q11 focal region. Sequencing of amplicons was carried out using an Illumina HiSeq 2500 sequencer x tissue samples or 8 12 plasma samples were sequenced per run.

Data was plotted with the relative fraction of one allele arbitrarily chosen on the y axis and the SNP location along the chromosomal region on the x axis such that the observed allele fractions at each of the chromosomal regions indicate the overall proportion of the two haplotypes present in the sample note that sample heterogeneity may confound precise determination of the relative copy number of the two haplotypes in any given cell from measurements made on bulk sample.

Allelic data distributions were modeled for the following hypotheses i all cells are normal ii presence of cells with a homolog 1 deletion and iii presence of cells with a homolog 2 deletion. The likelihood of each of the hypotheses was calculated based on observed Next Generation Sequencing NGS data at multiple heterozygous SNPs sequencing and PCR related errors were taken into account. The algorithm compares predicted distributions with actual allelic distributions as measured from the sample in question employing a Bayesian based maximum likelihood approach to determine the relative likelihood of each hypothesis given the observed data across multiple tumor fractions and using the haplotype information deduced from the tumor sample corresponding to the same individual. For example consider a heterozygous SNP with genotype AB with dimorphic alleles arbitrarily labeled as A and B . If the homolog with allele A is deleted in some cells then we expect the ratio of A reads to total reads to go down. Similarly if the homolog with allele B is deleted then we expect the ratio of B reads to total reads to go down. The change in this ratio is proportional the fraction of tumor DNA present in the plasma. For cases where one of the deletion hypotheses is more likely than the normal hypothesis across a sufficiently large range of tumor fractions tumor DNA quantity is determined using a maximum likelihood estimation method across those tumor fractions otherwise tumor DNA fraction is estimated to be equal to zero.

The capacity of this SNP based massively multiplex PCR mmPCR approach to accurately detect CNVs copy number variants was established using four separate methods described below. The performance of the assay was demonstrated using as input gDNA both from large numbers of cells and from single cells DNA from FFPE tissue and artificial cell free DNA testing standards that simulate cell free circulating tumor DNA ctDNA made by mixing appropriately sized DNA from the tumor and germ line samples.

First an assay targeting 27 744 SNPs dispersed across 6 whole chromosomes and 9 additional focal regions that cover common deletion syndromes were used to analyze gDNA from 71 characterized cell line samples having a single deletion in one of the eight deletion syndrome regions p and q arms were analyzed separately. Sensitivity was 100 71 71 and specificity including all normal regions among affected samples and an additional 25 unaffected samples was also 100 1 849 1 849 .

Second six characterized cancer cell lines and XX normal cell lines were analyzed using a 3 168 plex a 27 744 plex PCR and a SNP microarray. Visual inspection of the plotted allele fractions showed similar fractions over all regions with apparent copy number variations representative plots shown . Data was plotted with the relative fraction of one allele arbitrarily chosen on the y axis and the SNP location along the chromosomal region on the x axis such that the observed allele fractions at each of the chromosomal regions indicate the overall proportion of the two haplotypes present in the sample note that sample heterogeneity may confound precise determination of the relative copy number of the two haplotypes in any given cell from measurements made on bulk sample. To show that the assay has single molecule sensitivity individual cells were isolated from the aforementioned cancer cell lines and were analyzed as described above Plotted allele fractions from single cells were similar to both those observed from large quantities and also SNP arrays with minor allowances made for expected heterogeneity. To mimic a heterogeneous tumor profile and determine the capacity for this method to detect CNVs present in a subpopulation of cells cancer cells were mixed with normal cells at different ratios. Using a linear titration of the cancer cell line HCC2218 into the matched normal control cell line a corresponding linear change in the allele ratio was determined.

Third the ability of this methodology to accurately detect CNVs in tumor tissue was validated by visual comparison of three FF tissue samples using the 3 000 plex PCR and SNP microarray similar allele fractions were observed. Buffy coat samples from each of the samples were included as germline controls no CNVs were detected in these samples by either method. The same mmPCR methodology was successfully applied to detect CNVs from formalin fixed paraffin embedded FFPE tissue samples which typically pose a challenge to SNP microarrays. Similar allele fraction patterns were observed among three tumor samples using the 3 000 plex PCR approach and Affymetrix Oncoscan a commercially available assay that is capable of evaluating CNVs from FFPE samples two examples are shown in. Importantly no modifications of the multiplex PCR method were required to characterize the FFPE samples.

To determine whether the CNVs detected in the tumor tissue samples were somatic CNVs a subset of 13 breast cancer samples which had buffy coat adjacent non tumor tissue and tumor tissue samples available were analyzed. No CNVs were observed in any of the buffy coat samples while CNVs were detected in 84.6 11 13 of the tumor tissue samples. In xx x 13 of non adjacent tumor tissue samples CNVs were observed give details were they the same CNVs as in tumor tissue etc. .

Fourth validation of ctDNA quantification in plasma samples was carried out using artificial cell free DNA standards Plasmart mimicking plasma ctDNA. Cell free nucleic acid standards corresponding to ctDNA were created in one of two ways Cell free nucleic acid standards corresponding to ctDNA with well characterized CNVs were created by titrating DNA from a child a known CNV in the 22q11.2 region resulting from deletion of the maternal haplotype into the corresponding father s DNA which had a normal copy number at the 22q region. Alternately cell free nucleic acid standards for tumors were created by titrating tumor cell lines with the corresponding matched normal cell lines see Materials and Methods . Prior to mixing the DNA of the samples was processed enzymatically to recreate the DNA fragment size distribution observed in natural cell free DNA which is derived from an apoptotic process. The limit of quantitation LOQ is defined as the lowest concentration at which a mutation could be reliably detected with a given level of accuracy and precision. To determine the LOQ cell free nucleic acid standards containing various child father DNA ratios corresponding to ctDNA levels of 0 10 and cell free nucleic acid standards containing various tumor matched normal DNA ratios corresponding to ctDNA levels of 0 50 were run. Copy number variations were detected in samples above 0.2 ctDNA or above 0.45 ctDNA shown in . There were 12 samples per run and the DOR per SNP.

Following validation of the mmPCR massively multiplexed PCR method the technique was applied to the detection of CNVs in tumor tissue and plasma samples from 97 cancer patients. The 3 000 plex mmPCR method focusing on five chromosomal regions was applied for analyses of CNVs in these samples as this focused approach allows a greater depth of read. Overall somatic copy number variations were detected in at least one of the five regions assayed in 88.9 40 45 of breast tumor tissue samples 66.7 16 24 of lung tumor tissue samples and 46.4 13 28 of colon tumor tissue samples and were detected across all five regions of interest evaluated. The regions of interest included in this panel were not focused on cancer related CNVs use of a targeted panel of CNVs commonly associated with cancer would be expected to provide significantly greater coverage.

The ability of the mmPCR method to detect the somatic CNVs observed in the tumor tissue in the matched patient plasma samples was then investigated. Overall copy number variations were detected in x 40 of breast plasma samples y 16 of lung plasma samples and z 13 of colon plasma samples and were detected across all five regions of interest evaluated.

One of the potential advantages of liquid biopsies is that ctDNA may reveal the spectrum of tumor associated mutations that exist in the tumor unlike a focal tumor biopsy that could miss some or all tumor associated CNVs because of tissue heterogeneity. To determine the effects of tumor heterogeneity on the detection of CNVs in plasma versus focal biopsies a number of subsections from eight breast cancer samples were analyzed and compared to the matching plasma sample. XX 40 regions assayed in the 8 samples showed significant heterogeneity between biopsies. Of the 40 XX regions that were mostly homogenous Q involved a CNV and W Q of those CNVs were observed in the plasma. In the Y XX of the regions where there was a CNV in at least one of the biopsies the CNV was observed in B Y of the plasma. Interestingly one of these samples had a CNV on 22q11 detected in the plasma that was not visible in some of the tumor tissue sections while a second sample had a CNV on 1q detected in the plasma that was not visible in some of the tumor sections

It is understood that the examples and embodiments described herein are for illustrative purposes only and that various modifications or changes in light thereof will be suggested to persons skilled in the art and are to be included within the spirit and purview of this application and scope of the appended claims.

While the foregoing invention has been described in some detail for purposes of clarity and understanding it will be clear to one skilled in the art from a reading of this disclosure that various changes in form and detail can be made without departing from the true scope of the invention. For example many of the methods compositions described above can be used in various combinations.

All publications patents patent applications and or other documents cited in this application are incorporated by reference in their entirety for all purposes to the same extent as if each individual publication patent patent application and or other document were individually indicated to be incorporated by reference for all purposes.

2. The prenatal nucleic acid proficiency testing standard composition according to claim 1 wherein the first nucleosomal nucleic acid preparation is derived from a primary cell source.

3. The prenatal nucleic acid proficiency testing standard composition according to claim 1 wherein the first nucleosomal nucleic acid preparation is derived from a cell line.

4 A composition according to claim 1 wherein the first cell source and the second cell source are cell lines.

5 A composition according to claim 1 wherein the first cell source and the second cell source are primary cell sources.

6 The method of claim 1 wherein the first cell source is a primary cell source and the second cell source is a cell line.

7 The composition of claim 1 wherein the second cell source is a primary cell source and the first cell source is a cell line.

8 The composition according to claims 1 7 wherein the first cell source is genetically related to the second cell source.

9 The composition according to claims 1 7 wherein the first nucleosomal nucleic acid preparation has been prepared with an endonuclease.

10 The composition according to claims 1 7 wherein the first nucleosomal nucleic acid preparation has been prepared with a micrococcal endonuclease.

11 The composition according to claims 1 7 wherein the first nucleosomal nucleic acid preparation has been prepared with an endonuclease.

12 The composition according to claims 1 7 wherein the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid are nucleosomal ladder fractions.

13 The composition according to claim 12 wherein the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid comprise mononucleosomal ladder fractions.

14 The composition according to claim 13 wherein the first nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions.

15 The composition according to claim 14 wherein the first nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

16 The composition according to claim 13 wherein the second nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions.

17 The composition according to claim 16 wherein the second nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

18 The composition according to claim 14 wherein the second nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions.

19 The composition according to claim 15 wherein the second nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

20 A composition according to claim 2 wherein the primary cell source is blood cells from a buffy coat layer.

21 The composition of claims 1 19 wherein the amount of the second nucleosomal nucleic acid preparation is less than 40 of the total nucleic acid in the composition.

22 The composition of claim 21 wherein the amount of the second nucleosomal nucleic acid preparation is less than 30 of the total nucleic acid in the composition.

23 The composition of claim 22 wherein the second nucleosomal nucleic acid preparation is less than 20 of the total nucleic acid in the composition.

24 The composition of claim 23 wherein the amount of the second nucleosomal nucleic acid preparation is less than 10 of the total nucleic acid in the composition.

28 The composition of claim 8 wherein the second cell source is a cancerous cell and the first cell source is a non cancerous cell.

29 The composition of claim 28 wherein the first cell source and the second cell source are derived from the same individual.

30 The composition of claim 29 wherein the first cell source and the second cell source are cultured cell lines.

31 The composition of claim 25 wherein the amount of the second nucleosomal nucleic acid preparation is less than of the total nucleic acid in the composition.

32 The composition of claim 28 wherein the amount of the second nucleosomal nucleic acid preparation is less than 30 of the total nucleic acid in the composition.

33 The composition of claim 29 wherein the second nucleosomal nucleic acid preparation is less than 20 of the total nucleic acid in the composition.

34 The composition of claim 30 wherein the amount of the second nucleosomal nucleic acid preparation is less than 10 of the total nucleic acid in the composition.

35 The composition of claims 1 19 wherein the a first nucleosomal nucleic acid preparation is prepared by isolating nuclei from the first cell source and the second nucleic acid preparation is prepared by isolating nuclei from the second cell source.

36 The composition of claims 1 19 wherein the amount of the second nucleosomal nucleic acid preparation is less than 40 of the total nucleic acid in the composition.

37 The composition of claim 32 wherein the amount of the second nucleosomal nucleic acid preparation is less than 30 of the total nucleic acid in the composition.

38 The composition of claim 33 wherein the second nucleosomal nucleic acid preparation is less than 20 of the total nucleic acid in the composition.

39 The composition of claim 34 wherein the amount of the second nucleosomal nucleic acid preparation is less than 10 of the total nucleic acid in the composition.

40 The composition of claims 1 31 wherein the nucleosomal preparation is obtained the first cell source or the second cell source after apoptosis has been is induced in the cell source.

41 A method of making a cell free nucleic acid diagnostic testing standard composition comprising mixing a

42 The method according to claim 37 wherein the first nucleosomal nucleic acid preparation is derived from a primary cell source.

43 The method according to claim 37 wherein the first nucleosomal nucleic acid preparation is derived from a cell line.

44 A method according to claim 37 wherein the first cell source and the second cell source are cell lines.

45 A method according to claim 37 wherein the first cell source and the second cell source are primary cell sources.

46 The method of claim 37 wherein the first cell source is a primary cell source and the second cell source is a cell line.

47 The method of claim 37 wherein the second cell source is a primary cell source and the first cell source is a cell line.

48 The method according to claims 37 43 wherein the first cell source is genetically related to the second cell source.

49 The method according to claims 37 43 wherein the first nucleosomal nucleic acid preparation has been prepared with an endonuclease.

50 The method according to claims 37 43 wherein the first nucleosomal nucleic acid preparation has been prepared with a micrococcal endonuclease.

51 The method according to claims 37 43 wherein the first nucleosomal nucleic acid preparation has been prepared with an endonuclease.

52 The method according to claims 37 43 wherein the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid are nucleosomal ladder fractions.

53 The method according to claim 48 wherein the first nucleosomal nucleic acid preparation and the second nucleosomal nucleic acid comprise mononucleosomal ladder fractions.

54 The method according to claim 49 wherein the first nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions.

55 The method according to claim 50 wherein the first nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

56 The method according to claim 51 wherein the second nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions.

57 The method according to claim 52 wherein the second nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

58 The method according to claim 50 wherein the second nucleosomal nucleic acid preparation comprises dinucleosomal ladder fractions.

59 The method according to claim 51 wherein the second nucleosomal nucleic acid preparation comprises trinucleosomal ladder fractions.

60 A method according to claim 38 wherein the primary cell source is blood cells from a buffy coat layer.

61 The method of claims 37 55 wherein the amount of the second nucleosomal nucleic acid preparation is less than 40 of the second nucleic acid preparation.

62 The method of claim 57 wherein the amount of the second nucleosomal nucleic acid preparation is less than 30 of the second nucleic acid preparation.

63 The method of claim 58 wherein the second nucleosomal nucleic acid preparation is less than 20 of the second nucleic acid preparation.

64 The method of claim 57 wherein the amount of the second nucleosomal nucleic acid preparation is less than 10 of the second nucleic acid preparation.

68 The method of claim 61 wherein the amount of the second nucleosomal nucleic acid preparation is less than 40 of the second nucleic acid preparation.

69 The method of claim 64 wherein the amount of the second nucleosomal nucleic acid preparation is less than 30 of the second nucleic acid preparation.

70 The method of claim 65 wherein the second nucleosomal nucleic acid preparation is less than 20 of the second nucleic acid preparation.

71 The method of claim 66 wherein the amount of the second nucleosomal nucleic acid preparation is less than 10 of the second nucleic acid preparation.

72 The method of claims 37 55 wherein the a first nucleosomal nucleic acid preparation is prepared by isolating nuclei from the first cell source and the second nucleic acid preparation is prepared by isolating nuclei from the second cell source. The method of claims 1 19 wherein the amount of the second nucleosomal nucleic acid preparation is less than of the total nucleic acid in the composition.

73 The method of claim 68 wherein the amount of the second nucleosomal nucleic acid preparation is less than 30 of the total nucleic acid in the composition.

74 The method of claim 69 wherein the second nucleosomal nucleic acid preparation is less than 20 of the second nucleic acid preparation.

75 The method of claim 70 wherein the amount of the second nucleosomal nucleic acid preparation is less than 10 of the total nucleic acid in the composition.

76 The method of claims 37 67 wherein the nucleic acid preparation is obtained the first cell source or the second cell source after apoptosis has been is induced in the cell source.

77 A prenatal nucleic acid proficiency testing standard composition made by the methods of claims 37 72.

78 A kit for first prenatal proficiency testing the kit comprising a prenatal nucleic acid proficiency testing standard composition and a second prenatal nucleic acid proficiency testing standard composition wherein prenatal nucleic acid proficiency testing standard compositions are different from each other and each of the prenatal nucleic acid proficiency testing standard compositions is according to claims 1 36.

