---

title: Microfluidic device-based nucleic acid purification method
abstract: A method is provided for purifying nucleic acid from a sample in a microfluidic device. The method can be used to purify nucleic acids from any source known in the art that comprises nucleic acids, such as prokaryotic or eukaryotic organisms, viruses, cell, tissues, organs, etc. In a specific example, the tissue is whole blood. The method for purifying nucleic acid may run fully automated in the microfluidic device.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09169479&OS=09169479&RS=09169479
owner: RHEONIX, INC.
number: 09169479
owner_city: Ithaca
owner_country: US
publication_date: 20140326
---
This application is a divisional of U.S. application Ser. No. 13 400 857 filed on Feb. 21 2012 and entitled Microfluidic Device Based Nucleic Acid Purification Method which claims priority to U.S. Provisional Patent Application No. 61 444 952 filed on Feb. 21 2011 the entire disclosures of which are hereby incorporated herein by reference.

Embodiments of the invention are directed generally to methods for preparing purifying amplifying and detecting biological molecules of interest such as nucleic acids and more particularly to such methods performed using microfluidic devices.

The use of molecular diagnostics has expanded greatly since its inception in the early 1980s particularly as a means to permit the detection of slow growing or fastidious bacteria responsible for infectious diseases. The detection of viral pathogens including viral load testing has also been significantly improved by molecular diagnostics. As more data have become available regarding the human genome the use of molecular diagnostics in pharmacogenomics companion diagnostics and other personalized medicine applications continues to gain momentum.

Nucleic acid amplification is a standard technique known in the art by which nucleic acids may be isolated and more accurately and efficiently manipulated for use in molecular diagnostics and other nucleic acid screening purposes. Various methods have been described in the literature for amplifying nucleic acids such as deoxyribonucleic acid DNA and ribonucleic acid RNA present in a sample. Among these the most widely practiced is the polymerase chain reaction PCR described in U.S. Pat. No. 4 683 195 Mullis et al. entitled Process for amplifying detecting and or cloning nucleic acid sequences issued Jul. 28 1987 and U.S. Pat. No. 4 683 202 Mullis entitled Process for amplifying nucleic acid sequences issued Jul. 28 1987 . Briefly PCR consists of amplifying denatured complementary strands of target nucleic acid by annealing each strand to a short oligonucleotide primer wherein the primers are chosen so as to flank the sequence of interest. The primers are then mediated by a polymerase enzyme to yield extension products that are themselves complementary to the primers and hence serve as templates for synthesis of additional copies of the target sequence. Each successive cycle of denaturation primer annealing and primer extension essentially doubles the amount of target synthesized in the previous cycle resulting in exponential accumulation of the target.

PCR methodologies in general suffer from several limitations that are well known in the art. One such limitation is owing to the poor fidelity of commonly used thermostable polymerase enzymes such as Taq. This results in nucleotide base misincorporations that are propagated from one cycle to the next. It is estimated that such misincorporations may occur as often as once per one thousand bases of incorporation. A second limitation is that different cDNAs are amplified with different efficiencies resulting in underrepresentation of some cDNA sequences and overrepresentation of others in the amplified product. Even a small difference in efficiency may result in a several thousand fold differential in the representation of these cDNAs in the product after only as few as 30 cycles of amplification.

Another limitation is the presence of inhibitors of PCR in the starting material e.g. hemoglobin in blood . These inhibitors are often carried through purification and either limit or completely impede amplification reactions performed with the nucleic acids derived from the purification. Therefore this provides another rationale for needing better upstream purification strategies see e.g. J. Bessetti Profiles in DNA PCR Inhibition An Introduction to PCR Inhibitors Promega Corporation March 2007 .

Also significant is that nucleic acid amplification is only as accurate as the starting sample of nucleic acid to be amplified. Nucleic acid e.g. DNA or RNA of low purity will yield amplification products that may not reflect the composition of the starting sample and therefore cannot be used or relied on for diagnostic purposes.

It is also recognized that the conventional practice of biochemistry and molecular biology can require physical process resources on a scale that are frequently inversely proportional to the size of the subject being studied. For example the apparatus and process chemistry associated with the preparation and purification of a biological sample such as a nucleic acid fragment for prospective analysis may easily require a full scale bio laboratory with sterile facilities. Furthermore an environmentally isolated facility of similar scale may typically be required to carry out the known nucleic acid amplification procedures such as polymerase chain reaction PCR for amplifying the nucleic acid fragment.

There is therefore a need in the art for improved methods of nucleic acid purification for use in nucleic acid amplification techniques so that nucleic acids may be more accurately and efficiently purified identified and manipulated for use in molecular diagnostics and other nucleic acid screening purposes. There also exists a need for improved microfluidic systems for processing fluids for analysis of biological or chemical samples and in particular in the detection and analysis of biologically active macromolecules derived from such samples such as DNA RNA amino acids and proteins. It is beneficial and advantageous that the systems are mass producible inexpensive and preferably disposable that the systems be simple to operate and that many or substantially all of the fluid processing steps be automated that the systems be customizable and be modular such that the system can be easily and rapidly reconfigured to suit various applications in which the detection of macromolecules is desired and that the systems be able to provide straightforward and meaningful assay results. A more thorough discussion of the challenges and shortcomings in the art as well as exemplary solutions that may be utilized in conjunction with the teachings of the instant invention are disclosed in applicant s copending U.S. application Ser. No. 13 033 165 Pub. No. US2011 0275058 entitled Self contained Biological Assay Apparatus Methods and Applications the subject matter of which is incorporated herein by reference in its entirety.

An embodiment of the instant invention is a microfluidic device based method for purifying nucleic acid e.g. DNA or RNA from a lysed liquid sample solution. The method includes the steps of providing a suitable microfluidic device adding an organic solvent to the lysed solution adding magnetic beads to the lysed solution binding the nucleic acid to the beads separating the nucleic acid bound magnetic beads from the solution washing the nucleic acid bound magnetic beads in a wash solution separating the washed beads from the wash solution eluting the nucleic acid from the washed beads reestablishing an appropriate molecular charge on the eluted nucleic acid recapturing the nucleic acid from the solution washing the recaptured nucleic acid with a wash solution and eluting the recaptured nucleic acid to obtain a purified nucleic acid. As used herein and in the claims a suitable microfluidic device means a microfluidic device or system a system generally includes a companion instrument that interfaces with the microfluidic device to control the process steps performed by the microfluidic device having the capability to support and carry out the disclosed processes and claimed process steps. An exemplary suitable microfluidic device and companion instrument is one such as is disclosed in U.S. application Ser. No. 13 033 165 id. . Other examples of suitable microfluidic devices and systems are known in the art. For example the device may be composed of any materials commonly used for the production of microfluidic devices such as but not limited to silicon glass quartz elastomeric materials e.g. PDMS and polymers e.g. polystyrene acrylics COC s and others . The microfluidic devices may be produced by any known methods for producing microfluidic systems such as photolithography embossing laser etching machining injection molding and the devices may incorporate various materials or be constructed from a single material. The device may incorporate discrete assembled parts or be composed of various layers designed to provide the functions required to perform the method. The microfluidic device may be operated manually or robotically where all or at least some of the reagents are added to the device during the performance of the purification or the device may include all of the reagents required for the purification placed in reservoirs and operate fully automatically. Generally the microfluidic device will be associated with an instrument capable of implementing the functions designed into the microfluidic device. According to various non limiting exemplary aspects the method may further include the following steps features or characteristics 

Another embodiment of the invention is a microfluidic device based method for purifying nucleic acid e.g. DNA or RNA from a source containing nucleic acids. The method includes the steps of obtaining a liquid sample of the source lysing the liquid sample to create a liquid sample solution providing a suitable microfluidic device adding an organic solvent to the lysed solution adding magnetic beads to the lysed solution agitating the solution for a time sufficient to bind the nucleic acid to the beads separating the nucleic acid bound magnetic beads from the solution washing the nucleic acid bound magnetic beads in a wash solution separating the washed beads from the wash solution eluting the nucleic acid from the washed beads reestablishing an appropriate molecular charge on the eluted nucleic acid capturing the nucleic acid from the solution and eluting the captured nucleic acid to obtain a purified nucleic acid. According to various non limiting exemplary aspects the method may further include the following steps features or characteristics 

With further reference to magnet is raised into place under separation concentration channel and the lysed biological sample solution is pumped to waste reservoir through separation concentration channel where the magnetic beads with the captured nucleic acids are captured in separation concentration channel by the magnetic field generated by magnet . Magnet is then lowered out of place under separation concentration channel to remove the magnetic field that captured the magnetic beads. Then dispense a wash buffer into common reagent reservoir preparation pump to separation concentration channel and circulate alternatively clockwise counterclockwise multiple times through separation concentration channel to re suspend and wash the beads. Raise magnet into place under separation concentration channel and pump the solution containing the washed beads to waste reservoir through separation concentration channel where the magnetic beads with the captured nucleic acids are captured in separation concentration channel by the magnetic field generated by magnet . Lower magnet out of place under separation concentration channel to remove the magnetic field that captured the magnetic beads. Dispense the same or another wash buffer into common reagent reservoir preparation pump to separation concentration channel and circulate alternatively clockwise counterclockwise multiple times through separation concentration channel to re suspend and wash the beads. Raise magnet into place under separation concentration channel and pump the solution containing the washed beads to waste reservoir through separation concentration channel where the magnetic beads with the captured nucleic acids are captured in separation concentration channel by the magnetic field generated by magnet . Continue the above wash steps for the number of repetitions required by the assay. Dispense elution buffer into common reagent reservoir preparation and pump it to waste reservoir to clear any residual reagents from the reservoir channels and diaphragms of the preparation area. Lower magnet and dispense elution buffer into common reagent reservoir preparation and pump it to separation concentration channel and circulate alternatively clockwise counterclockwise multiple times through separation concentration channel to elute the nucleic acids from the beads. Raise magnet into place under separation channel and pump the solution containing the eluted nucleic acids to common reagent reservoir preparation through separation concentration channel where the magnetic beads are captured in separation concentration channel by the magnetic field generated by magnet .

With further reference to once the solution containing the eluted nucleic acids is pumped into common reagent reservoir dispense a high salt buffer into common reagent reservoir preparation . Dispense an organic alcohol e.g. ethanol into common reagent reservoir preparation so that the eluted nucleic acids now have a reestablished molecular charge so that the nucleic acids are capable of binding to an appropriate substrate for further purification. Pump the contents of common reagent reservoir preparation which now contains eluted nucleic acids with an appropriate molecular charge reestablished onto the top of the silica filter in silica filter reservoir pull the contents through silica filter in silica filter reservoir so that the nucleic acids bind to the silica filter and pump the remaining solution to waste reservoir .

With further reference to dispense the same or another wash buffer into common reagent reservoir elution pump onto the top of the silica filter in silica filter reservoir pull the contents through silica filter in silica filter reservoir and pump to waste reservoir . Repeat with the same or an alternative wash buffers as required by the assay.

With further reference to dispense elution buffer into reservoir elution and pump it up through the bottom of the silica filter in silica filter reservoir .

With further reference to pull the contents of silica filter reservoir that now contains eluted nucleic acids through the silica filter pump it to elution reservoir center and then equally to elution reservoir L and R or alternatively pump it directly in equal amounts to reservoir L and R by bypassing elution reservoir center . The eluted nucleic acids are then in condition for further analysis or amplification. The nucleic acids purified in this manner contain lower concentrations of impurities that inhibit amplification reactions commonly performed using purified nucleic acids.

The method embodiments disclosed herein can be used to purify nucleic acids from any source known in the art that comprises nucleic acids such as prokaryotic or eukaryotic organisms viruses cell tissues organs etc. In an illustrative aspect the tissue is whole blood. In another illustrative aspect the sample is liquefied or is dissolved in a liquid. In another illustrative aspect the method eliminates the step of culturing or growing cells or organisms containing a nucleic acid of interest prior to nucleic acid purification.

According to an exemplary aspect the method for purifying nucleic acid e.g. DNA can be preceded by the preparatory steps as follows lysing the liquid sample e.g. by adding lyticase or lysozyme enzyme or other suitable lytic enzyme chaotropic agent sonication laser lysis or another lysing system known in the art to have similar properties and optionally adding RNAse or DNAse treatment in a suitable microfluidic device thereby forming a solution comprising the liquid sample and cellular components agitating the solution incubating the solution at a suitable temperature e.g. at room temperature adding proteinase K or other proteinase known in the art to have similar properties to the solution agitating the solution incubating the solution at a suitable temperature e.g. at room temperature or up to 55 C. adding an art known lysis binding buffer to the solution any lysis buffer binding buffer known in the art can be used provided that it is optimized using art known methods for nucleic acid binding to the purification matrix agitating the solution incubating the solution e.g. at room temperature adding magnetic beads to the solution any suitable magnetic beads known in the art can be used wherein the beads are coated with a silica like chemistry that allows the nucleic acid to bind e.g. Invitrogen Dynabeads MyOne SILANE and again agitating the solution. Such preparatory steps may be conducted on standard bench top equipment or in a microfluidic chip that is the same chip or a different chip than the chip conducting the purification.

After the above preparatory steps are conducted the purification method can be conducted comprising the following steps adding about an equal volume of a suitable art known organic solvent e.g. isopropyl alcohol to the solution agitating the solution incubating the solution at a suitable temperature e.g. at room temperature collecting the magnetic beads e.g. by placing a magnet on the outside of the container or microfluidic system removing the supernatant using the microfluidic device adding a suitable art known wash buffer optimized for nucleic acid e.g. DNA binding to the magnetic beads for silica based binding wash buffers typically include a chaotropic and non chaotropic salt pH buffer ethanol and optionally detergent. Any buffer and wash system that is compatible with nucleic acid binding to silica can be used provided that the combination has been optimized for the system using art known methods agitating the solution to re suspend the beads collecting the beads optionally repeating the steps of adding the wash buffer to the magnetic beads agitating the solution to re suspend the beads and collecting the beads the residual liquid left in the container prior to DNA elution can be removed so that any buffer and or organic solvent contamination or interference with elution is removed adding a suitable art known nucleic acid elution buffer e.g. nuclease free water agitating the solution incubating the solution at a suitable temperature e.g. at room temperature to elute the nucleic acid e.g. DNA bound to the magnetic beads in this exemplary aspect RNAse treatment is not included since total nucleic acid including a desired DNA of interest is being bound and eluted rather only the DNA present is analyzed. Alternatively an RNAse treatment step could be included to remove RNA if desired or a DNAse step to remove all DNA to only purify RNA collecting the beads e.g. by placing a magnet on the outside of the container or microfluidic system adding a suitable nucleic acid lysis binding buffer to the eluted DNA agitating the solution adding about an equal volume of suitable art known organic solvent e.g. absolute ethanol to the solution agitating the solution transferring the solution to a nucleic acid purification column or filter any suitable column or filter for purifying nucleic acid known in the art can be used. In an exemplary aspect a silica filter is used in the microfluidic device eluting purified nucleic acid e.g. DNA from the purification column or filter and collecting the eluted nucleic acid e.g. DNA .

Once the eluted purified nucleic acid from the above described method for purifying nucleic acid is collected the purified nucleic acid can be e.g. estimated run on a standard gel for genomic DNA analysis and or subjected to PCR amplification or other nucleic acid amplification methods known in the art e.g. to amplify a target gene.

The embodied method for purifying nucleic acid can be adapted to be automated and run in the microfluidic device. For example in U.S. application Ser. No. 13 033 165 id. FIGS. 24A 24K and FIG. 25 show the flow of solution during a two stage DNA purification conducted on a microfluidic device.

1 Add 124 units of Lyticase enzyme to 500 l of whole blood and agitate the solution. Incubate the sample at room temperature for 30 minutes.

2 Add 71.4 l 20 mg ml proteinase K to the sample briefly agitate the solution and incubate the sample at room temperature for 2 minutes.

3 Add 500 l of lysis binding buffer to the sample agitate the solution and incubate at room temperature for 10 minutes.

5 Add 571 l of isopropyl alcohol agitate and incubate for 5 minutes at room temperature while agitating.

6 Collect the beads by placing a magnet on the outside of the tube channel or reservoir and carefully remove the supernatant using a microfluidic system. Dispose of supernatant.

7 Add 950 l of wash buffer to the magnetic beads and agitate the solution to resuspend the beads. Collect the beads and dispose of the supernatant.

9 Add 950 l of wash buffer 2 to the magnetic beads and agitate the solution to resuspend the beads. Collect the beads and dispose of the supernatant.

10 Remove any residual liquid left in the tube channel or reservoir prior to DNA elution to avoid organic solvent contamination.

11 Add 100 l of nuclease free water agitate the solution and incubate at room temperature for two minutes to elute the DNA bound to the magnetic beads.

12 Collect the beads by placing a magnet on the outside of the tube channel or reservoir and carefully transfer the supernatant to the second stage.

13 Add 100 l of nucleic acid lysis binding buffer to the 100 l of eluted DNA and agitate the solution.

15 Transfer the solution to a silica filter insert that is placed in a tube channel or reservoir of the microfluidic system.

19 Pump 50 l of nuclease free water backwards through the silica filter then reverse the direction of the fluid and collect the eluted DNA when the fluid exits the silica filter.

The purified DNA may then be estimated run on a gel for genomic DNA analysis and or subjected to PCR amplification or other suitable amplification system to amplify target gene s .

All references including publications patent applications and patents cited herein are hereby incorporated by reference in their entireties to the same extent as if each reference was individually and specifically indicated to be incorporated by reference and was set forth in its entirety herein.

The use of the terms a and an and the and similar referents in the context of describing the invention especially in the context of the following claims is to be construed to cover both the singular and the plural unless otherwise indicated herein or clearly contradicted by context. The terms comprising having including and containing are to be construed as open ended terms i.e. meaning including but not limited to unless otherwise noted. The term connected is to be construed as partly or wholly contained within attached to or joined together even if there is something intervening.

The recitation of ranges of values herein is merely intended to serve as a shorthand method of referring individually to each separate value falling within the range unless otherwise indicated herein and each separate value is incorporated into the specification as if it was individually recited herein.

All methods described herein can be performed in any suitable order unless otherwise indicated herein or otherwise clearly contradicted by context. The use of any and all examples or exemplary language e.g. such as provided herein is intended merely to better illuminate embodiments of the invention and does not impose a limitation on the scope of the invention unless otherwise claimed.

No language in the specification should be construed as indicating any non claimed element as essential to the practice of the invention.

It will be apparent to those skilled in the art that various modifications and variations can be made to the present invention without departing from the spirit and scope of the invention. There is no intention to limit the invention to the specific form or forms disclosed but on the contrary the intention is to cover all modifications alternative constructions and equivalents falling within the spirit and scope of the invention as defined in the appended claims. Thus it is intended that the present invention cover the modifications and variations of this invention provided they come within the scope of the appended claims and their equivalents. Therefore the embodiments are illustrative of the invention rather than limiting of the invention. Revisions and modifications may be made to methods materials structures and dimensions in accordance with the embodiments.

